<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html lang="en-GB" xml:lang="en-GB" xmlns:og="http://ogp.me/ns#" xmlns:m="http://www.w3.org/1998/Math/MathML" xmlns="http://www.w3.org/1999/xhtml" id="js">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">


<meta name="wpd_version" content="0.2">
<meta name="wpd_baseurl" content="http://www.biomedcentral.com/1755-8794/5/17/">
<meta name="wpd_url" content="http://www.biomedcentral.com/1755-8794/5/17/">
<meta name="wpd_date" content="2013-09-23T11:52Z">


 
    
    
                                                                                                                                                                                                            
                                                                                                                                        
    
            

            
        <script async="" src="cb=gapi.loaded_1"></script><script async="" src="cb=gapi.loaded_0"></script><script src="widgets.js" id="twitter-wjs"></script><script gapi_processed="true" src="plusone.js" async="" type="text/javascript"></script><script src="all.js" id="facebook-jssdk"></script><script src="ga.js" async="" type="text/javascript"></script><script>window.bmcIsMobile = "classic";</script>
        
        
        <title>BMC Medical Genomics | Full text | Molecular diagnostics for congenital hearing loss
including 15 deafness genes using a next generation
sequencing platform</title>
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
<link rel="apple-touch-icon" sizes="72x72" href="http://www.biomedcentral.com/sites/2050/images/70.gif">
<meta http-equiv="X-UA-Compatible" content="IE=7">


                
    
		
		
		
<!-- <?xml version="1.0" encoding="UTF-8"?>
<rdf:RDF xmlns:cc="http://web.resource.org/cc/" xmlns="http://purl.org/rss/1.0/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:prism="http://prismstandard.org/namespaces/1.2/basic/">
<cc:Work rdf:about="http://www.biomedcentral.com/1755-8794/5/17">
<cc:license rdf:resource="http://creativecommons.org/licenses/by/2.0/"/>
</cc:Work>
<cc:License rdf:about="http://creativecommons.org/licenses/by/2.0/">
<cc:permits rdf:resource="http://web.resource.org/cc/Reproduction"/>
<cc:permits rdf:resource="http://web.resource.org/cc/Distribution"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Notice"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Attribution"/>
<cc:permits rdf:resource="http://web.resource.org/cc/DerivativeWorks"/>
</cc:License>
<item rdf:about="http://www.biomedcentral.com/1755-8794/5/17">
<title>Molecular diagnostics for congenital hearing loss including 15 deafness genes using a next generation sequencing platform</title>
<dc:title>Molecular diagnostics for congenital hearing loss including 15 deafness genes using a next generation sequencing platform</dc:title>
<dc:creator>De Keulenaer, Sarah</dc:creator>
<dc:creator>Hellemans, Jan</dc:creator>
<dc:creator>Lefever, Steve</dc:creator>
<dc:creator>Renard, Jean-Pierre</dc:creator>
<dc:creator>De Schrijver, Joachim</dc:creator>
<dc:creator>Van de Voorde, Hendrik</dc:creator>
<dc:creator>Tabatabaiefar, Mohammad A</dc:creator>
<dc:creator>Van Nieuwerburgh, Filip</dc:creator>
<dc:creator>Flamez, Daisy</dc:creator>
<dc:creator>Pattyn, Filip</dc:creator>
<dc:creator>Scharlaken, Bieke</dc:creator>
<dc:creator>Deforce, Dieter</dc:creator>
<dc:creator>Bekaert, Sofie</dc:creator>
<dc:creator>Van Criekinge, Wim</dc:creator>
<dc:creator>Vandesompele, Jo</dc:creator>
<dc:creator>Van Camp, Guy</dc:creator>
<dc:creator>Coucke, Paul</dc:creator>
<dc:identifier>info:doi/10.1186/1755-8794-5-17</dc:identifier>
<dc:identifier>info:pmid/22607986</dc:identifier>
<dc:source>BMC Medical Genomics 2012, 5:17</dc:source>
<dc:date>2012-05-18</dc:date>

<prism:publicationName>BMC Medical Genomics</prism:publicationName>
<prism:publicationDate>2012-05-18</prism:publicationDate>
<prism:volume>5</prism:volume>
<prism:number>1</prism:number>
<prism:section>Research article</prism:section>
<prism:startingPage>17</prism:startingPage>
<prism:copyright>2012 De Keulenaer et al.; licensee BioMed Central Ltd.</prism:copyright>
</item>
</rdf:RDF>
 -->
 
 
<meta name="citation_journal_title" content="BMC Medical Genomics">
<meta name="citation_publisher" content="BioMed Central Ltd">


<meta name="citation_author" content="Sarah De Keulenaer">
<meta name="citation_author_institution" content="NXTGNT, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Jan Hellemans">
<meta name="citation_author_institution" content="NXTGNT, Ghent University, Ghent, Belgium">
<meta name="citation_author_institution" content="Center for Medical Genetics, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Steve Lefever">
<meta name="citation_author_institution" content="Center for Medical Genetics, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Jean-Pierre Renard">
<meta name="citation_author_institution" content="Biobix, Laboratory for Bioinformatics and Computational Genomics, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Joachim De Schrijver">
<meta name="citation_author_institution" content="Biobix, Laboratory for Bioinformatics and Computational Genomics, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Hendrik Van de Voorde">
<meta name="citation_author_institution" content="Center for Medical Genetics, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Mohammad A Tabatabaiefar">
<meta name="citation_author_institution" content="Department of Medical Genetics, University of Antwerp, Antwerp, Belgium">
<meta name="citation_author_institution" content="Department of Medical Genetics, School of Medicine, Ahvaz Jundishapur University of Medical Sciences, Ahvaz, Iran">
<meta name="citation_author" content="Filip Van Nieuwerburgh">
<meta name="citation_author_institution" content="Laboratory for Pharmaceutical Biotechnology, Ghent University, Ghent, Belgium">
<meta name="citation_author_institution" content="NXTGNT, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Daisy Flamez">
<meta name="citation_author_institution" content="Biobix, Laboratory for Bioinformatics and Computational Genomics, Ghent University, Ghent, Belgium">
<meta name="citation_author_institution" content="NXTGNT, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Filip Pattyn">
<meta name="citation_author_institution" content="Center for Medical Genetics, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Bieke Scharlaken">
<meta name="citation_author_institution" content="NXTGNT, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Dieter Deforce">
<meta name="citation_author_institution" content="Laboratory for Pharmaceutical Biotechnology, Ghent University, Ghent, Belgium">
<meta name="citation_author_institution" content="NXTGNT, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Sofie Bekaert">
<meta name="citation_author_institution" content="NXTGNT, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Wim Van Criekinge">
<meta name="citation_author_institution" content="Biobix, Laboratory for Bioinformatics and Computational Genomics, Ghent University, Ghent, Belgium">
<meta name="citation_author_institution" content="NXTGNT, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Jo Vandesompele">
<meta name="citation_author_institution" content="NXTGNT, Ghent University, Ghent, Belgium">
<meta name="citation_author_institution" content="Center for Medical Genetics, Ghent University, Ghent, Belgium">
<meta name="citation_author" content="Guy Van Camp">
<meta name="citation_author_institution" content="Department of Medical Genetics, University of Antwerp, Antwerp, Belgium">
<meta name="citation_author" content="Paul Coucke">
<meta name="citation_author_institution" content="NXTGNT, Ghent University, Ghent, Belgium">
<meta name="citation_author_institution" content="Center for Medical Genetics, Ghent University, Ghent, Belgium">


<meta name="citation_issn" content="1755-8794">
<meta name="citation_title" content="Molecular diagnostics for congenital hearing loss including 15 deafness genes using a next generation sequencing platform">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="1">
<meta name="citation_date" content="2012-05-18">
<meta name="citation_firstpage" content="17">
<meta name="citation_doi" content="10.1186/1755-8794-5-17">
<meta name="citation_pdf_url" content="http://www.biomedcentral.com/content/pdf/1755-8794-5-17.pdf">
<meta name="citation_pmid" content="22607986">
<link rel="alternate" type="application/pdf" title="PDF" href="http://www.biomedcentral.com/content/pdf/1755-8794-5-17.pdf">
<meta name="title" content="Molecular diagnostics for congenital hearing loss including 15 deafness genes using a next generation sequencing platform">
<meta name="description" content="Hereditary hearing loss (HL) can originate from mutations in one of many genes involved in the complex process of hearing. Identification of the genetic defects in patients is currently labor intensive and expensive. While screening with Sanger sequencing for GJB2 mutations is common, this is not the case for the other known deafness genes (> 60). Next generation sequencing technology (NGS) has the potential to be much more cost efficient. Published methods mainly use hybridization based target enrichment procedures that are time saving and efficient, but lead to loss in sensitivity. In this study we used a semi-automated PCR amplification and NGS in order to combine high sensitivity, speed and cost efficiency.">

<meta name="dc.title" content="Molecular diagnostics for congenital hearing loss including 15 deafness genes using a next generation sequencing platform">
<meta name="dc.creator" content="De Keulenaer, Sarah">
<meta name="dc.creator" content="Hellemans, Jan">
<meta name="dc.creator" content="Lefever, Steve">
<meta name="dc.creator" content="Renard, Jean-Pierre">
<meta name="dc.creator" content="De Schrijver, Joachim">
<meta name="dc.creator" content="Van de Voorde, Hendrik">
<meta name="dc.creator" content="Tabatabaiefar, Mohammad A">
<meta name="dc.creator" content="Van Nieuwerburgh, Filip">
<meta name="dc.creator" content="Flamez, Daisy">
<meta name="dc.creator" content="Pattyn, Filip">
<meta name="dc.creator" content="Scharlaken, Bieke">
<meta name="dc.creator" content="Deforce, Dieter">
<meta name="dc.creator" content="Bekaert, Sofie">
<meta name="dc.creator" content="Van Criekinge, Wim">
<meta name="dc.creator" content="Vandesompele, Jo">
<meta name="dc.creator" content="Van Camp, Guy">
<meta name="dc.creator" content="Coucke, Paul">
<meta name="dc.description" content="Hereditary hearing loss (HL) can originate from mutations in one of many genes involved in the complex process of hearing. Identification of the genetic defects in patients is currently labor intensive and expensive. While screening with Sanger sequencing for GJB2 mutations is common, this is not the case for the other known deafness genes (> 60). Next generation sequencing technology (NGS) has the potential to be much more cost efficient. Published methods mainly use hybridization based target enrichment procedures that are time saving and efficient, but lead to loss in sensitivity. In this study we used a semi-automated PCR amplification and NGS in order to combine high sensitivity, speed and cost efficiency.">
<meta name="dc.source" content="BMC Medical Genomics 2012 5:17">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="BioMed Central Ltd">
<meta name="dc.date" content="2012-05-18">
<meta name="dc.type" content="Research article">
<meta name="dc.identifier" content="10.1186/1755-8794-5-17">
<meta name="dc.identifier" content="info:pmid/22607986">
<meta name="dc.language" content="en">
<meta name="dc.copyright" content="2012 De Keulenaer et al.; licensee BioMed Central Ltd.">
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0/">
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

<meta name="prism.issn" content="1755-8794">
<meta name="prism.publicationName" content="BMC Medical Genomics">
<meta name="prism.publicationDate" content="2012-05-18">
<meta name="prism.volume" content="5">
<meta name="prism.number" content="1">
<meta name="prism.section" content="Research article">
<meta name="prism.startingPage" content="17">
<meta name="prism.copyright" content="2012 De Keulenaer et al.; licensee BioMed Central Ltd.">
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com">


        <meta name="citation_abstract_html_url" content="http://www.biomedcentral.com/1755-8794/5/17/abstract">
            <meta name="citation_fulltext_html_url" content="http://www.biomedcentral.com/1755-8794/5/17">
            <link rel="alternate" type="text/xml" title="XML version" href="http://www.biomedcentral.com/content/download/xml/1755-8794-5-17.xml">
            <link rel="image_src" href="http://www.biomedcentral.com/content/figures/1755-8794-5-17-toc.gif">
    
    
    
    
                    
    
<script type="text/javascript">

// must run before css
document.documentElement.id = "js";
</script>

<script src="jquery.min.js"></script>

						
 
    
    
                                                                                                                                                                                                            
                                                                                                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    

						


	                
    
                
    



    

			                
    	

    

    


                
    


<!--[if IE]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie-0.css"/>
    <![endif]-->

<!--[if IE 7]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie7-0.css"/>
    <![endif]-->
			
<script type="text/javascript">

    //configuration
    OAS_url = 'http://oas.biomedcentral.com/RealMedia/ads/';

	// control of advert's position on page [AMC]
    
    	
	
    OAS_sitepage = "biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17";
    	
		OAS_listpos = 'Top,x96,Bottom,Right3';
	
	
			
		OAS_query = '';
		
	

    OAS_target = '_top';
    //end of configuration
    OAS_version = 10;
    OAS_rn = '001234567890'; OAS_rns = '1234567890';
    OAS_rn = new String (Math.random()); OAS_rns = OAS_rn.substring (2, 11);
    function OAS_NORMAL(pos) {
		document.write('<a href="' + OAS_url + 'click_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" target="' + OAS_target + '">');
		document.write('<img src="' + OAS_url + 'adstream_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" border="0"/></a>');
    }
    
	function refererTerms(referer) {

	if(referer.match("q=") != null) {
		referer = referer.split("q=");
		referer = referer[1].split("&");
		referer = referer[0];
	} else {
		referer = referer;
	}

		if(referer.match("%20") != null && referer.match("%20").length >= 1) {
			referer =  referer.replace(/%20/gi,"+");
			return referer;
		} else { 
			return referer; 
		}
	}
 </script>

 
 
<link rel="stylesheet" type="text/css" href="17.css" media="all">
</head>
<body class=" theme    "><div id="oas-campaign" style="display: none;">biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17</div>
 <div id="oas-positions" style="display: none;">Bottom,Top</div> 


 <script type="text/javascript">

	// Set the version of JavaScript to 10 for a Browser 'Mozilla/3' on webTV 
    OAS_version = 11;
    if ((navigator.userAgent.indexOf('Mozilla/3') != -1) || (navigator.userAgent.indexOf('Mozilla/4.0 WebTV') != -1)) {
     OAS_version = 10; }
    if (OAS_version >= 11) {
    document.write('<scr' + 'ipt type="text/javascript" src="' + OAS_url + 'adstream_mjx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '?' + OAS_query + '"><\/script>'); }
</script><script type="text/javascript" src="1532137682@Top,x96,Bottom,Right3"></script>
	
<script type="text/javascript">

    document.write('');
	
    function OAS_AD(pos) {
        if (OAS_version >= 11)
         OAS_RICH(pos);
        else
         OAS_NORMAL(pos);
    }
 </script>
	





<script type="text/javascript">

    jsonLoaders = function(){
        var _loaders = [];
        return {
            register: function(config){
                config.prefix = config.prefix || '/webapi/1.0/blogs/';
                config.feedLength = config.feedLength || 3;

                _loaders.push(config);
            },
            list: function(){return _loaders}
        };
    }();

    function totext() {
	    document.write('<a href="javascript:returnToText();">Return to text</a>');
    }

    function retrieve_doi(){
        var metas = document.getElementsByTagName('meta');
        var i;
        for (i = 0; i < metas.length; i++)
            if (metas[i].getAttribute('name') == "citation_doi")
                break;
        var citation_doi = metas[i].getAttribute('content');
        return citation_doi;
    }

</script>




        <link title="Latest articles" rel="alternate" href="http://www.biomedcentral.com/bmcmedgenomics/latest/rss" type="application/rss+xml" exposetoheader="true">
    <link title="Most viewed" rel="alternate" href="http://www.biomedcentral.com/bmcmedgenomics/mostviewed/rss/" type="application/rss+xml" exposetoheader="true">
    <link title="Latest comments" rel="alternate" href="http://www.biomedcentral.com/bmcmedgenomics/latestcomments/rss" type="application/rss+xml" exposetoheader="true">
    


	<script type="text/javascript">

	var _gaq = _gaq || [];
	_gaq.push(['_setAccount', 'UA-72326-1']);
	_gaq.push(['_gat._anonymizeIp']);
	_gaq.push(['_setLocalRemoteServerMode']);
	
	/*for cross domain tracking*/
	//_gaq.push(['_setAllowLinker', true]);
    //_gaq.push(['_setDomainName', 'biomedcentral.com/bmcmedgenomics']); //this journal's domain
	/**/
	
		_gaq.push(['_trackPageview']); 

	(function() {
		var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
		ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
		var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);

		//var iframe = document.getElementById('casesdb-search-widget');
		//if(iframe !== null) {
		//	var pageTracker = _gat._getTrackerByName();
		//	iframe.src = pageTracker._getLinkerUrl('http://www.casesdatabase.com/');
		//}
	})();
	</script>

	
        




                                                



 
    
    
                                                                                                                                                                                                            
                                                                                                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<script type="text/javascript">


function createCookie(name,value,days,path) {
	if (days) {
		var date = new Date();
		date.setTime(date.getTime()+(days*24*60*60*1000));
		var expires = "; expires="+date.toGMTString();
	}
	else{ var expires = "";}
	if (path!=null)
	{
	    document.cookie = name+"="+value+expires+"; path=/";
	}
	else
	{
	    document.cookie = name+"="+value+expires+";";
	}
}
function createCookie1(name,value,days) {
	if (days) {
		var date = new Date();
		date.setTime(date.getTime()+(days*24*60*60*1000));
		var expires = "; expires="+date.toGMTString();
	}
	else{ var expires = "";}
		    document.cookie = name+"="+value+expires;
	}

function eraseCookie(name) {
    // path describes the location of the cookie
    var path= "/";
	createCookie("database_statistics","",-1,path);
	// null value as a parameter tells that cookie existing in current application context like cookie in  http://biomedcentral.com.local/about/tandc
	createCookie("database_statistics_request_id","",-1,null);
	hidediv();
}
function hidediv() 
{ 
	if (document.getElementById) 
	{
	    document.getElementById('dbstats_main').style.visibility = 'hidden';
	}
}

function getCookieExists(c_name)
{
    var i,x,y,ARRcookies=document.cookie.split(";");
    for (i=0;i<ARRcookies.length;i++)
    {
          x=ARRcookies[i].substr(0,ARRcookies[i].indexOf("="));
          y=ARRcookies[i].substr(ARRcookies[i].indexOf("=")+1);
          x=x.replace(/^\s+|\s+$/g,"");
          if (x==c_name)
            {
                return unescape(y);
            }
     }
}

var request_Id;
var search_cookie = "database_statistics_request_id" + "="
var more_dbstats_link;
//readingDBStatscookie() will be called from JQuery function jQuery(document).ready(function() in behaviour.js which is called on "onload()" event
// in order to read the data from the lates cookie

function readingDBStatsCookie()
{
 if ( getCookieExists("database_statistics")!=null)
   {	
	if ( getCookieExists("database_statistics_request_id") != null)
   {
        if ( getCookieExists("database_statistics_request_id") != null)
        {
            if (document.cookie.length > 0)
            {
                // Search for a cookie.
                offset = document.cookie.indexOf(search_cookie)
                if (offset != -1)
                {
                        offset += search_cookie.length
                        // set index of beginning of value
                        end = document.cookie.indexOf(";",offset)
                        if (end == -1)
                        {
                            end = document.cookie.length
                        }
                       request_Id=document.cookie.substring(offset, end);
                       more_dbstats_link = "http://www.biomedcentral.com:80/newjournal/dbstats?content_type=html&request_id="+request_Id;
			  
			  createCookie("database_statistics_request_id","",-1);
				
                       var link = document.createElement('a');
                       link.setAttribute('href', more_dbstats_link);
                       link.innerHTML="<b>More DBStats</b>";
                       link.title="More dbstats";
                       //var text = "<a href=\""+more_dbstats_link+"\" >More DB Stats</a>";
                    document.getElementById('more_dbstats_link').appendChild(link);
                }
            }
        }
	}
 }
}
  if (getCookieExists("database_statistics")!=null)
     {
           document.write( "<div id =\"dbstats_main\" style=\"background-color:white\">"   );
             document.write("<div id =\"dbstats\" style=\"background-color:lightgray;margin-left:300px;margin-right:300px;margin-top:-10px\">");
             document.write("
");
                    document.write("<b>DB Stats</b>");
                    document.write("- consistent gets:");
                    document.write("<span id = \"consistent_gets\">NA</span> ");
                    document.write(", physical reads:");
                    document.write("<span id = \"physical_reads\">NA</span>");
                    document.write(", bytes sent via SQL*Net to client: ");
                    document.write("<span id = \"SQL_NET\">NA</span>");
                    document.write("&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp");
                    document.write("<span id=\"more_dbstats_link\">");
                    document.write("</span>	");
                    document.write("<span id=\"close_dbstats_link\">");
                    document.write("[<a href='javascript:eraseCookie()' >x</a>]");
                    document.write("</span>	");
             document.write("</div>");
           document.write("</div> ");
     }
 </script>
<div id="branding" role="banner">
    		        			
		<dl class="google-ad wide ">
	<dt class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Top' begin -->
		<script type="text/javascript">

		OAS_AD('Top');

		</script>

		



<script type="text/javascript">

//<![CDATA[
var filePath                     = "http://imageceu1.247realmedia.com/RealMedia/ads/Creatives/biomed/Paid-AP-LabArchives-ALL-LB-Sep_2013/labarchives2.swf/1377766547";
var TFSMFlash_PRETAG             = "";
var TFSMFlash_POSTTAG            = "";
var TFSMFlash_VERSION            = "9";
var TFSMFlash_WMODE              = "window";

			var TFSMFlash_OASCLICK           = "http://oas.biomedcentral.com/5c/biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17/L40/864717047/Top/biomed/Paid-AP-LabArchives-ALL-LB-Sep_2013/Paid-AP-LabArchives-ALL-LB-Sep_2013.html/534c2f6351564a6e7436344141655771";
	


var TFSMFlash_SWFCLICKVARIABLE   = "?clickTag="+TFSMFlash_OASCLICK + "";
var TFSMFlash_SWFFILE            = filePath + TFSMFlash_SWFCLICKVARIABLE;
var TFSMFlash_FSCOMMAND          = "";
var TFSMFlash_IMAGEALTERNATE     = "http://imageceu1.247realmedia.com/RealMedia/ads/Creatives/biomed/Paid-AP-LabArchives-ALL-LB-Sep_2013/labarchives2.gif/1377766547";


var TFSMFlash_OASALTTEXT         = "";
var TFSMFlash_OASTARGET          = "_blank";
var TFSMFlash_OASPROTOCOL        = "http://";
var TFSMFlash_OASDIM             = "width='728px' height='90px'";
var TFSMFlash_OASADID            = "OAS_RMF_Top_FLASH";
document.write("<"+"scr"+"ipt type=\"text/javascript\" src=\"http://imageceu1.247realmedia.com/RealMedia/ads/Creatives/biomed/Paid-AP-LabArchives-ALL-LB-Sep_2013/TFSMFlashWrapper301.js/1377766547\"><\/scr"+"ipt"+">");
//]]>
</script><script type="text/javascript" src="1377766547"></script><a href="http://oas.biomedcentral.com/5c/biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17/L40/864717047/Top/biomed/Paid-AP-LabArchives-ALL-LB-Sep_2013/Paid-AP-LabArchives-ALL-LB-Sep_2013.html/534c2f6351564a6e7436344141655771" target="_top"><img src="1377766547_001.dat" alt="" border="0" height="90px" width="728px"></a>
<noscript><a href="http://oas.biomedcentral.com/5c/biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17/L40/864717047/Top/biomed/Paid-AP-LabArchives-ALL-LB-Sep_2013/Paid-AP-LabArchives-ALL-LB-Sep_2013.html/534c2f6351564a6e7436344141655771" target="_blank"><img src="http://imageceu1.247realmedia.com/RealMedia/ads/Creatives/biomed/Paid-AP-LabArchives-ALL-LB-Sep_2013/labarchives2.gif/1377766547" width="728" height="90" alt="" style="border:0px black solid;" /></a></noscript>



		<!-- OAS AD 'Top' end -->
		</dd>

		</dl>

		<noscript>

			<dl class="google-ad wide noscript">  
    <dt class="hide"></dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17/16978656959@Top?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17/11231819928@Top?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
					        <div class="sup-panel-outer">
        <div class="sup-panel-inner">
           
       

            <ul id="login" class="login">
            <li id="loginLink"><a href="http://www.biomedcentral.com/bmcmedgenomics/logon">Log on</a></li>
    </ul>
            <ul class="nav-sup">
    
        <li id="BMC" class="current">
            <a href="http://www.biomedcentral.com/"><span class="img-title">&nbsp;</span>
            <span class="text-title">BioMed Central</span></a>
        </li>
		
        		<li><a href="http://www.biomedcentral.com/journals"><span>Journals</span></a></li>
		
								<li>
				<a href="http://www.biomedcentral.com/gateways"><span>Gateways</span></a>
			</li>
					    	
</ul>
        </div>
    </div>
    <div class="branding-inner">
          <div class="logo">
        <a href="http://www.biomedcentral.com/bmcmedgenomics/"><img alt="BMC Medical Genomics" title="BMC Medical Genomics Logo" src="logo.gif"></a>     
      </div>    
        
                    



            <div id="impact-factor" class="official">
    <img src="official.gif" alt="official impact factor" title="3.47">
    <span id="impact-factor-value">3.47</span>
</div>
    
                            

<div class="module search gray small  " role="search">
    <div class="module-inner padded">
        <form method="get" class="searchForm" action="http://www.biomedcentral.com/quickSearchRedirectURL" onsubmit="window.location.href=this['url'].value + '?terms=' + encodeURIComponent(this['terms'].value); return false;">
            <fieldset class="search">
			                <span>

                    Search <select name="url">
                                <option value="http://www.biomedcentral.com/bmcmedgenomics/search/results" selected="">BMC Medical Genomics</option>
                                <option value="http://www.biomedcentral.com/search/results">BioMed Central</option>
                           </select> for
                </span>
                            <input style="width: 143px;" id="searchTerms" name="terms" alt="Search terms" class="text " type="text">
                 <button class="w37" value="Go" type="submit">Go</button>
            </fieldset>
        </form>
    </div>
	<a class="advanced-search" href="http://www.biomedcentral.com/bmcmedgenomics/search">Advanced search</a>
</div> 

                        <div id="mobile-nav-list">

	<ul class="primary-nav" role="navigation">
		<li><a href="http://www.biomedcentral.com/bmcmedgenomics/"><span id="home-tab">Home</span></a></li>
														<li class="current"><a href="http://www.biomedcentral.com/bmcmedgenomics/content"><span id="articles-tab">Articles</span></a></li>
										
	

	
            <li><a href="http://www.biomedcentral.com/bmcmedgenomics/authors/instructions"><span id="authors-tab">Authors</span></a></li>
        <li><a href="http://www.biomedcentral.com/bmcmedgenomics/about/reviewers"><span id="reviewers-tab">Reviewers</span></a></li>
    		<li><a href="http://www.biomedcentral.com/bmcmedgenomics/about"><span id="about-tab">About this journal</span></a></li>
	<li><a href="http://www.biomedcentral.com/bmcmedgenomics/my"><span id="my-tab">My BMC Medical Genomics</span></a></li>
		
    		</ul>
</div>
            </div>
</div>
<hr class="hide">
<div class="container">
	<div id="content" class="overflow-visible" role="main">
			    
	

<!--div class="branding-mobile">
		<h1 class="logo">
	    <a href="/bmcmedgenomics/">
		    <img alt="BMC Medical Genomics" title="BMC Medical Genomics Logo" src="/sites/2050/images/logo.gif" />
		</a>
		</h1>
	</div><br/-->



			
	



<div class="outline-wrapper block mobile-hidden" id="left-article-box">
<ul id="box-outline" class="box">
<ul id="outline" class="outline">
   <li>
      <a href="#">Top</a>
   </li>
   <li>
      <a href="#abs">Abstract</a>
   </li>
   <li>
      <a href="#sec1">Background</a>
   </li>
   <li>
      <a href="#sec2">Results</a>
   </li>
   <li>
      <a href="#sec3">Discussion</a>
   </li>
   <li>
      <a href="#sec4">Conclusions</a>
   </li>
   <li>
      <a href="#sec5">Methods</a>
   </li>
   <li>
      <a href="#sec6">Abbreviations</a>
   </li>
   <li>
      <a href="#sec7">Competing interests</a>
   </li>
   <li>
      <a href="#sec8">Authorsâ€™ contributions</a>
   </li>
   <li>
      <a href="#ack">Acknowledgements</a>
   </li>
   <li>
      <a href="#refs">References</a>
   </li>
   <li class="lowest">
      <a href="#bm">Pre-publication history</a>
   </li>
</ul>
</ul>
<dl id="box-outline1">
<dl class="google-ad" id="x96bannerpg"><dt style="display: block;" class="hide"><a class="skyscraper-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</a></dt><dd><div class="issue-information block" style="background: url('bg4.png') repeat-y scroll 0px 0px transparent; border: medium none; padding: 3px 0px 0px 3px; margin: 0px 0px 0px 10px;"><div class="container" style="background: url('pfeil1.gif') no-repeat scroll 115px 5px transparent;"><h5 style="margin: 2px 0px 4px 4px;">
<a href="http://delivery.pbgrd.com/openx/www/delivery/ck.php?oaparams=2__bannerid=107__zoneid=5__OXLCA=1__cb=20976aef24__doi=10.1186%2F1755-8794-5-17__iid=13825291534863510118617558794517__pos=1__oadest=http://www.antibodies-online.com/publication/10.1186%2F1755-8794-5-17/?a_aid=5c8abb57&amp;doi=10.1186%2F1755-8794-5-17&amp;utm_source=partner&amp;utm_medium=biomed&amp;utm_campaign=adscience&amp;utm_content=headline" target="_top">
<span style="">Related Products<br>and Services</span>
</a>
</h5><div style="border-top: 1px solid rgb(168, 168, 168); width: 131px; color: black; font-size: 95%; font-weight: bold; padding: 3px 0px 0px 3px; margin: 0px 0px 4px; line-height: 10px;">By Gene Name</div><ul class="square normal" style="max-width: 140px; padding: 0px;"><li style="margin: 0px 0px 1px; white-space: nowrap;"><a href="http://delivery.pbgrd.com/openx/www/delivery/ck.php?oaparams=2__bannerid=107__zoneid=5__OXLCA=1__cb=20976aef24__doi=10.1186%2F1755-8794-5-17__iid=13825291534863510118617558794517__pos=2__oadest=http://www.antibodies-online.com/antigen/Otoferlin+%28OTOF%29/?doi=10.1186%2F1755-8794-5-17&amp;a_aid=5c8abb57&amp;utm_source=partner&amp;utm_medium=biomed&amp;utm_campaign=adscience&amp;utm_content=antigen&amp;utm_term=Otoferlin%2FOTOF" target="_top" style="padding: 0px 0px 0px 4px; margin-bottom: 2px; text-decoration: none; border-bottom: 1px dotted rgb(219, 219, 219); line-height: 12px; width: 131px;">Otoferlin<span style="margin: 0px 2px 0px 3px;">/</span>OTOF</a></li><li style="margin: 0px 0px 1px; white-space: nowrap;"><a href="http://delivery.pbgrd.com/openx/www/delivery/ck.php?oaparams=2__bannerid=107__zoneid=5__OXLCA=1__cb=20976aef24__doi=10.1186%2F1755-8794-5-17__iid=13825291534863510118617558794517__pos=3__oadest=http://www.antibodies-online.com/antigen/Cadherin-Like+23+%28CDH23%29/?doi=10.1186%2F1755-8794-5-17&amp;a_aid=5c8abb57&amp;utm_source=partner&amp;utm_medium=biomed&amp;utm_campaign=adscience&amp;utm_content=antigen&amp;utm_term=Cadherin-Like+23" target="_top" style="padding: 0px 0px 0px 4px; margin-bottom: 2px; text-decoration: none; border-bottom: 1px dotted rgb(219, 219, 219); line-height: 12px; width: 131px;">Cadherin-Like 23</a></li></ul><div id="beacon_c7fea0323f4" style="position: absolute; left: 0px; top: 0px;">
<img style="width: 1px; height: 1px;" alt="" src="lg1.php" height="1" width="1">
</div><div style="border-top: 1px solid rgb(168, 168, 168); width: 131px; color: black; font-size: 95%; font-weight: bold; padding: 3px 0px 0px 3px; margin: 0px 0px 4px; line-height: 10px;">Supported by</div><table><tbody><tr><td style="padding: 0px 3px;"><a href="http://delivery.pbgrd.com/openx/www/delivery/ck.php?oaparams=2__bannerid=107__zoneid=5__OXLCA=1__cb=20976aef24__doi=10.1186%2F1755-8794-5-17__iid=13825291534863510118617558794517__pos=4__oadest=http://www.antibodies-online.com/publication/10.1186%2F1755-8794-5-17/?a_aid=5c8abb57&amp;doi=10.1186%2F1755-8794-5-17&amp;utm_source=partner&amp;utm_medium=biomed&amp;utm_campaign=adscience&amp;utm_content=logo" target="_top"><img src="ABO_16.png" title="antibodies-online"></a></td></tr></tbody></table></div></div></dd></dl></dl>
</div>

<div class="mobile-hidden" id="right-panel">
 	
<div id="mobile-sidebar">
	<div id="mobile-sidebar-tab">
		<span>
			<a href="#" id="mobile-sidebar-tab-link"><img src="sidebar-hidden.png"><img style="display: none;" src="sidebar-shown.png"></a>
		</span>
	</div>	<div id="mobile-sidebar-list">

<div id="article-navigation-bar">
    <div class="issue-information block">
        <h5><a href="http://www.biomedcentral.com/bmcmedgenomics"><strong>BMC Medical Genomics</strong></a></h5>
 		        <ul class="square normal">
            <li><a href="http://www.biomedcentral.com/bmcmedgenomics/content/5">Volume 5</a></li>
				
                    </ul>
    </div>

    <div class="article-information block" id="article-info">
        


<div id="viewing-options-links">
	<h5>Viewing options</h5>
	<ul class="square normal">
        
            <li>
                             <a href="http://www.biomedcentral.com/1755-8794/5/17/abstract">Abstract</a>
                        </li>
        				<li>
							<strong>Full text</strong>
					</li>
				
		
		
		<li class="pdfFileSize">			<a onclick="_gaq.push(['_trackEvent', 'PDF download', 'Article Sidebar', '/1755-8794/5/17/', 1, true]);" href="http://www.biomedcentral.com/content/pdf/1755-8794-5-17.pdf">PDF</a>
			<span>(473KB)</span>		</li>
		
		<li>
                    </li>
		
			</ul>
</div>
		
<div id="associated-material-links">
    <h5>Associated material</h5>
    <ul class="square normal">
                        <li>
            <a href="http://www.biomedcentral.com/pubmed/22607986">PubMed record</a>
        </li>
                            <li><a href="http://www.biomedcentral.com/1755-8794/5/17/about">Article metrics</a></li>
                <li>
                        <a href="http://www.biomedcentral.com/1755-8794/5/17/comments">Readers' comments </a>
                    </li>
        <li>
                            <a href="http://www.biomedcentral.com/1755-8794/5/17/prepub">Pre-publication history</a>                    </li>
    </ul>
</div>
		




<div id="related-literature-links">
    <h5>Related literature</h5>
    <ul class="normal">
                    <li><a href="http://www.biomedcentral.com/1755-8794/5/17/about#citations">Cited by</a></li>
                <li id="google-blog-search"><a href="http://www.google.com/search?tbm=blg&amp;hl=en&amp;source=hp&amp;biw=1280&amp;bih=899&amp;q=%22Molecular+diagnostics+for+congenital+hearing+loss%0D%0Aincluding+15+deafness+genes+using+a+next+generation%0D%0Asequencing+platform%22&amp;btnG=Search">Google blog search</a></li>
        <h6>Other articles by authors</h6>
        <li>
            <a class="collapser"><i>&nbsp;</i>on Google Scholar</a>
<ul id="authg" class="hidebeforeload">                         <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22S+De%20Keulenaer%22">De Keulenaer S</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22J+Hellemans%22">Hellemans J</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22S+Lefever%22">Lefever S</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22JP+Renard%22">Renard JP</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22J+De%20Schrijver%22">De Schrijver J</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22H+Van%20de%20Voorde%22">Van de Voorde H</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22MA+Tabatabaiefar%22">Tabatabaiefar MA</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22F+Van%20Nieuwerburgh%22">Van Nieuwerburgh F</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22D+Flamez%22">Flamez D</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22F+Pattyn%22">Pattyn F</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22B+Scharlaken%22">Scharlaken B</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22D+Deforce%22">Deforce D</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22S+Bekaert%22">Bekaert S</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22W+Van%20Criekinge%22">Van Criekinge W</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22J+Vandesompele%22">Vandesompele J</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22G+Van%20Camp%22">Van Camp G</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22P+Coucke%22">Coucke P</a>
            </li>
            </ul>          </li>
                <li>
            		<a class="collapser"><i>&nbsp;</i>on PubMed</a>
	<ul id="authpm" class="hidebeforeload"> 									<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=De_Keulenaer_S%20[Author]">De Keulenaer S</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Hellemans_J%20[Author]">Hellemans J</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Lefever_S%20[Author]">Lefever S</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Renard_JP%20[Author]">Renard JP</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=De_Schrijver_J%20[Author]">De Schrijver J</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Van_de_Voorde_H%20[Author]">Van de Voorde H</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Tabatabaiefar_MA%20[Author]">Tabatabaiefar MA</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Van_Nieuwerburgh_F%20[Author]">Van Nieuwerburgh F</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Flamez_D%20[Author]">Flamez D</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Pattyn_F%20[Author]">Pattyn F</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Scharlaken_B%20[Author]">Scharlaken B</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Deforce_D%20[Author]">Deforce D</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Bekaert_S%20[Author]">Bekaert S</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Van_Criekinge_W%20[Author]">Van Criekinge W</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Vandesompele_J%20[Author]">Vandesompele J</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Van_Camp_G%20[Author]">Van Camp G</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Coucke_P%20[Author]">Coucke P</a>
				</li>
						</ul> 	        </li>
    </ul>     <h6>Related articles/pages</h6>
    <ul class="square normal">
        <li><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1755-8794/5/17">on Google</a></li>
        <li><a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1755-8794/5/17">on Google Scholar</a></li>
                    <li><a href="http://www.biomedcentral.com/pubmed/related/22607986">on PubMed</a></li>
            </ul>

</div>		
        
    
    
        

    	






<h5>Tools</h5>
	<ul class="square normal">
									<li>
						<a href="http://www.biomedcentral.com/1755-8794/5/17/citation">Download references</a>
					</li>
				<li><a href="http://www.biomedcentral.com/content/download/xml/1755-8794-5-17.xml">Download XML</a></li>
				<li>
						<a href="http://www.biomedcentral.com/1755-8794/5/17/email">Email to a friend</a>
					</li>
		<li><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186/1755-8794-5-17&amp;volume=5&amp;issue=1&amp;title=Molecular+diagnostics+for+congenital+hearing+loss+including+15+deafness+genes+using+a+next+generation+sequencing+platform&amp;author_name=Sarah%20De%20Keulenaer&amp;start_page=1&amp;end_page=8">Order reprints</a></li>
		<li>
						<a href="http://www.biomedcentral.com/1755-8794/5/17/postcomment">Post a comment</a>
					</li>			
		<ul id="download-to-links">
			<a class="downloadto_title btn" href="#"><span class="icon">&nbsp;</span>Download to ...</a>
			
			<ul id="downloadto" class="iconlist hidebeforeload" style="display: none;">
				<li>
					<a href="http://redirect.papersapp.com/redirect?url=http%3A%2F%2Fwww.biomedcentral.com%2F1755-8794%2F5%2F17" onclick="largepopup(this.href,'papers',800,600);return false"><span class="share-icons papers"></span>Papers</a>
				</li>
				<li>
					<a href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1755-8794/5/17" onclick="largepopup(this.href,'mendeley',800,600);return false"><span class="share-icons mendeley"></span>Mendeley</a>
				</li>
			</ul>
		</ul>
		
		<ul id="mobile-downloadto" class="iconlist">
		
		<h6>Download to ...</h6>
			<li>
				<a href="http://redirect.papersapp.com/redirect?url=http%3A%2F%2Fwww.biomedcentral.com%2F1755-8794%2F5%2F17" onclick="largepopup(this.href,'papers',800,600);return false"><span class="share-icons papers"></span>Papers</a>
			</li>
			
			<li>
				<a href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1755-8794/5/17" onclick="largepopup(this.href,'mendeley',800,600);return false"><span class="share-icons mendeley"></span>Mendeley</a>
			</li>
		</ul>

		
	</ul>
		




<h5>Share this article</h5>
<ul id="social-networking-links">
	<li>
	<div fb-xfbml-state="parsed" class="fb-like fb_edge_widget_with_comment fb_iframe_widget" data-href="http://www.biomedcentral.com/1755-8794/5/17" data-send="false" data-layout="button_count" data-width="100" data-show-faces="false" data-action="recommend"><span><iframe src="17_1.html" class="fb_ltr" title="Like this content on Facebook." style="border: medium none; overflow: hidden; height: 21px; width: 130px;" name="f3efb9d673e84fe" id="f4e003ad2966e" scrolling="no"></iframe></span></div>
	</li>
	<li>
		<a href="http://twitter.com/share" data-url="http://www.biomedcentral.com/1755-8794/5/17" data-counturl="http://www.biomedcentral.com/1755-8794/5/17" data-hashtags="bmcmedicalgenomics
" data-text="Molecular diagnostics for congenital hearing loss
including 15 deafness genes using a next generation
sequencing platform" class="twitter-share-button">Tweet</a>
	</li>
	<li>
			<div id="___plusone_0" style="position: absolute; width: 450px; left: -10000px;"><iframe data-gapiattached="true" src="17_2.html" name="I0_1382529157918" id="I0_1382529157918" vspace="0" tabindex="0" style="position: absolute; top: -10000px; width: 450px; margin: 0px; border-style: none;" marginwidth="0" marginheight="0" hspace="0" frameborder="0" scrolling="no" width="100%"></iframe></div><g:plusone data-gapistub="true" data-onload="true" data-gapiscan="true" size="medium" width="100"></g:plusone>
	</li>

		<a class="sharethis_title btn" href="#"><span class="icon">&nbsp;</span>More options...</a>
	<ul style="display: none;" class="iconlist hidebeforeload" id="sharethis">
			<li>
				<a onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false" href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1755-8794/5/17"><span class="share-icons citeulike"></span>Citeulike</a>
			</li>
				<li>
			<a onclick="largepopup(this.href,'linkedin',800,600);return false" href="http://www.linkedin.com/shareArticle?mini=true&amp;url=%26%2335%3Bbmcmedicalgenomics%0D%0AMolecular%20diagnostics%20for%20congenital%20hearing%20loss%0D%0Aincluding%2015%20deafness%20genes%20using%20a%20next%20generation%0D%0Asequencing%20platform%20http%3A%2F%2Fwww.biomedcentral.com%2F1755-8794%2F5%2F17&amp;title=Molecular%20diagnostics%20for%20congenital%20hearing%20lossincluding%2015%20deafness%20genes%20using%20a%20next%20generationsequencing%20platform"><span class="share-icons linkedin"></span>LinkedIn</a>  
	</li>
			<li>
				<a onclick="largepopup(this.href,'Del.icio.us',800,600);return false" href="http://del.icio.us/post?url=http://www.biomedcentral.com/1755-8794/5/17&amp;title=Molecular%20diagnostics%20for%20congenital%20hearing%20lossincluding%2015%20deafness%20genes%20using%20a%20next%20generationsequencing%20platform"><span class="share-icons delicious"></span>Del.icio.us</a>
	</li>
			<li>
				<a href="http://www.biomedcentral.com/1755-8794/5/17/email"><span class="share-icons email"></span>Email</a>
	</li>
			<li>
				<a onclick="largepopup(this.href,'facebook',800,600);return false" href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1755-8794/5/17"><span class="share-icons facebook"></span>Facebook</a>
	</li>
	<li class="gp">
			<div class="googlehider mobile-hide"> 
				<div id="___plusone_1" style="position: absolute; width: 450px; left: -10000px;"><iframe data-gapiattached="true" src="17_3.html" name="I1_1382529157921" id="I1_1382529157921" vspace="0" tabindex="0" style="position: absolute; top: -10000px; width: 450px; margin: 0px; border-style: none;" marginwidth="0" marginheight="0" hspace="0" frameborder="0" scrolling="no" width="100%"></iframe></div><div data-gapistub="true" data-onload="true" data-gapiscan="true" class="g-plusone" data-annotation="none"></div>
			</div>
				<span class="share-icons googleplus mobile-hide"></span>
				<div class="googlehider2 mobile-hide"> 
				<div id="___plusone_2" style="position: absolute; width: 450px; left: -10000px;"><iframe data-gapiattached="true" src="17_4.html" name="I2_1382529157923" id="I2_1382529157923" vspace="0" tabindex="0" style="position: absolute; top: -10000px; width: 450px; margin: 0px; border-style: none;" marginwidth="0" marginheight="0" hspace="0" frameborder="0" scrolling="no" width="100%"></iframe></div><div data-gapistub="true" data-onload="true" data-gapiscan="true" class="g-plusone mobile-hide" data-annotation="none"></div>
			</div>
				<span class="share-icons googleplus"></span>
				<span class="gp-link"><a href="#">Google+</a></span>
	</li>
			<li>
				<a onclick="largepopup(this.href,'mendeley',800,600);return false" href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1755-8794/5/17"><span class="share-icons mendeley"></span>Mendeley</a>
			</li>
			<li>
			<a onclick="largepopup(this.href,'twitter',800,600);return false" href="http://twitter.com/?status=Molecular%20diagnostics%20for%20congenital%20hearing%20loss%0D%0Aincluding%2015%20deafness%20genes%20using%20a%20next%20generation%0D%0Asequencing%20platform%20http%3A%2F%2Fwww.biomedcentral.com%2F1755-8794%2F5%2F17+%23bmcmedicalgenomics"><span class="share-icons twitter"></span>Twitter</a>  
			</li>
			
		</ul>
</ul>

    </div>

    
				
    			
		














		                                    		                                    		
	

	    
 
    
    
                                                                                                                                                                                                            
                                                                                                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<div id="signup-to-etoc" class="button-collection-margins block mobile-border-bottom">	
					<h3>Email updates</h3>
			<p id="receive_issue_alerts_journal_updates_text">Keep up to date with the latest news and content from BMC Medical Genomics and BioMed Central.</p>
			
				

<form id="articleSignupForUpdates" method="post" action="http://www.biomedcentral.com/bmcmedgenomics/signuptoupdates.html">
	<div id="message-box"></div>
    <fieldset class="block-form">
        <input name="emailAddress" onblur="javascript: if (this.value == '') { this.value = 'email address'; }" onfocus="javascript: if (this.value == 'email address') { this.value = ''; }" value="email address" class="text " id="email" type="text">
        <input value="/bmcmedgenomics" name="returnUrl" id="returnUrl" type="hidden">
        <input value="" name="listName" id="listName" type="hidden">
        <input value="BMC Medical Genomics" name="journalName" id="journalName" type="hidden">
        <button class="w74 right" type="submit" onmouseover="this.className='w74 hover right'" onmouseout="this.className='w74 right'" value="Sign up">Sign up</button>
    </fieldset>
</form>	
</div>
	
			<div id="advert-offset">
					
		<dl class="google-ad  ">
	<dt class="hide">
				
					<a class="skyscraper-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</a>
			</dt>
		<dd>

		<!-- OAS AD 'Right3' begin -->
		<script type="text/javascript">

		OAS_AD('Right3');

		</script><script type="text/javascript"><!--//<![CDATA[
	var m3_u = (location.protocol=='https:'?'https://delivery.pbgrd.com/openx/www/delivery/ajs.php':'http://delivery.pbgrd.com/openx/www/delivery/ajs.php');
	var m3_r = Math.floor(Math.random()*99999999999);

	var doi = null;

	var metas = document.getElementsByTagName('meta');
	var i;
	for (i = 0; i < metas.length; i++) {
		if (metas[i].getAttribute('name') == "citation_doi") {
			break;
		}
	}
	if (metas[i] != undefined) doi = metas[i].getAttribute('content');

	if (doi==null || doi=='') {
		doi = 'NODOI - ' + encodeURIComponent(document.URL);
	}
	else if (doi.substring(0,3)!='10.' || doi.indexOf('/')<0) {
		doi = 'BADDOI - ' + encodeURIComponent(document.URL);
	}

	if (!document.MAX_used) document.MAX_used = ',';
	document.write ("<scr"+"ipt type='text/javascript' src='"+m3_u);
	document.write ("?zoneid=43");

	document.write ("&doi=" + doi + "&campaignName=biomedright");

	document.write ('&amp;cb=' + m3_r);
	if (document.MAX_used != ',') document.write ("&amp;exclude=" + document.MAX_used);
	document.write (document.charset ? '&amp;charset='+document.charset : (document.characterSet ? '&amp;charset='+document.characterSet : ''));
	document.write ("&amp;loc=" + escape(window.location));
	if (document.referrer) document.write ("&amp;referer=" + escape(document.referrer));
	if (document.context) document.write ("&context=" + escape(document.context));
	if (document.mmm_fo) document.write ("&amp;mmm_fo=1");
	document.write ("'><\/scr"+"ipt>");
//]]>--></script><script type="text/javascript" src="ajs.php"></script><script type="text/javascript"><!--//<![CDATA[
if (navigator.appName == "Microsoft Internet Explorer") {
var windowheight = window.document.documentElement.clientHeight;
} else {
var windowheight = window.innerHeight;
}
var loaded = false;
var toscr = document.getElementById("advert-offset").offsetTop + document.getElementById("article-navigation-bar").offsetTop - windowheight;
function pgload() {
if (loaded == false) {
var scrpos = (window.pageYOffset || document.body.scrollTop || document.documentElement.scrollTop);
if (scrpos > toscr + 300) {
loaded = true;
var journal = "";
var metas = document.getElementsByTagName('meta');
var i;
for (i = 0; i < metas.length; i++) {
if (metas[i].getAttribute('name') == "citation_journal_title") {
break;
}
}
if (metas[i] != undefined) journal = encodeURIComponent(metas[i].getAttribute('content'));
var sctag = document.createElement('script');
var proto = (location.protocol=='https:') ? 'https' : 'http';
var startspace = -1 * toscr;
var scrollspace = windowheight;
sctag.src = proto + "://delivery.pbgrd.com/banner/banner-biomed-right-v1.php?doi=10.1186/1755-8794-5-17&clickurl=http://delivery.pbgrd.com/openx/www/delivery/ck.php?oaparams=2__bannerid=168__zoneid=43__cb=087c90ec5a__doi=10.1186%2F1755-8794-5-17__oadest=&zoneid=43&bannerid=168&pageurl=http%3A%2F%2Fwww.biomedcentral.com%2F1755-8794%2F5%2F17%2F" + "&journal=" + journal + "&startspace=" + startspace + "&scrollspace=" + scrollspace + "&verty=" + screen.height;
document.getElementById("advert-offset").appendChild(sctag);
}
}
}
if (window.addEventListener) {
window.addEventListener('scroll', function() { pgload(); }, false);
} else if (window.attachEvent) {
window.attachEvent('onscroll', function() { pgload(); });
}
if (window.addEventListener) {
window.addEventListener("load", function() {
window.onscroll = document.documentElement.onscroll = function() { setMenuOffset(); advertOffset(); };
});
} else if (window.attachEvent) {
window.attachEvent("onload", function() {
window.onscroll = document.documentElement.onscroll = function() { setMenuOffset(); advertOffset(); };
});
}
//]]>--></script><div id="beacon_087c90ec5a" style="position: absolute; left: 0px; top: 0px;"><img src="lg.php" alt="" style="width: 1px; height: 1px;" height="1" width="1"></div>


		<!-- OAS AD 'Right3' end -->
		</dd>

		</dl>

		<noscript>

			<dl class="google-ad  noscript">  
    <dt class="hide"><a href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17/12063010402@Right3?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17/19521939871@Right3?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
					</div>
		
</div>
		<div class="mobile-sidebar-gradient top"></div>
		<div class="mobile-sidebar-gradient bottom"></div>
	</div>

</div>








 </div>
<div class="rounded custom white article full-text">
<div class="wrap-inner content">
    <div class="padded-inner">            


    
 
    
    
                                                                                                                                                                                                            
                                                                                                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<div id="topmatter">
	


	   








				<a href="http://www.biomedcentral.com/about/access"><img alt="Open Access" src="openaccess-large.png" class="access mr15"></a>
					<span class="articletype">Research article</span>
<h1>Molecular diagnostics for congenital hearing loss including 15 deafness genes using a next generation sequencing platform</h1>

        
			<div class="multipleins">
		
	<p class="authors">
        																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																																										         <strong>Sarah De Keulenaer</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins1">1</a></sup>, <strong>Jan Hellemans</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins1">1</a></sup><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins2">2</a></sup>, <strong>Steve Lefever</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins2">2</a></sup>, <strong>Jean-Pierre Renard</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins3">3</a></sup>, <strong>Joachim De Schrijver</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins3">3</a></sup>, <strong>Hendrik Van de Voorde</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins2">2</a></sup>, <strong>Mohammad Amin Tabatabaiefar</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins5">5</a></sup><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins6">6</a></sup>, <strong>Filip Van Nieuwerburgh</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins1">1</a></sup><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins4">4</a></sup>, <strong>Daisy Flamez</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins1">1</a></sup><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins3">3</a></sup>, <strong>Filip Pattyn</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins2">2</a></sup>, <strong>Bieke Scharlaken</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins1">1</a></sup>, <strong>Dieter Deforce</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins1">1</a></sup><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins4">4</a></sup>, <strong>Sofie Bekaert</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins1">1</a></sup>, <strong>Wim Van Criekinge</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins1">1</a></sup><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins3">3</a></sup>, <strong>Jo Vandesompele</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins1">1</a></sup><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins2">2</a></sup>, <strong>Guy Van Camp</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins5">5</a></sup> and <strong>Paul Coucke</strong><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins1">1</a></sup><sup><a href="http://www.biomedcentral.com/1755-8794/5/17/#ins2">2</a></sup><sup>*</sup>	</p>

			<div id="affiliations">
		<div class="module gray  inner">
<div class="module-inner padded-inner">
		<ul>
			<li>                                                                                                                                                                                                                        
                            		<p class="authors">
							<span>*</span>
									 Corresponding author:										            Paul  Coucke <a href="mailto:Paul.Coucke@UGent.be">Paul.Coucke@UGent.be</a>    					</p>
      
</li>
			
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            		</ul>

		
		 <p class="options"><a class="affiliations-toggle" href="#"><i class="arrow active"></i>Author Affiliations</a></p>            <section> 
						<div class="collapsible-content">
							
				<div id="ins_container" style="display: block;">
										    						    <p>
						        <sup id="ins1">1</sup>                                NXTGNT, Ghent University, Ghent, Belgium
                            </p>
																    						    <p>
						        <sup id="ins2">2</sup>                                Center for Medical Genetics, Ghent University, Ghent, Belgium
                            </p>
																    						    <p>
						        <sup id="ins3">3</sup>                                Biobix, Laboratory for Bioinformatics and Computational Genomics, Ghent University, Ghent, Belgium
                            </p>
																    						    <p>
						        <sup id="ins4">4</sup>                                Laboratory for Pharmaceutical Biotechnology, Ghent University, Ghent, Belgium
                            </p>
																    						    <p>
						        <sup id="ins5">5</sup>                                Department of Medical Genetics, University of Antwerp, Antwerp, Belgium
                            </p>
																    						    <p>
						        <sup id="ins6">6</sup>                                Department of Medical Genetics, School of Medicine, Ahvaz Jundishapur University of Medical Sciences, Ahvaz, Iran
                            </p>
																
				</div>
										<p id="authoremails">            For all author emails, please <a href="http://www.biomedcentral.com/bmcmedgenomics/logon">log on</a>.
    </p>
						</div>
			</section>
				</div>
</div>
		</div>
	  </div>

            <section class="cit">
                        <div class="collapsible-content">
        <p><em>BMC Medical Genomics</em> 2012, <strong>5</strong>:17&nbsp;<span class="pseudotab">doi:10.1186/1755-8794-5-17</span></p>
    	
    
    
        		<br>
		<p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.biomedcentral.com/1755-8794/5/17">http://www.biomedcentral.com/1755-8794/5/17</a></p>
		<br>
                    <table cellpadding="0" cellspacing="0">
        <tbody>
        <tr><td>Received:</td><td>12 December 2011</td></tr>                <tr><td>Accepted:</td><td>7 May 2012</td></tr>        <tr><td>Published:</td><td>18 May 2012</td></tr>        </tbody>
        </table>
    		<!--<br/>-->
		<div style="line-height: 140%;">
        <p>

            Â© 2012 De Keulenaer et al.; licensee BioMed Central Ltd. <br>
        </p>
					<p>

				This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.
			</p>
		    	</div>
	</div></section>

</div>
<div id="article-body">
    <section><a name="abs"></a><h3>Abstract</h3>
   <div class="collapsible-content">
      <h4>Background</h4>
      <p style="line-height: 160%;">Hereditary hearing loss (HL) can originate from mutations in one of many genes involved
         in the complex process of hearing. Identification of the genetic defects in patients
         is currently labor intensive and expensive. While screening with Sanger sequencing
         for <em>GJB2</em> mutations is common, this is not the case for the other known deafness genes (&gt; 60).
         Next generation sequencing technology (NGS) has the potential to be much more cost
         efficient. Published methods mainly use hybridization based target enrichment procedures
         that are time saving and efficient, but lead to loss in sensitivity. In this study
         we used a semi-automated PCR amplification and NGS in order to combine high sensitivity,
         speed and cost efficiency.
      </p>
      <h4>Results</h4>
      <p style="line-height: 160%;">In this proof of concept study, we screened 15 autosomal recessive deafness genes
         in 5 patients with congenital genetic deafness. 646 specific primer pairs for all
         exons and most of the UTR of the 15 selected genes were designed using primerXL. Using
         patient specific identifiers, all amplicons were pooled and analyzed using the Roche
         454 NGS technology. Three of these patients are members of families in which a region
         of interest has previously been characterized by linkage studies. In these, we were
         able to identify two new mutations in <em>CDH23</em> and <em>OTOF</em>. For another patient, the etiology of deafness was unclear, and no causal mutation
         was found. In a fifth patient, included as a positive control, we could confirm a
         known mutation in <em>TMC1.</em></p>
      <h4>Conclusions</h4>
      <p style="line-height: 160%;">We have developed an assay that holds great promise as a tool for screening patients
         with familial autosomal recessive nonsyndromal hearing loss (ARNSHL). For the first
         time, an efficient, reliable and cost effective genetic test, based on PCR enrichment,
         for newborns with undiagnosed deafness is available.
      </p>
   </div>
</section><span id="keywords">
   <h5 class="inline">Keywords: </h5>Deafness; Next generation sequencing; PCR based enrichment; Genetic diagnostics</span><section><a name="sec1"></a><h3>Background</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">Hearing loss (HL) is the most common birth defect in industrialized countries and
         the most prevalent sensorineural disorder. One out of every 500 newborns has bilateral
         permanent sensorineural HL of more than 40â€‰dB HL <a name="d31349e286"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]. It is estimated that in developed countries, genetic causes are responsible in at
         least two-thirds of prelingual cases. In most of the cases there are no clinical abnormalities
         other than the hearing loss (i.e. nonsyndromic hearing loss, NSHL). Inherited NSHL
         is monogenic, with over 100 mapped loci and 46 causally implicated genes <a name="d31349e290"></a>[<a onclick="LoadInParent('#B2'); return false;" href="#B2">2</a>].
      </p>
      <p style="line-height: 160%;"><em>GJB2</em> mutations are the most frequent cause of autosomal recessive non-syndromic hearing
         loss (ARNSHL) and account for about 20â€‰% of the cases <a name="d31349e298"></a>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>]. Therefore, newborns that are diagnosed with severe-to-profound HL in the absence
         of other abnormal findings on physical examination are analyzed for mutations in the
         <em>GJB2</em> gene. In some cases, when imaging of the inner ear shows abnormalities such as an
         enlarged vestibular aquaduct, the <em>SLC26A4</em> gene is analyzed. Besides these genes there is hardly any other gene that is routinely
         analyzed in DNA diagnostics. For this reason, a positive result is only obtained in
         less than 20â€‰% of deaf children for which DNA diagnostics is requested <a name="d31349e308"></a>[<a onclick="LoadInParent('#B4'); return false;" href="#B4">4</a>]. While substantial progress has been made in recent years in identifying the responsible
         deafness genes, the key challenge lies in determining which gene is responsible in
         a patient. Sequencing of all genes by traditional DNA sequencing technology is labor
         intensive and not cost effective <a name="d31349e312"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>].
      </p>
      <p style="line-height: 160%;">Recently the next generation DNA sequencing technology has come of age <a name="d31349e318"></a>[<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>]. Companies such as Roche (454 Genome Sequencer FLX), Applied Biosystems (Solid System)
         and Illumina (Genome Analyzer) have brought high throughput DNA sequencers to the
         market that can sequence several hundred million to a few billions of basepairs in
         a few days. Until now, this technology has mainly been used for research purposes.
         Several genes for hearing loss have been identified using this technology <a name="d31349e322"></a>[<a onclick="LoadInParent('#B7'); return false;" href="#B7">7</a>]<a name="d31349e325"></a>[<a onclick="LoadInParent('#B8'); return false;" href="#B8">8</a>]. Most likely, the identification of other as yet unidentified deafness genes will
         follow. On the other hand, applications in DNA diagnostics in general are still rare.
         A first genetic test encompassing the NSHL genes using massively parallel sequencing
         technology has been described <a name="d31349e329"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>]<a name="d31349e332"></a>[<a onclick="LoadInParent('#B9'); return false;" href="#B9">9</a>]. In these studies, an array-based enrichment approach was used on a limited number
         of patients. Although this new technology holds the promise to significantly reduce
         the cost and workload per sample, the enrichment used leads to a significant loss
         in sensitivity, which is deemed unacceptable according to current standards using
         PCR amplification. Array based enrichment procedures inherently suffer from incomplete
         selection for two reasons. Firstly, due to the presence of repetitive sequences, not
         all fragments can be included in the selection set. Secondly, hybridization based
         enrichment suffers from selection bias and uneven capture efficiency. Much deeper
         sequencing will be needed to ensure complete or sufficient coverage of the selected
         fragments, in comparison to unbiased selection techniques such as PCR. In combination,
         these factors lead to a reduced sensitivity, while sensitivity requirements for DNA
         diagnostics are generally required to be very high. Here, we report the evaluation
         of a PCR based enrichment strategy followed by a 454 NGS approach for 5 patients using
         NGS implicated in ARNSHL.
      </p>
   </div>
</section>
<section><a name="sec2"></a><h3>Results</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">Five patients with familial congenital deafness were screened for 15 deafness genes
         on the 454 Genome Sequencer in order to evaluate if next generation sequencing enables
         the detection of mutations. Three of these patients are members of families in which
         a region of interest has previously been characterized by linkage studies. Therefore,
         mutations in respectively <em>CDH23</em> (for patients 1 and 2) and <em>OTOF</em> (for patient 4) were expected. In patient 5, a known mutation (c.236â€‰+â€‰1â€‰Gâ€‰&gt;â€‰A) in
         <em>TMC1</em> had to be confirmed. For patient 3, no mutation was known.
      </p>
      <p style="line-height: 160%;">We chose the Roche 454 next generation sequencing platform because of the longer read
         lengths (up to 400â€‰bp) of the pyrosequencing approach. We decided to screen the 15
         most important autosomal recessive deafness genes, selected on their reported mutation
         frequency <a name="d31349e353"></a>[<a onclick="LoadInParent('#B4'); return false;" href="#B4">4</a>] (Table <a name="d31349e356"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T1">1</a>). By limiting the number of genes to 15, we estimated to analyze 10 to 15 patients
         within a single sequencing run <a name="d31349e360"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>].
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T1','T1',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T1"><strong>Table 1.</strong></a> <strong>Analyzed genes</strong></p></div></div>
      <p style="line-height: 160%;">The enrichment protocol prior to the sequencing was based on an individual PCR amplification
         of all amplicons covering the coding region of the 15 genes. A prerequisite to use
         this PCR approach is that the reaction conditions for all the amplicons are uniform,
         enabling a semi-automated workflow by using 384-well plates and liquid handling robots.
         In a pilot experiment, we designed 162 forward and reversed primers for the coding
         regions of <em>OTOF</em> and <em>CDH23</em>, using our in-house developed primer design software package primerXL, in order to
         evaluate the uniformity of the reaction conditions. Amplification was performed on
         normal genomic DNA (Roche diagnostics) and all amplicons, except 3, could be amplified.
         An average Cq of 27 was obtained (Figure <a name="d31349e673"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/figure/F1">1</a>). The 3 failed amplicons were redesigned. Subsequently, primers were designed for
         the coding regions of the remaining 13 genes. In the end, a set of 646 primer pairs
         covering all the coding sequences and most of the UTRs of the 15 deafness genes in
         our setup was developed (Figure <a name="d31349e676"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/figure/F2">2</a>).
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/figure/F1','F1',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/figure/F1"><img src="1755-8794-5-17-1.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 1.</strong></a> <strong>Cq values for</strong><strong>â€‰<em>OTOF</em>â€‰</strong><strong>and</strong><strong>â€‰<em>CDH23</em>â€‰</strong><strong>.</strong> The Cq value is displayed in function of the corresponding amplicon for the <em>OTOF</em> and <em>CDH23</em> gene. The plot shows a drop-out of two amplicons (red).
      </p></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/figure/F2','F2',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/figure/F2"><img src="1755-8794-5-17-2.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 2.</strong></a> <strong>Cq values for the 15 genes.</strong> The Cq values for the amplicons covering all of the 15 genes are displayed. One amplicon
         of the <em>TRIOBP</em> gene failed during this PCR (red).
      </p></div></div>
      <p style="line-height: 160%;">In the next step, the patientsâ€™ DNA samples were amplified. Initially, the PCRs for
         patients 1, 2 and 3 were performed. These patients were analyzed with the standard
         chemistry on a full PicoTiterPlate using 2 large regions (Table <a name="d31349e735"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T2">2</a>). The total number of reads after filtering and mapping was 325.272, with an average
         read length of 250â€‰bp. The resulting mean coverage was 221x (patient 1), 168x (patient
         2) and 194x (patient 3). Approximately 95â€‰% of the screened amplicons had a coverage
         above 38, which makes them available for variant analysis (Table <a name="d31349e738"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T2">2</a>) <a name="d31349e742"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>]. As such high average coverage is not necessary for reliable variant analysis; the
         next run was modified accordingly.
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T2','T2',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T2"><strong>Table 2.</strong></a> <strong>Sequencing results</strong></p></div></div>
      <p style="line-height: 160%;">For patients 4 and 5, 1/5 of the capacity of a Standard run with 2-lane gasket was
         used for each patient. The average coverage was lower than expected and therefore
         an additional Titanium run (1/12) was performed to have extra reads (Table <a name="d31349e958"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T2">2</a>), resulting in an average total coverage of 73 and 88 for respectively patient 4
         and patient 5 (Table <a name="d31349e961"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T2">2</a>).
      </p>
      <p style="line-height: 160%;">A novel missense mutation in <em>CDH23</em> could be identified for patients 1 and 2, a result that is in agreement with the
         convincing linkage data previously found (data not shown). Both are homozygous for
         c.5527â€‰Gâ€‰&gt;â€‰T resulting in an amino acid substitution from Asp to Tyr at position p.1843
         in exon 43. The mutation was found in 241 reads out of 241 and in 101 reads out of
         101 for patient 1 and 2 respectively. Subsequently, the mutation was confirmed with
         Sanger sequencing. The mutation is classified as probably damaging by Polyphen, is
         not tolerated by SIFT (Table <a name="d31349e969"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T3','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T3">3</a>) and was not reported before, nor as disease causing mutation nor as a SNP. The mutation
         was analyzed in the whole family and showed full co-segregation with the phenotype.
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T3','T3',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T3"><strong>Table 3.</strong></a> <strong>New variants observed in patients 1, 2 and 4</strong></p></div></div>
      <p style="line-height: 160%;">In patient 3, we couldnâ€™t identify a mutation that is undeniably disease causing.
         After filtering all the variants, obtained with data analysis and filter settings
         as described in â€˜Methodsâ€™, we obtained a list of 171 variants. Ninety-seven of these
         are located in the introns (further than 10 basepairs away from the exonic regions).
         Another set of 22 variants is located in the UTRs and are all reported as polymorphisms.
         Of the 171 variants, 48 of them are previously reported as polymorphism or were identified
         in several patients in the lab. Eventually, 7 unique variants remained. Five of them
         are synonymous SNPs with no effect on splicing based on splice prediction programs.
         One deletion (c.3636_3637del) located in the <em>OTOF</em> gene, results in a frame shift (p.Phe1212fs). This mutation could not be confirmed
         with Sanger sequencing (see Table <a name="d31349e1071"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T4','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T4">4</a>). The second mutation results in an amino acid substitution in the <em>CDH23</em> gene and was found in 50 out of 96 reads. This variant is not known as a polymorphism.
         However, itâ€™s unclear that this missense mutation is causal as the prediction tools
         are not conclusive (see Table <a name="d31349e1077"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T4','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T4">4</a>). No additional mutation was found in the <em>CDH23</em> gene. Further evaluation of the known polymorphisms revealed a low population frequency
         for 4 of them. Although all were reported as non-pathogenic, we took a closer look
         at these. One variant occurs in the <em>TMPRSS3</em> gene with a relative frequency in the reads of 1, previously reported as rs35227181
         and considered to be non-pathogenic. Also the prediction tools didnâ€™t classify the
         variant as disease causing and therefore we considered this base change as a rare
         variant. In <em>OTOF </em>a compound heterozygous variant was found: c.2317â€‰Câ€‰&gt;â€‰T (p.Arg773Cys) and c.4936â€‰Câ€‰&gt;â€‰T
         (p.Pro1646Ser). Both variants were detected with a relative frequency of 0.5 and are
         predicted to be possibly damaging (Table <a name="d31349e1090"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T4','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T4">4</a>). The two variants were previously described in the literature, where they were considered
         as non-pathogenic <a name="d31349e1094"></a>[<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>]<a name="d31349e1097"></a>[<a onclick="LoadInParent('#B12'); return false;" href="#B12">12</a>]. Another variant was found in <em>PCDH15</em> in 380 out of 381 reads. The variant is known as rs4935502 and non-pathogenic according
         to dbSNP. Nonetheless, it is regarded as probably damaging by PolyPhen-2 and has a
         Grantham score of 126 (see Table <a name="d31349e1104"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T4','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T4">4</a>). All the homopolymer repeats of these deafness genes (Table <a name="d31349e1107"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T1">1</a>) were reanalyzed with Sanger sequencing since the 454 technology does not allow reliable
         sequencing of repeats greater than 6. No mutation could be detected in this analysis.
         It remains possible that the mutation is located in another deafness gene, not yet
         incorporated in our set, or that patient 3 is a non-genetic deafness case.
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T4','T4',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T4"><strong>Table 4.</strong></a> <strong>Variants observed in patient 3</strong></p></div></div>
      <p style="line-height: 160%;">The variant analysis of patient 4 revealed a new homozygous missense mutation in the
         <em>OTOF</em> gene (exon 26: c.3263â€‰Tâ€‰&gt;â€‰C (p.Leu1088Pro)). This mutation is in agreement with linkage
         analysis, previously performed, suggesting a disease causing mutation in the <em>OTOF</em> gene. Prediction programs revealed the mutation as possibly damaging (PolyPhen) and
         not tolerated (SIFT) and therefore can most likely be considered as disease causing.
         The mutation was investigated in all available family members and showed full co-segregation
         with the deafness. The prediction results of the novel mutations, with the different
         software approaches, are listed in Table <a name="d31349e1328"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T3','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T3">3</a>.
      </p>
      <p style="line-height: 160%;">In patient 5, included as a control, we could confirm the heterozygous mutation in
         the <em>TMC1</em> gene previously found with Sanger sequencing, with a relative variant frequency of
         0.5. The mutation NM_138691.2:c.236â€‰+â€‰1â€‰Gâ€‰&gt;â€‰A was found in 13 reads out of 26. This
         substitution is located in the donor splice site of intron 7.
      </p>
   </div>
</section>
<section><a name="sec3"></a><h3>Discussion</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">We have developed an assay to improve the molecular diagnosis of autosomal recessive
         nonsyndromic hearing loss (ARNSHL) by simultaneous sequencing of the exons, UTRs and
         alternative transcripts of 15 deafness genes. The selected list of genes includes
         the most frequently mutated recessive genes in patients with hearing loss (Table <a name="d31349e1342"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T1">1</a>). In contrast to hybridization based capture approaches <a name="d31349e1346"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>], we performed a PCR based enrichment strategy for all target regions of the 15 genes
         followed by sequencing with a 454 Genome Sequencer FLX. The optimization of the PCR
         amplification conditions made an efficient semi-automated high throughput processing
         of the many PCR reactions feasible and straightforward. The PCR reactions of all difficult
         amplicons can easily be repeated and sequenced with the conventional Sanger method.
         We are able to complete the screening in a relatively short period of time. The adapter
         ligation method (Shotgun protocol) was preferred over the fusion primer approach for
         amplicon sequencing, in order to reduce the sequencing cost and to obtain a more efficient
         workflow. When screening a limited number of genes, it is more advantageous to use
         gene specific primers with the forward and reverse adapters already incorporated in
         the oligoâ€™s. For the 15 genes (646 amplicons) in our setting, the fusion primer approach
         is cost prohibitive, since the adapter sequence needs to be incorporated in every
         single primer set. Therefore, the adapters were ligated in one reaction to the pool
         of amplicons.
      </p>
      <p style="line-height: 160%;">Two new mutations were discovered and a known heterozygous mutation could be confirmed.
         The depth of sequencing coverage was high (greater than 38) for over 90â€‰% of the amplicons,
         indicating that the majority of amplicons are covered sufficiently. Sanger sequencing
         remains indispensable to confirm results, to analyze homopolymer regions and to sequence
         drop-out amplicons. Since this was a pilot study, we restricted the screening to 5
         patients. However, by this approach we are able to screen 15 patients within a single
         Titanium run, making this workflow cost-effective. While further validation with a
         larger panel of positive controls is needed, our approach holds great promise. Although
         it was an advantage that the region of interest had already been localized by linkage
         analysis for patients 1â€“2 and 4; we believe that we would have found the mutation
         without the knowledge of the linked region. The main reason for this is that there
         were no other relevant mutations found in the remaining 14 analyzed genes that could
         be causal. The new variants that we found in <em>CDH23</em> and <em>OTOF</em> were homozygous with a relative frequency of 1. These mutations in <em>CDH23</em> and <em>OTOF</em> were only found in respectively patients 1â€“2 and 4 and not in the other analyzed
         deafness samples (e.g. we often did find the same SNPâ€™s in different patients). At
         the same time, there was no doubt about the causality of both mutations as predicted
         by the different software tools.
      </p>
      <p style="line-height: 160%;">Genetic counseling prior to analysis of genes potentially revealing Usher syndrome
         (<em>CDH23, PCDH15 </em>and<em> MYO7A</em>) is important and this should be explained to parents who agree to have this type
         of diagnostics performed for their child.
      </p>
      <p style="line-height: 160%;">With patient 3, in whom we were not able to determine the disease causing mutation,
         we illustrate that the major difficulty with this kind of analysis is the interpretation
         of the variants. In some cases, the disease causing nature of the variant will not
         be convincing and additional investigations as segregation in the family or functional
         analysis will be essential.
      </p>
   </div>
</section>
<section><a name="sec4"></a><h3>Conclusions</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">Our data demonstrate that the use of NGS technology holds promises as a tool for screening
         congenital deafness genes. The availability of a more profound diagnostic test compared
         to the actual â€œgene by geneâ€ analysis approach, will provide better opportunities
         to identify the disease causing mutation in patients permitting prompt management
         and accurate genetic counseling. Once a substantial number of patients originating
         from a specific population are analyzed, we will be able to determine the relative
         contribution of this set of 15 deafness genes to recessive hearing loss in this specific
         population, data which is currently unavailable. From a technical point of view, the
         screening can be performed even on a sporadic case. However less positive results
         will be obtained since there are a significant amount of deafness cases caused by
         non-genetic factors. A continued refinement of the NGS technology will further improve
         the sequencing accuracy and reduce the cost. At the same time bio-informatics tools
         will improve. This will be critical for the interpretation of NGS-data and essential
         for the diagnostic laboratories using NGS technology.
      </p>
   </div>
</section>
<section><a name="sec5"></a><h3>Methods</h3>
   <div class="collapsible-content">
      <h4>Patient material</h4>
      <p style="line-height: 160%;">Genomic DNA was obtained from five patients with congenital HL. Patient 1 and 2 are
         the probands of a consanguineous recessive Iranian family with at least 9 affected
         members for which linkage to the <em>CDH23</em> locus has been found previously. Patient 3, a member of a Turkish family, has congenital
         deafness. This patient has parents with normal hearing and <em>GJB2</em> mutations have been excluded. Patient 4 is the proband of a consanguineous Iranian
         family with recessive deafness, for which linkage to the <em>OTOF</em> locus has been proven. The family had 7 affected members. Both Iranian families were
         seen by a geneticist and filled in a general clinical questionnaire. However, no detailed
         audiometric or ophthalmological examinations were performed, because the families
         were collected in remote locations. Both families suffered from profound early childhood
         hearing loss and no obvious signs of syndromal hearing loss were noted. In patient
         5, a known mutation in the <em>TMC1</em> gene was identified previously, and this patient served as a positive control. As
         patient 3 was part of routine deafness screening, specific ethical approval was not
         required. Both Iranian families are part of a research project at the University of
         Antwerp, for which ethical approval was obtained.
      </p>
      <h4>Primerdesign and amplicon PCR</h4>
      <p style="line-height: 160%;">Fifteen autosomal recessive deafness genes were selected, based on their reported
         mutation frequency (Table <a name="d31349e1407"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T1">1</a>) <a name="d31349e1411"></a>[<a onclick="LoadInParent('#B4'); return false;" href="#B4">4</a>]. The genes with the highest frequency of mutations reported in the literature were
         chosen. Primer design using the in house developed primerXL pipeline (Lefever et al.,
         in preparation), resulted in 646 oligonucleotide pairs covering all the coding sequences
         (CDS) and most of the UTRs of the 15 genes responsible for ARNSHL. After the first
         round of primer design, with the most stringent conditions (no SNPs in primer annealing
         region, amplicon length between 250â€“350â€‰bp, GC content between 30 and 80â€‰%), 97.1â€‰%
         of all the regions of the 15 genes could be amplified successfully. The missing regions
         were caused by systematic drop-out of some amplicons during PCR. The primer design
         was optimized in different steps by accepting less stringent conditions; first by
         tolerating an increase in the number of generated primers (for example to allow the
         presence of a single SNP in the 5â€™ region of the primer), then by lowering the permitted
         amplicon length and finally by slightly varying the melting temperature (Tm) of the
         primers. All PCR reactions are performed using the same reaction conditions, enabling
         an automated workflow. The average length across the 646 amplicons is 319â€‰bp, resulting
         in an aggregate target size of approximately 200 Kb. The primers used in this step
         are modified at their 5' end with a universal M13 linker sequence. The primer sequences
         were synthesized by Integrated DNA technologies and delivered in 384-well plates as
         a 100â€‰Î¼M stock solution (forward and reverse primer separately).
      </p>
      <p style="line-height: 160%;">All 646 amplification reactions of the 15 genes were carried out as singleplex PCR
         in two 384-well plates per patient. A master mix for each sample was prepared and
         consisted of 1x Kapa Taq buffer (Sopachem), 1â€‰mM MgCl<sub>2</sub> (Roche Diagnostics), 0.12â€‰mM dNTPâ€™s (Invitrogen), 0.02 U/Î¼l Kapa Taq polymerase (Sopachem),
         0.32x LC Green Plus (BiokÃ©) and 25â€‰ng gDNA per reaction. Then, 1.25â€‰Î¼l of the forward/reverse
         primer mix (1â€‰Î¼M stock solution) was added to the 8.75â€‰Î¼l master mix using a Freedom
         EVO Tecan liquid handling robotic workstation, resulting in a final volume of 10â€‰Î¼l
         per reaction. The PCRs were first tested on Human Genomic DNA (Roche diagnostics).
         The amplification reactions were carried out on a C1000 real-time thermal cycler (Bio-Rad)
         with following cycling conditions: 95Â°C-5â€™, 94Â°C-30â€, 58Â°C-30â€, 72Â°C-50â€; 40 cycles
         (+ plate read after each one), followed by a melt curve (65Â°Câ€‰&gt;â€‰95Â°C for 5â€ increment
         0.5Â°C). The Cq value was determined and the size of the amplified products was verified
         on a MultiNA (Shimadzu Biotech). To obtain an equal representation of every amplicon
         in the pool, PCR products were pooled in an equimolar manner for each patient, based
         on the end-point fluorescent values. Subsequently, 100â€‰Î¼l of these pools were purified
         with the High Pure PCR Cleanup Micro Kit (Roche Diagnostics). The quality of the PCR
         pools was verified on an Agilent 2100 Bioanalyzer with the DNA 1000â€‰K chip (Agilent
         technologies) and the concentration was measured with the Quant-it Picogreen DNA assay
         (Invitrogen).
      </p>
      <h4>Next generation sequencing</h4>
      <p style="line-height: 160%;">The Roche A and B adapters with a MID (Multiplex IDentifier), were ligated to each
         purified pool of amplicons according to the Shotgun protocol for Low Molecular Weight
         (LMW) samples (Manual: â€˜GS FLX Shotgun DNA Library Preparation Method Manualâ€™ (December
         2007), skipping the fragmentation step. A different MID sequence was used for every
         patient (Figure <a name="d31349e1425"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/figure/F3">3</a>). Quality control of the library preparation was carried out on a RNA 6000 pico labchip
         (Agilent technologies) and concentration of the library was measured with the Quant-it
         Ribogreen RNA assay (Invitrogen). Emulsion PCR and pyrosequencing on the 454 Genome
         Sequencer FLX (GS FLX) were carried out as described (Titanium emPCR method manual,
         Titanium sequencing method manual Roche).
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/figure/F3','F3',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/figure/F3"><img src="1755-8794-5-17-3.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 3.</strong></a> <strong>Workflow for the PCR enrichment approach.</strong> Amplicons are equimolar pooled per patient, continuously the A and B (454) adapters
         with MID are ligated to each pool of amplicons. All the patients, each tagged with
         a unique MID, can be pooled before proceeding to emulsion PCR.
      </p></div></div>
      <h4>Sanger sequencing homopolymers</h4>
      <p style="line-height: 160%;">We identified the homopolymer regions, defined as 6 or more repeats of the same base,
         located in the exonic regions of the 15 deafness genes (Table <a name="d31349e1447"></a><a onclick="popup('http://www.biomedcentral.com/1755-8794/5/17/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1755-8794/5/17/table/T1">1</a>). These regions were analyzed with the conventional Sanger sequencing method for
         patient 3, since we couldnâ€™t confirm a causal mutation in this patient.
      </p>
      <h4>Data analysis</h4>
      <p style="line-height: 160%;">Mapping of the sequenced reads was performed by BLAT <a name="d31349e1457"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>], software that has been integrated in V.I.P. (Variant Identification Pipeline) <a name="d31349e1461"></a>[<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a>]. Variants were identified using the variant identification module included into V.I.P.
         with filter settings: homopolymersâ€‰&lt;â€‰7, Quality score â‰¥ 30, relative variant frequency
         â‰¥ 0.35 and total coverage â‰¥ 20. The novel identified variants were analyzed by Alamut
         version 1.53 (Interactive Biosoftware).
      </p>
   </div>
</section>
<section><a name="sec6"></a><h3>Abbreviations</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">ARNSHL, Autosomal recessive non-syndromic hearing loss; Bp, Basepairs; CDS, Coding
         sequences; GS FLX, Genome Sequencer FLX; HL, Hearing loss; LMW, Low molecular weight;
         MID, Multiplex Identifier; NGS, Next generation sequencing; PCR, Polymerase chain
         reaction; PTP, Picotiterplate; SNP, Single nucleotide polymorphism; UTR, Untranslated
         region; VIP, Variant identification pipeline.
      </p>
   </div>
</section>
<section><a name="sec7"></a><h3>Competing interests</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">The authors declare that they have no competing interests.</p>
   </div>
</section>
<section><a name="sec8"></a><h3>Authorsâ€™ contributions</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">SL designed the primers. FP contributed to the primer design. JH and SDK validated
         the primers. JR, HVDV and SDK performed the practical work. JDS analyzed the data.
         MAT and GVC performed the linkage studies. PC and SDK interpreted the data and wrote
         the paper. GVC contributed to the manuscript writing. JPR contributed in a major way
         to the optimization of the workflow. JH, FVN, DF, BS, DD, SB, WVC, JVDS, GVC and PC
         performed the initial study design. All authors read and approved the final manuscript.
      </p>
   </div>
</section>
<section><a name="ack"></a><h3>Acknowledgements</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">This research has been made possible by funding from Hercules Foundation [AUGE/039]
         and Ghent University IOF StepStone funding. This study was supported by a research
         grant from â€˜Action on Hearing Loss UKâ€™ to GVC and PC. The funders had no role in study
         design, data collection and analysis, decision to publish, or preparation of the manuscript.
         We thank Fatemeh Alasti for providing samples 1 and 2.
      </p>
   </div>
</section>
<section><a name="refs"></a><h3>References</h3>
   <div id="article-references" class="collapsible-content">
      <ol id="references">
         <li id="B1">
            <p><a name="B1"></a> Morton CC,  Nance WE: <strong> Current concepts: Newborn hearing screening - A silent revolution. </strong></p><p><em>New England Journal of Medicine</em> 2006,  <strong>354</strong><strong>:</strong>2151-2164.  <a target="_top" href="http://www.biomedcentral.com/pubmed/16707752">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16707752">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B1"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B2">
            <p><a name="B2"></a><strong> The Hereditary Hearing Loss Homepage </strong>[<a href="http://hereditaryhearingloss.org/">http://hereditaryhearingloss.org</a> <a href="http://www.webcitation.org/query.php?url=http://hereditaryhearingloss.org&amp;refdoi=10.1186/1755-8794-5-17" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>]
            </p>
         </li>
         <li id="B3">
            <p><a name="B3"></a> Putcha GV,  Bejjani BA,  Bleoo S,  Booker JK,  Carey JC,  Carson N, <em>et al</em>.: <strong> A multicenter study of the frequency and distribution of GJB2 and GJB6 mutations in
                  a large North American cohort. </strong></p><p><em>Genetics in Medicine</em> 2007,  <strong>9</strong><strong>:</strong>413-426.  <a target="_top" href="http://www.biomedcentral.com/pubmed/17666888">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17666888">Publisher&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B3"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B4">
            <p><a name="B4"></a> Hilgert N,  Smith RJH,  Van Camp G: <strong> Forty-six genes causing nonsyndromic hearing impairment: Which ones should be analyzed
                  in DNA diagnostics? </strong></p><p><em>Mutation Research-Reviews in Mutation Research</em> 2009,  <strong>681</strong><strong>:</strong>189-196.  <a target="_top" href="http://www.biomedcentral.com/pubmed/18804553">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18804553">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18804553">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B4"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B5">
            <p><a name="B5"></a> Shearer AE,  Deluca AP,  Hildebrand MS,  Taylor KR,  Gurrola J,  Scherer S, <em>et al</em>.: <strong> Comprehensive genetic testing for hereditary hearing loss using massively parallel
                  sequencing. </strong></p><p><em>Proceedings of the National Academy of Sciences of the United States of America</em> 2010,  <strong>107</strong><strong>:</strong>21104-21109.  <a target="_top" href="http://www.biomedcentral.com/pubmed/21078986">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21078986">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21078986">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B5"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B6">
            <p><a name="B6"></a> Zhang J,  Chiodini R,  Badr A,  Zhang GF: <strong> The impact of next-generation sequencing on genomics. </strong></p><p><em>Journal of Genetics and Genomics</em> 2011,  <strong>38</strong><strong>:</strong>95-109.  <a target="_top" href="http://www.biomedcentral.com/pubmed/21477781">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21477781">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21477781">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B6"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B7">
            <p><a name="B7"></a> Schraders M,  Haas SA,  Weegerink NJD,  Oostrik J,  Hu H,  Hoefsloot LH, <em>et al</em>.: <strong> Next-Generation Sequencing Identifies Mutations of SMPX, which Encodes the Small Muscle
                  Protein, X-Linked, as a Cause of Progressive Hearing Impairment. </strong></p><p><em>American Journal of Human Genetics</em> 2011,  <strong>88</strong><strong>:</strong>628-634.  <a target="_top" href="http://www.biomedcentral.com/pubmed/21549342">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21549342">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21549342">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B7"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B8">
            <p><a name="B8"></a> Rehman AU,  Morell RJ,  Belyantseva IA,  Khan SY,  Boger ET,  Shahzad M, <em>et al</em>.: <strong> Targeted Capture and Next-Generation Sequencing Identifies C9orf75, Encoding Taperin,
                  as the Mutated Gene in Nonsyndromic Deafness DFNB79. </strong></p><p><em>American Journal of Human Genetics</em> 2010,  <strong>86</strong><strong>:</strong>378-388.  <a target="_top" href="http://www.biomedcentral.com/pubmed/20170899">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20170899">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20170899">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B8"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B9">
            <p><a name="B9"></a> Brownstein Z,  Friedman LM,  Shahin H,  Oron-Karni V,  Kol N,  Rayyan AA, <em>et al</em>.: <strong> Targeted genomic capture and massively parallel sequencing to identify genes for hereditary
                  hearing loss in middle eastern families. </strong></p><p><em>Genome Biology</em> 2011,  <strong>12</strong><strong>:</strong>9.  <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B9"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B10">
            <p><a name="B10"></a> De LK,  De SJ,  Clement L,  Baetens M,  Lefever S,  De KS, <em>et al</em>.: <strong> Practical tools to implement massive parallel pyrosequencing of PCR products in next
                  generation molecular diagnostics. </strong></p><p><em>PLoS One</em> 2011,  <strong>6</strong><strong>:</strong>e25531.  <a target="_top" href="http://www.biomedcentral.com/pubmed/21980484">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21980484">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21980484">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B10"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B11">
            <p><a name="B11"></a> Smith RJH,  Gurrola JG: <strong> Kelley. </strong></p><p><em>PM</em> 1993. <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B11"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B12">
            <p><a name="B12"></a> Migliosi V,  Modamio-Hoybjor S,  Moreno-Pelayo MA,  Rodriguez-Ballesteros M,  Villamar M,  Telleria D, <em>et al</em>.: <strong> Q829X, a novel mutation in the gene encoding otoferlin (OTOF), is frequently found
                  in Spanish patients with prelingual non-syndromic hearing loss. </strong></p><p><em>J Med Genet</em> 2002,  <strong>39</strong><strong>:</strong>502-506.  <a target="_top" href="http://www.biomedcentral.com/pubmed/12114484">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12114484">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12114484">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B12"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B13">
            <p><a name="B13"></a> Kent WJ: <strong> BLAT - The BLAST-like alignment tool. </strong></p><p><em>Genome Research</em> 2002,  <strong>12</strong><strong>:</strong>656-664.  <a target="_top" href="http://www.biomedcentral.com/pubmed/11932250">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11932250">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11932250">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B13"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
         <li id="B14">
            <p><a name="B14"></a> De Schrijver JM,  De LK,  Lefever S,  Sabbe N,  Pattyn F,  Van NF, <em>et al</em>.: <strong> Analysing 454 amplicon resequencing experiments using the modular and database oriented
                  Variant Identification Pipeline. </strong></p><p><em>BMC Bioinformatics</em> 2010,  <strong>11</strong><strong>:</strong>269.  <a target="_top" href="http://www.biomedcentral.com/pubmed/20487544">PubMed&nbsp;Abstract</a> | <a target="_top" href="http://dx.doi.org/10.1186/1471-2105-11-269">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20487544">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="http://www.biomedcentral.com/sfx_links?ui=1755-8794-5-17&amp;bibl=B14"><img src="sfx_links" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script><a href="javascript:returnToText();">Return to text</a></p>
            <p></p>
         </li>
      </ol>
   </div>
</section>
<section><a name="bm"></a><h3>Pre-publication history</h3>
   <div class="collapsible-content">
      <p style="line-height: 160%;">The pre-publication history for this paper can be accessed here:</p>
      <p style="line-height: 160%;"><a href="http://www.biomedcentral.com/1755-8794/5/17/prepub">http://www.biomedcentral.com/1755-8794/5/17/prepub</a></p>
   </div>
</section>
    </div>
    </div></div>
</div>


		</div>
</div>
						
		<dl class="google-ad wide ">
	<dt class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Bottom' begin -->
		<script type="text/javascript">

		OAS_AD('Bottom');

		</script><a href="http://oas.biomedcentral.com/5c/biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17/L40/375272545/Bottom/biomed/Contra-IR-WODC-Terrapinn-2013/WODC_2013_728x90.gif/534c2f6351564a6e7436344141655771?x" target="_top"><img src="WODC_2013_728x90.gif" alt="" border="0"></a>
		<!-- OAS AD 'Bottom' end -->
		</dd>

		</dl>

		<noscript>

			<dl class="google-ad  noscript">  
    <dt class="hide"><a href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17/11647821715@Bottom?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17/16596001478@Bottom?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
				    <hr class="hide">
	
    

						
 
    
    
                                                                                                                                                                                                            
                                                                                                                                        
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<div id="footer">

									<span class="views"><a href="http://www.biomedcentral.com/1755-8794/5/17/?fmt_view=mobile">Mobile view</a> | <strong>Desktop view</strong></span>
			
	<div class="content">
    <div class="rounded ">
<div class="wrap-inner ">
        	
    <ul class="desktop">
                                                    
				        <li class="noborder"><a href="http://www.biomedcentral.com/about/tandc">Terms and Conditions</a></li>
<li><a href="http://www.biomedcentral.com/about/privacy">Privacy statement</a></li>
<li><a href="http://www.biomedcentral.com/presscenter">Press</a></li>
	<li><a href="http://www.biomedcentral.com/advertisers">Information for advertisers</a></li>
<li><a href="http://www.biomedcentral.com/about/bmcjobs">Jobs at BMC</a></li>
<li><a href="http://www.biomedcentral.com/support">Support</a></li>
<li><a href="http://www.biomedcentral.com/about/contact">Contact us</a></li>


    
    	    </ul>
    <p id="copyright">Â© 2013 
	BioMed Central Ltd unless otherwise stated. Part of Springer Science+Business&nbsp;Media.</p>
        </div>
</div>

	</div>

</div>
<div class="springer">
    <a href="http://www.springer.com/" target="blank"><img style="float: right; margin: 0px;" src="Springer_Branding_footer_logo.png"></a>
</div>
					    

<script>

// getElementsByClassName Polyfill
var getElementsByClassName=function(e,t,n){if(document.getElementsByClassName){getElementsByClassName=function(e,t,n){n=n||document;var r=n.getElementsByClassName(e),i=t?new RegExp("\\b"+t+"\\b","i"):null,s=[],o;for(var u=0,a=r.length;u<a;u+=1){o=r[u];if(!i||i.test(o.nodeName)){s.push(o)}}return s}}else if(document.evaluate){getElementsByClassName=function(e,t,n){t=t||"*";n=n||document;var r=e.split(" "),i="",s="http://www.w3.org/1999/xhtml",o=document.documentElement.namespaceURI===s?s:null,u=[],a,f;for(var l=0,c=r.length;l<c;l+=1){i+="[contains(concat(' ', @class, ' '), ' "+r[l]+" ')]"}try{a=document.evaluate(".//"+t+i,n,o,0,null)}catch(h){a=document.evaluate(".//"+t+i,n,null,0,null)}while(f=a.iterateNext()){u.push(f)}return u}}else{getElementsByClassName=function(e,t,n){t=t||"*";n=n||document;var r=e.split(" "),i=[],s=t==="*"&&n.all?n.all:n.getElementsByTagName(t),o,u=[],a;for(var f=0,l=r.length;f<l;f+=1){i.push(new RegExp("(^|\\s)"+r[f]+"(\\s|$)"))}for(var c=0,h=s.length;c<h;c+=1){o=s[c];a=false;for(var p=0,d=i.length;p<d;p+=1){a=i[p].test(o.className);if(!a){break}}if(a){u.push(o)}}return u}}return getElementsByClassName(e,t,n)}
</script>

						



<script type="text/javascript">

if (typeof jQuery == 'undefined') { document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.min.js' type='text/javascript'%3E%3C/script%3E")); }


document.documentElement.id = "js";

var site = {
			abbreviation: "/bmcmedgenomics",
	        type: "BMC_CORE",
        portalId: "9001",
                section: "browse",
                page: "fulltext",
    		
									homePortal: "biomedcentral.com",
		
			siteShowAds: "true",
				showAds: "true",
				url: "biomedcentral.com/bmcmedgenomics",
			portal: false,
	        adpage_url: "biomedcentral.com/bmcmedgenomics/article/10.1186/1755/8794/5/17",
        id: "2050",
	name: "BMC Medical Genomics",
	plugins: {
		scrollable: (getElementsByClassName("scrollable").length > 0),
				flowplayer: ((document.getElementById("flowplayer") != null) || (getElementsByClassName("myPlayer").length > 0) || (document.getElementById("videoarticle") != null) || (getElementsByClassName("flowplayer").length > 0) || (document.getElementById("audio") != null) || (document.getElementById("elifeplayer") != null)),
		lightbox: ((document.getElementById("image-highlight") != null) || (getElementsByClassName("lightbox").length > 0)),
		thickbox: (getElementsByClassName("thickbox").length > 0),
		altmetric: ((document.getElementById("altmetric") != null) || (getElementsByClassName("altmetric").length > 0))
	}
	
};



if(typeof quoteSearchUri == "undefined") { quoteSearchUri=null; }
</script>



        <script type="text/javascript" src="behaviours-0.js"></script>







		<script>

	if (site.plugins.scrollable == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.scrollable-1.0.1.min.js' %3E%3C/script%3E")); 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.tools.min.js'%3E%3C/script%3E"));
	}

	if (site.plugins.flowplayer == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/flowplayer-3.2.6.min.js'%3E%3C/script%3E"));
	}

	if (site.plugins.lightbox == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/browse_imagehighlight.js'%3E%3C/script%3E"));
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.colorbox.js' %3E%3C/script%3E"));
	}

	if (site.plugins.thickbox == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/thickbox-compressed.js' %3E%3C/script%3E"));
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/thickbox-flashhidden-patch.js' %3E%3C/script%3E"));
	}
	</script>





            <script type="text/javascript" src="jquery.fancybox-1.3.1.pack-0.js"></script>
            <script type="text/javascript" src="jquery.debug-0.js"></script>
                  <script type="text/javascript" src="articles-0.js"></script>
		<div class=" fb_reset" id="fb-root"><div style="position: absolute; top: -10000px; height: 0px; width: 0px;"><div><iframe src="17_5.html" style="border: medium none;" tab-index="-1" title="Facebook Cross Domain Communication Frame" aria-hidden="true" id="fb_xdm_frame_http" allowtransparency="true" name="fb_xdm_frame_http" frameborder="0" scrolling="no"></iframe><iframe src="https://s-static.ak.facebook.com/connect/xd_arbiter.php?version=27#channel=f36603f45bf38ba&amp;channel_path=%2F1755-8794%2F5%2F17%2F%3Ffb_xd_fragment%23xd_sig%3Df1b9c5c804914cc%26&amp;origin=http%3A%2F%2Fwww.biomedcentral.com" style="border: medium none;" tab-index="-1" title="Facebook Cross Domain Communication Frame" aria-hidden="true" id="fb_xdm_frame_https" allowtransparency="true" name="fb_xdm_frame_https" frameborder="0" scrolling="no"></iframe></div></div><div style="position: absolute; top: -10000px; height: 0px; width: 0px;"><div></div></div></div>











<!--[if lte IE 6]>
            <script type="text/javascript" src="/javascript/plugins/DD_belatedPNG_0.0.8a-0.js"></script>
    <script type="text/javascript">
        DD_belatedPNG.fix('.png_bg, ul.primary-nav li a, ul.primary-nav li a span, ul.secondary-nav li a, ul.secondary-nav li a span, ul li i, .plus-button, .facebook, .twitter, .fancy-bg, #fancybox-close');
    </script>
	                <link rel="stylesheet" type="text/css" href="http://www.bmcimg.com/css/hacks/ie6-0.css"/>
    <![endif]-->

<!-- Claudia Whitcombe wanted this comment here -->

       <div class="hide">
      <dl class="google-ad" id="x96banner">
        <dt style="display: block;" class="hide">
          <a class="skyscraper-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</a>
       </dt>
        <dd>
          <!-- OAS AD 'x96' begin -->
          <script type="text/javascript">

            OAS_AD('x96');
          </script><script type="text/javascript"><!--//<![CDATA[
	var m3_u = (location.protocol=='https:'?'https://delivery.pbgrd.com/openx/www/delivery/ajs.php':'http://delivery.pbgrd.com/openx/www/delivery/ajs.php');
	var m3_r = Math.floor(Math.random()*99999999999);

	var doi = null;

	var metas = document.getElementsByTagName('meta');
	var i;
	for (i = 0; i < metas.length; i++) {
		if (metas[i].getAttribute('name') == "citation_doi") {
			break;
		}
	}
	if (metas[i] != undefined) doi = metas[i].getAttribute('content');

	if (doi==null || doi=='') {
		doi = 'NODOI - ' + encodeURIComponent(document.URL);
	}
	else if (doi.substring(0,3)!='10.' || doi.indexOf('/')<0) {
		doi = 'BADDOI - ' + encodeURIComponent(document.URL);
	}

	if (!document.MAX_used) document.MAX_used = ',';
	document.write ("<scr"+"ipt type='text/javascript' src='"+m3_u);
	document.write ("?zoneid=5");

	document.write ("&doi=" + doi + "&campaignName=biomed");

	document.write ('&amp;cb=' + m3_r);
	if (document.MAX_used != ',') document.write ("&amp;exclude=" + document.MAX_used);
	document.write (document.charset ? '&amp;charset='+document.charset : (document.characterSet ? '&amp;charset='+document.characterSet : ''));
	document.write ("&amp;loc=" + escape(window.location));
	if (document.referrer) document.write ("&amp;referer=" + escape(document.referrer));
	if (document.context) document.write ("&context=" + escape(document.context));
	if (document.mmm_fo) document.write ("&amp;mmm_fo=1");
	document.write ("'><\/scr"+"ipt>");
//]]>--></script><script type="text/javascript" src="ajs_001.php"></script><script type="text/javascript"><!--//<![CDATA[
var headerheight = 0;
var naviheight = 0;
var startspace = 0;
var scrollspace = 0;
if (navigator.appName == "Microsoft Internet Explorer") {
var windowheight = window.document.documentElement.clientHeight;
} else {
var windowheight = window.innerHeight;
}
var headerheight = document.getElementById("content").offsetTop;
//	var naviheight = document.getElementById("box-outline1").offsetTop;
//	var startspace = windowheight -(headerheight+15+naviheight);
//	var scrollspace = windowheight-(20+naviheight);
if (document.getElementById("box-outline") == null) {
var startspace = windowheight -(headerheight+15);
var scrollspace = windowheight -(headerheight+15);
} else {
var naviheight = document.getElementById("box-outline1").offsetTop;
var startspace = windowheight -(headerheight+15+naviheight);
var scrollspace = windowheight-(20+naviheight);
}
var journal = "";
var metas = document.getElementsByTagName('meta');
var i;
for (i = 0; i < metas.length; i++) {
if (metas[i].getAttribute('name') == "citation_journal_title") {
break;
}
}
if (metas[i] != undefined) journal = encodeURIComponent(metas[i].getAttribute('content'));
var proto = (location.protocol=='https:') ? 'https' : 'http';
var pg_base = proto + '://delivery.pbgrd.com/images/';
var asrand = Math.floor(Math.random()*99999999999);
var pg_dest = pg_base + '?project=biomed&amp;asrand=' + asrand;
document.write ('<img src="' + pg_dest + '" alt="" width=1 height=1>');
document.write ('<scr' + 'ipt type="text/javascript" src="' + proto + '://delivery.pbgrd.com/banner/html-banner-biomed-multispv11.php?doi=10.1186/1755-8794-5-17&clickurl=http://delivery.pbgrd.com/openx/www/delivery/ck.php?oaparams=2__bannerid=103__zoneid=5__cb=d336898718__doi=10.1186%2F1755-8794-5-17__oadest=&campaign=biomed_newlabels&zoneid=5&bannerid=103&pageurl=http%3A%2F%2Fwww.biomedcentral.com%2F1755-8794%2F5%2F17%2F&bstyle=multi');
document.write ("&journal=" + journal);
document.write ("&startspace=" + startspace);
document.write ("&scrollspace=" + scrollspace);
document.write ("&verty=" + screen.height);
if (document.getElementById("box-outline") == null) {
document.write ("&mode=abstracts");
}
document.write ('"><\/scr'+'ipt>');
//]]>--></script><img src="untitled" alt="" height="1" width="1"><script type="text/javascript" src="html-banner-biomed-multispv11.php"></script><script type="text/javascript">document.getElementById("box-outline1").appendChild(document.getElementById("x96bannerpg"))</script>
<div id="beacon_d336898718" style="position: absolute; left: 0px; top: 0px;"><img src="lg_001.php" alt="" style="width: 1px; height: 1px;" height="1" width="1"></div>

        </dd>
      </dl>
    </div><iframe style="width: 1px; height: 1px; position: absolute; top: -100px;" src="17_7.html" id="oauth2relay836478196" name="oauth2relay836478196"></iframe>  
    



<div id="fancybox-tmp"></div><div id="fancybox-loading"><div></div></div><div id="fancybox-overlay"></div><div id="fancybox-wrap"><div id="fancybox-outer"><div class="fancy-bg" id="fancy-bg-n"></div><div class="fancy-bg" id="fancy-bg-ne"></div><div class="fancy-bg" id="fancy-bg-e"></div><div class="fancy-bg" id="fancy-bg-se"></div><div class="fancy-bg" id="fancy-bg-s"></div><div class="fancy-bg" id="fancy-bg-sw"></div><div class="fancy-bg" id="fancy-bg-w"></div><div class="fancy-bg" id="fancy-bg-nw"></div><div id="fancybox-inner"></div><a id="fancybox-close"></a><a href="javascript:;" id="fancybox-left"><span class="fancy-ico" id="fancybox-left-ico"></span></a><a href="javascript:;" id="fancybox-right"><span class="fancy-ico" id="fancybox-right-ico"></span></a></div></div></body>
</html>
