<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="en" class="no-js js" itemtype="http://schema.org/Article" itemscope="" xml:lang="en" xmlns:xsd="http://www.w3.org/2001/XMLSchema-datatypes#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:doi="http://dx.doi.org/" xmlns:dc="http://purl.org/dc/terms/" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">


<meta name="wpd_version" content="0.2">
<meta name="wpd_baseurl" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0053083">
<meta name="wpd_url" content="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0053083">
<meta name="wpd_date" content="2013-09-23T12:19Z">
<script src="articles" async="" id="_jqjsp0"></script>
  <title>PLOS ONE: Comprehensive Mutation Analysis for Congenital Muscular Dystrophy: A Clinical PCR-Based Enrichment and Next-Generation Sequencing Panel</title>





    <!--[if lte IE 7]>
<link rel="stylesheet" type="text/css"  href="/css/lte_ie7-min.css?v=3bykQUyQmReeuobVyPozcJ9LxRc" />
    <![endif]-->











  

  <!--chartbeat -->
  <script src="ga.js" async="" type="text/javascript"></script><script type="text/javascript">var _sf_startpt = (new Date()).getTime()</script>
  <script>document.documentElement.className += ' js';</script>

  
  <meta http-equiv="X-UA-Compatible" content="IE=EmulateIE7; IE=EmulateIE9">
  <meta name="description" content="PLOS ONE: an inclusive, peer-reviewed, open-access resource from the PUBLIC LIBRARY OF SCIENCE. Reports of well-performed scientific studies from all disciplines freely available to the whole world.">
  <meta name="keywords" content="PLOS, Public Library of Science, Open Access, Open-Access, Science, Medicine, Biology, Research, Peer-review, Inclusive, Interdisciplinary, Ante-disciplinary, Physics, Chemistry, Engineering">
  <meta name="almHost" content="http://alm.plos.org/api/v3/articles">
  <meta name="searchHost" content="http://api.plos.org/search">
  <meta name="termsHost" content="http://api.plos.org/terms">
  <meta name="solrApiKey" content="plos">
  <meta name="almAPIKey" content="3pezRBRXdyzYW6ztfwft">
  <meta name="currentJournal" content="PLoSONE">
  <meta name="almRequestBatchSize" content="">

  <meta name="citation_publisher" content="Public Library of Science">
  <meta name="citation_doi" content="10.1371/journal.pone.0053083">

    <meta name="citation_title" content="Comprehensive Mutation Analysis for Congenital Muscular Dystrophy: A Clinical PCR-Based Enrichment and Next-Generation Sequencing Panel">
    <meta itemprop="name" content="Comprehensive Mutation Analysis for Congenital Muscular Dystrophy: A Clinical PCR-Based Enrichment and Next-Generation Sequencing Panel">

      <meta name="citation_author" content="C. Alexander Valencia">
            <meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America">
            <meta name="citation_author_institution" content="Division of Human Genetics, Cincinnati Children's Hospital Medical Center, Cincinnati, Ohio, United States of America">
            <meta name="citation_author_institution" content="Department of Pediatrics, University of Cincinnati Medical School, Cincinnati, Ohio, United States of America">
      <meta name="citation_author" content="Arunkanth Ankala">
            <meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America">
      <meta name="citation_author" content="Devin Rhodenizer">
            <meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America">
      <meta name="citation_author" content="Shruti Bhide">
            <meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America">
      <meta name="citation_author" content="Martin Robert Littlejohn">
            <meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America">
      <meta name="citation_author" content="Lisa Mari Keong">
            <meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America">
      <meta name="citation_author" content="Anne Rutkowski">
            <meta name="citation_author_institution" content="Cure CMD and Kaiser SCPMG, Los Angeles, California, United States of America">
      <meta name="citation_author" content="Susan Sparks">
            <meta name="citation_author_institution" content="Carolinas Medical Center, Charlotte, North Carolina, United States of America">
      <meta name="citation_author" content="Carsten Bonnemann">
            <meta name="citation_author_institution" content="Neuromuscular and Neurogenetic Disorders of Childhood Section, Neurogenetics Branch, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, Maryland, United States of America">
      <meta name="citation_author" content="Madhuri Hegde">
            <meta name="citation_author_institution" content="Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America">

    <meta name="citation_date" content="2013/1/11">

  <meta name="citation_pdf_url" content="http://dx.plos.org/10.1371/journal.pone.0053083.pdf">

      <meta name="citation_journal_title" content="PLOS ONE">
    <meta name="citation_firstpage" content="e53083">
    <meta name="citation_issue" content="1">
    <meta name="citation_volume" content="8">
    <meta name="citation_issn" content="1932-6203">

    <meta name="citation_journal_abbrev" content="PLoS ONE">

      <meta name="citation_reference" content="citation_title=Mutation screening in 86 known X-linked mental retardation genes by droplet-based multiplex PCR and massive parallel sequencing; citation_volume=3; citation_number=1; citation_pages=41-49; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=Massively parallel sequencing of ataxia genes after array-based enrichment; citation_volume=31; citation_number=2; citation_pages=494-499; citation_date=2010; ">
      <meta name="citation_reference" content="citation_title=The congenital muscular dystrophies in 2004: a century of exciting progress; citation_volume=14; citation_number=3; citation_pages=635-649; citation_date=2004; ">
      <meta name="citation_reference" content="citation_title=Congenital muscular dystrophy. Part I: a review of phenotypical and diagnostic aspects; citation_volume=67; citation_number=4; citation_pages=144-168; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=Congenital muscular dystrophies and the extracellular matrix; citation_volume=13; citation_number=5; citation_pages=80-89; citation_date=2006; ">
      <meta name="citation_reference" content="citation_title=Relative frequency of congenital muscular dystrophy subtypes: analysis of the UK diagnostic service 2001–2008; citation_volume=22; citation_number=6; citation_pages=522-527; citation_date=2012; ">
      <meta name="citation_reference" content="citation_title=Congenital muscular dystrophies: new aspects of an expanding group of disorders; citation_volume=1772; citation_number=7; citation_pages=159-172; citation_date=2007; ">
      <meta name="citation_reference" content="citation_title=Congenital muscular dystrophy. Part II: a review of pathogenesis and therapeutic perspectives; citation_volume=67; citation_number=8; citation_pages=343-362; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=The congenital muscular dystrophies: recent advances and molecular insights; citation_volume=9; citation_number=9; citation_pages=427-443; citation_date=2006; ">
      <meta name="citation_reference" content="citation_title=Exome sequencing identifies WDR35 variants involved in Sensenbrenner syndrome; citation_volume=87; citation_number=10; citation_pages=418-423; citation_date=2010; ">
      <meta name="citation_reference" content="citation_title=Exome sequencing identifies MLL2 mutations as a cause of Kabuki syndrome; citation_volume=42; citation_number=11; citation_pages=790-793; citation_date=2010; ">
      <meta name="citation_reference" content="citation_title=Exome sequencing identifies the cause of a mendelian disorder; citation_volume=42; citation_number=12; citation_pages=30-35; citation_date=2010; ">
      <meta name="citation_reference" content="citation_title=Direct selection of human genomic loci by microarray hybridization; citation_volume=4; citation_number=13; citation_pages=903-905; citation_date=2007; ">
      <meta name="citation_reference" content="citation_title=Hybrid selection of discrete genomic intervals on custom-designed microarrays for massively parallel sequencing; citation_volume=4; citation_number=14; citation_pages=960-974; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=Genome-wide in situ exon capture for selective resequencing; citation_volume=39; citation_number=15; citation_pages=1522-1527; citation_date=2007; ">
      <meta name="citation_reference" content="citation_title=Microarray-based genomic selection for high-throughput resequencing; citation_volume=4; citation_number=16; citation_pages=907-909; citation_date=2007; ">
      <meta name="citation_reference" content="citation_title=Detection of genomic variation by selection of a 9 mb DNA region and high throughput sequencing; citation_volume=4; citation_number=17; citation_pages=e6659; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing; citation_volume=27; citation_number=18; citation_pages=182-189; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=Microdroplet-based PCR enrichment for large-scale targeted sequencing; citation_volume=27; citation_number=19; citation_pages=1025-1031; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=Targeted sequencing with microfluidics; citation_volume=27; citation_number=20; citation_pages=998-999; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=Sequencing technologies - the next generation; citation_volume=11; citation_number=21; citation_pages=31-46; citation_date=2010; ">
      <meta name="citation_reference" content="citation_title=Next-generation sequencing: from basic research to diagnostics; citation_volume=55; citation_number=22; citation_pages=641-658; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=Assessment of target enrichment platforms using massively parallel sequencing for the mutation detection for congenital muscular dystrophy; citation_volume=14; citation_number=23; citation_pages=233-246; citation_date=2012; ">
      <meta name="citation_reference" content="citation_title=Keeping up with the next generation: massively parallel sequencing in clinical diagnostics; citation_volume=10; citation_number=24; citation_pages=484-492; citation_date=2008; ">
      <meta name="citation_reference" content="citation_title=Targeted high-throughput DNA sequencing for gene discovery in retinitis pigmentosa; citation_volume=664; citation_number=25; citation_pages=325-331; citation_date=2010; ">
      <meta name="citation_reference" content="citation_title=Targeted capture and next-generation sequencing identifies C9orf75, encoding taperin, as the mutated gene in nonsyndromic deafness DFNB79; citation_volume=86; citation_number=26; citation_pages=378-388; citation_date=2010; ">
      <meta name="citation_reference" content="citation_title=Targeted next-generation sequencing appoints c16orf57 as clericuzio-type poikiloderma with neutropenia gene; citation_volume=86; citation_number=27; citation_pages=72-76; citation_date=2010; ">
      <meta name="citation_reference" content="citation_title=Multiplex amplification of large sets of human exons; citation_volume=4; citation_number=28; citation_pages=931-936; citation_date=2007; ">
      <meta name="citation_reference" content="citation_title=Simultaneous mutation and copy number variation (CNV) detection by multiplex PCR-based GS-FLX sequencing; citation_volume=30; citation_number=29; citation_pages=472-476; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=Genetic diagnosis by whole exome capture and massively parallel DNA sequencing; citation_volume=106; citation_number=30; citation_pages=19096-19101; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=Targeted capture and massively parallel sequencing of 12 human exomes; citation_volume=461; citation_number=31; citation_pages=272-276; citation_date=2009; ">
      <meta name="citation_reference" content="citation_title=Identification by whole-genome resequencing of gene defect responsible for severe hypercholesterolemia; citation_volume=19; citation_number=32; citation_pages=4313-4318; citation_date=2010; ">
      <meta name="citation_reference" content="citation_title=Assuring the quality of next-generation sequencing in clinical laboratory practice; citation_volume=30; citation_number=33; citation_pages=1033-1036; citation_date=2012; ">

  <link rel="canonical" href="http://dx.plos.org/info:doi/10.1371/journal.pone.0053083">

    <meta name="twitter:card" content="summary">
    <meta name="twitter:site" content="@plosone">
    <meta name="twitter:url" content="http://dx.plos.org/info:doi/10.1371/journal.pone.0053083">
    <meta name="twitter:title" content="Comprehensive Mutation Analysis for Congenital Muscular Dystrophy: A Clinical PCR-Based Enrichment and Next-Generation Sequencing Panel">
    <meta name="twitter:description" content="The congenital muscular dystrophies (CMDs) comprise a heterogeneous group of heritable muscle disorders with often difficult to interpret muscle pathology, making them challenging to diagnose. Serial Sanger sequencing of suspected CMD genes, while the current molecular diagnostic method of choice, can be slow and expensive. A comprehensive panel test for simultaneous screening of mutations in all known CMD-associated genes would be a more effective diagnostic strategy. Thus, the CMDs are a model disorder group for development and validation of next-generation sequencing (NGS) strategies for diagnostic and clinical care applications. Using a highly multiplexed PCR-based target enrichment method (RainDance) in conjunction with NGS, we performed mutation detection in all CMD genes of 26 samples and compared the results with Sanger sequencing. The RainDance NGS panel showed great consistency in coverage depth, on-target efficiency, versatility of mutation detection, and genotype concordance with Sanger sequencing, demonstrating the test's appropriateness for clinical use. Compared to single tests, a higher diagnostic yield was observed by panel implementation. The panel's limitation is the amplification failure of select gene-specific exons which require Sanger sequencing for test completion. Successful validation and application of the CMD NGS panel to improve the diagnostic yield in a clinical laboratory was shown.">
 <!--end articleInfoX-->

  <link rel="pingback" href="http://www.plosone.org/pingback">


  <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
  <link rel="home" title="home" href="http://www.plosone.org/">
  <link rel="alternate" type="application/rss+xml" title="PLOS ONE: New Articles" href="http://www.plosone.org/article/feed">
<script src="jquery-1.9.1.min.js" type="text/javascript"></script>
<link rel="stylesheet" type="text/css" href="infodoi10.1371journal.pone.css" media="all">
</head>
<body><div style="display: none;" id="MathJax_Message"></div>

  <div id="page-wrap">
    <div id="topbanner" class="cf">

<!-- Div for the ad at the top of journal home page-->
<div class="center">
  <div class="title">Advertisement</div>
  <iframe id="a3ac9da4" name="a3ac9da4" src="infodoi10.1371journal.pone_1.html" frameborder="0" height="90" scrolling="no" width="730">

    <a href='http://ads.plos.org/www/delivery/ck.php?n=a3ac9da4&amp;cb=4384'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=345&amp;cb=2618&amp;n=a3ac9da4'
      border='0' alt=''/>
    </a>
  </iframe>
</div>    </div>

    <div id="pagehdr-wrap">
      <div id="pagehdr">
        <div id="user" class="nav">
          <ul>
            <li><a href="http://www.plos.org/">plos.org</a></li>
            <li><a href="https://register.plos.org/ambra-registration/register.action">create account</a></li>
            <li class="btn-style"><a href="http://www.plosone.org/user/secure/secureRedirect.action?goTo=%2Farticle%2FfetchArticle.action%3FarticleURI%3Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0053083">sign in</a>
            </li>
          </ul>
        </div>
        <div class="logo">
          <a href="http://www.plosone.org/"><img src="logo.png" alt="PLOS ONE"></a>
        </div>

<div id="nav-main" class="nav">
  <ul>
        <li id="mn-01"><a href="http://www.plosone.org/taxonomy" class="areas-link">Subject Areas</a></li>
    <li id="mn-02"><a href="javascript:void(0);">For Authors</a>
      <div class="submenu" style="width: 540px; margin-left: -300px;">
        <div class="block">
          <div class="submit-script">
            <h3>Submit your Manuscript</h3>
            <ul>
              <li>Fair, rigorous peer review</li>
              <li>Broad scope and wide reach</li>
            </ul>
            <a href="http://www.plosone.org/static/submissionInstructions" class="btn">get started</a>
          </div>
        </div>
        <div class="menu">
          <ul>
            <li><a href="http://www.plosone.org/static/publish">Why Publish with PLOS ONE</a></li>
            <li><a href="http://www.plosone.org/static/publication">Publication Criteria</a></li>
            <li><a href="http://www.plosone.org/static/editorial">Editorial Policies</a></li>
            <li><a href="http://www.plosone.org/static/guidelines">Preparing A Manuscript</a></li>
            <li><a href="http://www.plosone.org/static/figureGuidelines">Figure and Table Guidelines</a></li>
          <li><a href="http://www.plosone.org/static/supportingInformation">Supporting Information Guidelines</a></li>
            <li><a href="http://www.plosone.org/static/submissionInstructions">Submitting a Manuscript</a></li>
          </ul>
        </div>
      </div>
    </li>

    <li id="mn-03"><a href="javascript:void(0);">About Us</a>
      <div class="submenu" style="width: 248px; margin-left: -30px;">
        <div class="menu">
          <ul>
            <li><a href="http://www.plosone.org/static/information">Journal Information</a></li>
            <li><a href="http://www.plosone.org/static/edboard">Editorial Board</a></li>
            <li><a href="http://www.plosone.org/static/reviewerGuidelines">Reviewer Guidelines</a></li>
            <li><a href="http://www.plosone.org/static/almInfo">Article-Level Metrics</a></li>
            <li><a href="http://www.plosone.org/static/license">Open-Access License</a></li>
            <li><a href="http://www.plosone.org/static/downloads">Media Downloads</a></li>
            <li><a href="http://www.plosone.org/static/commentGuidelines">Guidelines for Comments and Corrections</a></li>
            <li><a href="http://www.plosone.org/static/help">Help Using this Site</a></li>
            <li><a href="http://www.plosone.org/static/contact">Contact Us</a></li>
          </ul>
        </div>
      </div>
    </li>
  </ul>
<div id="db">
  <form name="searchForm" action="http://www.plosone.org/search/simple?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0053083" method="get">
<input name="from" value="globalSimpleSearch" id="from" type="hidden"><input name="filterJournals" value="PLoSONE" id="filterJournals" type="hidden">    <fieldset>
      <legend>Search</legend>
      <label for="search">Search</label>
      <div class="wrap">
        <input id="search" name="query" placeholder="Search" type="text">
        <input alt="SEARCH" src="icon.search.gif" type="image">
      </div>
    </fieldset>
  </form>
    <a id="advSearch" href="http://www.plosone.org/search/advanced?noSearchFlag=true&amp;query=&amp;articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0053083&amp;filterJournals=PLoSONE">advanced search</a>
</div></div>

      </div>
      <!-- pagehdr-->
    </div><div id="taxonomy-browser" class="areas">
<div class="wrapper">
<p class="breadcrumb"></p>
<div class="levels">
<div class="levels-container cf">
<div class="levels-position"></div>
</div>
<a href="#" class="prev"></a>
<a href="#" class="next active"></a>
</div>
</div>
</div>
    <!-- pagehdr-wrap -->

  <!--body and html tags gets closed in global_footer.ftl-->
<script type="text/javascript" src="MathJax.js"></script>

<div id="pagebdy-wrap">
  <div id="pagebdy">

    <div id="article-block" class="cf">

<div class="article-meta cf">
  <ul id="almSignPost" style="display: none;"></ul>
  <div class="article-type">
    <span class="type oa">Open Access</span>
      <span class="type pr">Peer-Reviewed</span>
  </div>
</div>

<div class="header" id="hdr-article">

<div class="article-kicker">
  <span style="cursor: text;" id="article-type-heading">Research Article</span>


</div>  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">

    Comprehensive Mutation Analysis for Congenital Muscular Dystrophy: A Clinical PCR-Based Enrichment and Next-Generation Sequencing Panel
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            C. Alexander Valencia
              <span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: C. Alexander Valencia, Arunkanth Ankala</p>

              
              

                <p>Affiliations:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America, 
                  Division of Human Genetics, Cincinnati Children's Hospital Medical Center, Cincinnati, Ohio, United States of America, 
                  Department of Pediatrics, University of Cincinnati Medical School, Cincinnati, Ohio, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Arunkanth Ankala
              <span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: C. Alexander Valencia, Arunkanth Ankala</p>

              
              

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Devin Rhodenizer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Shruti Bhide, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Martin Robert Littlejohn, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Lisa Mari Keong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Anne Rutkowski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Cure CMD and Kaiser SCPMG, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Susan Sparks, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Carolinas Medical Center, Charlotte, North Carolina, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Carsten Bonnemann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Neuromuscular and Neurogenetic Disorders of Childhood Section, Neurogenetics Branch, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Madhuri Hegde
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:mhegde@emory.edu">mhegde@emory.edu</a></p>

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>

</div><!--end header-->
<div class="main cf" id="pjax-container">
  

<div class="nav items-5" id="nav-article">
  <ul>
  <li>
        <span class="active" name="article">Article</span>
  </li>
  <li>
      <a href="http://www.plosone.org/article/authors/info%3Adoi%2F10.1371%2Fjournal.pone.0053083" name="authors">About the Authors</a>
  </li>
  <li>
      <a href="http://www.plosone.org/article/metrics/info%3Adoi%2F10.1371%2Fjournal.pone.0053083" name="metrics">Metrics</a>
  </li>
  <li>
      <a href="http://www.plosone.org/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0053083" name="comments">Comments</a>
  </li>
  <li>
      <a href="http://www.plosone.org/article/related/info%3Adoi%2F10.1371%2Fjournal.pone.0053083" name="related">Related Content</a>
  </li>
  </ul>
</div>

<script type="text/javascript">

  var selected_tab = "article";
</script>
  <div id="figure-thmbs" class="carousel cf">
    <div class="wrapper">
      <div class="slider">
              <div class="item clone">
                <a href="#pone-0053083-g002" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="fetchObject.action" alt="">
                  </span>
                </a>
              </div><div class="item clone">
                <a href="#pone-0053083-t008" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t008" title="Table 8">
                  <span class="thmb-wrap">
                    <img src="fetchObject_001.action" alt="">
                  </span>
                </a>
              </div><div class="item clone">
                <a href="#pone-0053083-g003" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="fetchObject_002.action" alt="">
                  </span>
                </a>
              </div><div class="item empty clone"></div><div class="item">
                <a href="#pone-0053083-t001" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="fetchObject_003.action" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0053083-t002" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="fetchObject_004.action" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0053083-t003" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="fetchObject_005.action" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0053083-t004" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="fetchObject_006.action" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0053083-t005" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t005" title="Table 5">
                  <span class="thmb-wrap">
                    <img src="fetchObject_007.action" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0053083-t006" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t006" title="Table 6">
                  <span class="thmb-wrap">
                    <img src="fetchObject_008.action" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0053083-g001" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.g001" title="Figure 1">
                  <span class="thmb-wrap">
                    <img src="fetchObject_009.action" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0053083-t007" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t007" title="Table 7">
                  <span class="thmb-wrap">
                    <img src="fetchObject_010.action" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0053083-g002" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.g002" title="Figure 2">
                  <span class="thmb-wrap">
                    <img src="fetchObject.action" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0053083-t008" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t008" title="Table 8">
                  <span class="thmb-wrap">
                    <img src="fetchObject_001.action" alt="">
                  </span>
                </a>
              </div>
              <div class="item">
                <a href="#pone-0053083-g003" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.g003" title="Figure 3">
                  <span class="thmb-wrap">
                    <img src="fetchObject_002.action" alt="">
                  </span>
                </a>
              </div>
      <div class="item empty"></div><div class="item clone">
                <a href="#pone-0053083-t001" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t001" title="Table 1">
                  <span class="thmb-wrap">
                    <img src="fetchObject_003.action" alt="">
                  </span>
                </a>
              </div><div class="item clone">
                <a href="#pone-0053083-t002" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t002" title="Table 2">
                  <span class="thmb-wrap">
                    <img src="fetchObject_004.action" alt="">
                  </span>
                </a>
              </div><div class="item clone">
                <a href="#pone-0053083-t003" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t003" title="Table 3">
                  <span class="thmb-wrap">
                    <img src="fetchObject_005.action" alt="">
                  </span>
                </a>
              </div><div class="item clone">
                <a href="#pone-0053083-t004" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t004" title="Table 4">
                  <span class="thmb-wrap">
                    <img src="fetchObject_006.action" alt="">
                  </span>
                </a>
              </div></div>
    </div>
  <div class="controls"><span class="button prev"></span><span class="button next"></span></div><div class="buttons"><span class="active">1</span><span>2</span><span>3</span></div></div><div id="fig-toggle" class="cf"><span>Hide Figures</span></div>

  <div class="nav-col">
    <div style="position: static;" class="nav" id="nav-article-page"><ul class="nav-page"><li class="active"><a href="#abstract0" class="scroll">Abstract</a></li><li><a href="#s1" class="scroll">Introduction</a></li><li><a href="#s2" class="scroll">Results</a></li><li><a href="#s3" class="scroll">Discussion</a></li><li><a href="#s4" class="scroll">Materials and Methods</a></li><li><a href="#ack" class="scroll">Acknowledgments</a></li><li><a href="#authcontrib" class="scroll">Author Contributions</a></li><li><a href="#references" class="scroll">References</a></li></ul>
      <ul>
        <li class="nav-col-comments"><a href="http://www.plosone.org/article/comments/info%3Adoi%2F10.1371%2Fjournal.pone.0053083">Reader Comments (0)</a></li>
          <li id="nav-figures"><a data-doi="info:doi/10.1371/journal.pone.0053083">Figures</a></li>
      </ul>
    </div>
  </div>

  <div class="article">







<div class="abstract"><a id="abstract0" name="abstract0" toc="abstract0" title="Abstract"></a><h2>Abstract</h2><a id="article1.front1.article-meta1.abstract1.p1" name="article1.front1.article-meta1.abstract1.p1"></a><p>The congenital muscular dystrophies (CMDs) comprise a heterogeneous group of heritable muscle disorders with often difficult to interpret muscle pathology, making them challenging to diagnose. Serial Sanger sequencing of suspected CMD genes, while the current molecular diagnostic method of choice, can be slow and expensive. A comprehensive panel test for simultaneous screening of mutations in all known CMD-associated genes would be a more effective diagnostic strategy. Thus, the CMDs are a model disorder group for development and validation of next-generation sequencing (NGS) strategies for diagnostic and clinical care applications. Using a highly multiplexed PCR-based target enrichment method (RainDance) in conjunction with NGS, we performed mutation detection in all CMD genes of 26 samples and compared the results with Sanger sequencing. The RainDance NGS panel showed great consistency in coverage depth, on-target efficiency, versatility of mutation detection, and genotype concordance with Sanger sequencing, demonstrating the test's appropriateness for clinical use. Compared to single tests, a higher diagnostic yield was observed by panel implementation. The panel's limitation is the amplification failure of select gene-specific exons which require Sanger sequencing for test completion. Successful validation and application of the CMD NGS panel to improve the diagnostic yield in a clinical laboratory was shown.</p>
</div>


<div class="articleinfo"><p><strong>Citation: </strong>Valencia CA, Ankala A, Rhodenizer D, Bhide S, Littlejohn MR, et al.  (2013) Comprehensive Mutation Analysis for Congenital Muscular Dystrophy: A Clinical PCR-Based Enrichment and Next-Generation Sequencing Panel. PLoS ONE 8(1):
          e53083.
            doi:10.1371/journal.pone.0053083</p><p><strong>Editor: </strong>Noam Shomron, 
        Tel Aviv University, Israel
      </p><p></p><p><strong>Received:</strong> October 12, 2012; <strong>Accepted:</strong> November 23, 2012; <strong>Published:</strong> January 11, 2013</p><p><strong>Copyright:</strong> © 2013 Valencia et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p><p><strong>Funding: </strong>This research was supported in part by PHS Grants (UL1 RR025008, KL2 R0025009, or TL1 RR025010) from the Clinical and Translational Science Award Program, National Institutes of Health (NIH), National Center for Research Resources. CB is supported by the intramural program of the National Institute of Neurological Disorders and Stroke, NIH. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p><strong>Competing interests:</strong> The authors have declared that no competing interests exist.</p></div>





<div id="section1" class="section"><a id="s1" name="s1" toc="s1" title="Introduction"></a><h3>Introduction</h3><a id="article1.body1.sec1.p1" name="article1.body1.sec1.p1"></a><p>The main objective of human genetics is to identify the genetic variants underlying specific phenotypes and provide molecular diagnoses to guide clinical management <a href="#pone.0053083-Hu1">[1]</a>. Genetic heterogeneity in inherited disorders including breast cancer, intellectual disability, ataxia, hearing loss, immunodeficiency, cardiomyopathies, and inherited muscle disorders such as the congenital muscular dystrophies has driven the development of novel screening and testing approaches <a href="#pone.0053083-Hoischen1">[2]</a>. Over the past decade, our molecular understanding of the congenital muscular dystrophies (CMDs) has expanded greatly <a href="#pone.0053083-Muntoni1">[3]</a>. CMDs are rare genetic muscle disorders that present early at birth or within the first 2 years of life, with variable inheritance patterns. Generally, they are characterized by congenital hypotonia, delayed motor development, progressive muscle weakness, and dystrophic features on muscle biopsy <a href="#pone.0053083-Reed1">[4]</a>. CMDs are genetically and phenotypically heterogeneous and include disorders caused by: 1) recessive or dominant <em>COL6A1</em>, <em>COL6A2</em>, and <em>COL6A3</em> mutations, which manifest as Ullrich or Bethlem CMD; 2) recessive <em>LAMA2</em> mutations, which result in merosin-deficient CMD (MDC1A); 3) recessive mutations in the <em>POMT1</em>, <em>POMT2</em>, <em>POMGNT1</em>, <em>FKRP</em>, <em>FKTN</em>, or <em>LARGE</em> genes, which manifest as purely muscular or syndromic conditions, such as Fukuyama CMD, muscle-eye-brain disease (MEB), Walker-Warburg syndrome (WWS), or CMD type 1C (MDC1C); 4) recessive mutations in the <em>SEPN1</em> gene, which manifest as rigid-spine syndrome (RSS); 5) dominant lamin A/C mutations, which result in the congenital form of Emery Dreifuss muscular dystrophy; and 6) recessive <em>ITGA7</em> mutations, which manifest as CMD with integrin A7 deficiency <a href="#pone.0053083-Reed1">[4]</a>, <a href="#pone.0053083-Schessl1">[5]</a>. Novel genes remain to be identified in patients with clinical collagen VI-like, dystroglycanopathy-like and not previously described phenotypes with congenital onset and dystrophic muscle biopsy findings. In a recent survey of a national UK referral service for CMD diagnostics, a genetic diagnosis was reached in 53 of 116 patients, with most common diagnoses being collagen VI related disorders (19%), dystroglycanopathy (12%) and merosin deficient congenital muscular dystrophy (10%) <a href="#pone.0053083-Clement1">[6]</a>.</p>
<a id="article1.body1.sec1.p2" name="article1.body1.sec1.p2"></a><p>Diagnosing a specific CMD subtypes may present a challenge and requires a multidisciplinary expertise (neurology, pathology, genetics, and neuroradiology). Currently, there are only a few centers with the expertise to recognize and delineate the wide range of overlapping clinical features <a href="#pone.0053083-Reed1">[4]</a>, <a href="#pone.0053083-Lisi1">[7]</a>, <a href="#pone.0053083-Reed2">[8]</a>. Even then, patients undergo a battery of immunostains and individual gene sequencing analyses to arrive at an exact diagnosis <a href="#pone.0053083-Mendell1">[9]</a>. The most frequently used CMD diagnostic method is sequential gene-by-gene mutation detection by Sanger sequencing. Sequencing a gene is rightly the preferred approach for mutation detection in CMD, since most mutations identified to date are either point mutations or small insertions and deletions. However, the difficulty in disease delineation and the increasing number of genes implicated mandate a more comprehensive molecular diagnostic approach to improve both diagnostic and cost efficiencies.</p>
<a id="article1.body1.sec1.p3" name="article1.body1.sec1.p3"></a><p>Within the last 5 years, high-throughput sequencing technology, referred to as next-generation sequencing (NGS) has successfully identified mutations in novel genes for a number of conditions, including Sensenbrenner syndrome, Kabuki syndrome, and Miller syndrome <a href="#pone.0053083-Gilissen1">[10]</a>–<a href="#pone.0053083-Ng2">[12]</a>. NGS facilitates target re-sequencing for rapid, accurate and lower cost diagnostic applications. However, with several target enrichment strategies, including microarray-based capture, in-solution capture, and polymerase chain reaction (PCR)-based amplification <a href="#pone.0053083-Albert1">[13]</a>–<a href="#pone.0053083-Kirkness1">[20]</a>, and NGS sequencing platforms, such as Roche 454 GS FLX, Illumina Genome Analyzer, Applied Biosystems SOLiD, Helicos Biosciences HeliScope, and Pacific Biosciences SMRT, being commercially available, selection and validation of the technologies becomes crucial <a href="#pone.0053083-Metzker1">[21]</a>, <a href="#pone.0053083-Voelkerding1">[22]</a>.</p>
<a id="article1.body1.sec1.p4" name="article1.body1.sec1.p4"></a><p>In this study, we describe the development and validation of an NGS panel, using RainDance (as enrichment technology) and Applied Biosystems SOLiD3 (as sequencing platform), for comprehensive mutation detection in CMD genes, along with its diagnostic yield and clinical implementation in patients with confirmed diagnosis, serving as positive controls, and with clinically suspected CMD (<a href="#pone-0053083-t001">Table 1</a>).</p>
<div class="figure" id="pone-0053083-t001"><div class="img"><a name="pone-0053083-t001" title="Click for larger image " href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0053083.t001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t001"><img src="fetchObject_003.action" alt="thumbnail" class="thumbnail"><div class="expand"></div></a></div><div class="figure-inline-download">

            Download:
            <ul><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t001/powerpoint">

                    PPT
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t001/powerpoint">

                  PowerPoint slide
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t001/largerimage">

                    PNG
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t001/largerimage">

                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t001.PNG_L">83KB</span>)
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t001/originalimage">

                    TIFF
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t001/originalimage">

                  original image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t001.TIF">475KB</span>)
                </a></li></ul></div><p><strong>Table 1.  <span>Summary of the clinical features of CMD patients.</span></strong></p><span>doi:10.1371/journal.pone.0053083.t001</span></div></div>

<div id="section2" class="section"><a id="s2" name="s2" toc="s2" title="Results"></a><h3>Results</h3>
<h4>Sequencing yields and optimal target base coverage</h4>
<a id="article1.body1.sec2.sec1.p1" name="article1.body1.sec2.sec1.p1"></a><p>The targeted next-generation sequencing panel was designed to amplify all exons of the 12 known CMD associated genes (<a href="#pone-0053083-t002">Table 2</a>). The overall target enrichment and next-generation sequencing yielded an average of 1,195,183 reads; 57% of these reads mapped to the genome and 35% mapped to the targeted regions. The total coverage of all targeted bases ranged from 87 to 94% at 5× and from 84 to 94% at 20×. The mean gene depth of coverage across all samples ranged from 0× for <em>POMT1</em> to 108× for <em>POMGNT1</em>, with an average of 50×. Despite the high mean gene read depth and target region coverage, several exons, including exon 1 of <em>SEPN1</em> and all exons of <em>POMT1</em> had no mapped reads <a href="#pone.0053083-Valencia1">[23]</a>.</p>
<div class="figure" id="pone-0053083-t002"><div class="img"><a name="pone-0053083-t002" title="Click for larger image " href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0053083.t002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t002"><img src="fetchObject_004.action" alt="thumbnail" class="thumbnail"><div class="expand"></div></a></div><div class="figure-inline-download">

            Download:
            <ul><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t002/powerpoint">

                    PPT
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t002/powerpoint">

                  PowerPoint slide
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t002/largerimage">

                    PNG
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t002/largerimage">

                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t002.PNG_L">92KB</span>)
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t002/originalimage">

                    TIFF
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t002/originalimage">

                  original image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t002.TIF">382KB</span>)
                </a></li></ul></div><p><strong>Table 2.  <span>CMD-associated genes included in the clinical CMD NGS panel.</span></strong></p><span>doi:10.1371/journal.pone.0053083.t002</span></div><a id="article1.body1.sec2.sec1.p2" name="article1.body1.sec2.sec1.p2"></a><p>Of all the exons targeted, 49 consistently showed less than 20× average coverage across all samples, which could be due largely to sequence complexity, problematic library synthesis, or unusual GC content of the fragments (<a href="#pone-0053083-t003">Table 3</a>). <em>SEPN1</em>, <em>COL6A1</em>, and <em>POMT2</em> had very high GC content, namely, 87%, 73%, and 73%, respectively, specifically in their first coding exon, while <em>POMT1</em> had an average of 56% GC content across all exons. In contrast, exons 34, 13, 6, 7, and 27, of <em>COL6A1</em>, <em>COL6A3</em>, <em>FKTN</em>, <em>FKTN</em>, and <em>LAMA2</em>, respectively, had very low GC content and consequently had no mapped reads in the NGS data.</p>
<div class="figure" id="pone-0053083-t003"><div class="img"><a name="pone-0053083-t003" title="Click for larger image " href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0053083.t003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t003"><img src="fetchObject_005.action" alt="thumbnail" class="thumbnail"><div class="expand"></div></a></div><div class="figure-inline-download">

            Download:
            <ul><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t003/powerpoint">

                    PPT
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t003/powerpoint">

                  PowerPoint slide
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t003/largerimage">

                    PNG
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t003/largerimage">

                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t003.PNG_L">23KB</span>)
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t003/originalimage">

                    TIFF
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t003/originalimage">

                  original image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t003.TIF">76KB</span>)
                </a></li></ul></div><p><strong>Table 3.  <span>Exons with consistently low coverage (&lt;20× average) across all samples.</span></strong></p><span>doi:10.1371/journal.pone.0053083.t003</span></div>

<h4>High variant detection rates in targeted regions with excellent coverage, phred scores, and allele percentages</h4>
<a id="article1.body1.sec2.sec2.p1" name="article1.body1.sec2.sec2.p1"></a><p>Known variants/mutations in the 5 variant-positive control samples re-detected by the CMD NGS test had coverage, Phred score, and allele percentages well above acceptable thresholds (<a href="#pone-0053083-t004">Table 4</a>). This included all previous Sanger-detected variants in exons not problematic for NGS. As expected, none of these variants were detected in the wild-type control (CMD-13) sample. However, additional variants (ranging from 0–14 per sample), not confirmed by Sanger sequencing, passed these threshold values and are thus false positives (<a href="#pone-0053083-t005">Table 5</a>). These false-positive calls are due to several factors including low coverage, low Phred scores and skewed allele percentages of those specific genomic areas. As shown in the <a href="#pone-0053083-t005">Table 5</a>, the false-positive rates ranged from 0% in CMD-11 to 37% in CMD-8. Overall, a total of 85 variants were detected in all 5 variant-positive control samples, of which 68 are reported in the dbSNP database and are, thus, true variants. This in turn reflects the low false-positive rate of the targeted approach via this panel test. Similarly, in the 20 blinded samples, 271 variants were detected, of which 98 are dbSNP calls (<a href="#pone-0053083-t006">Table 6</a>).</p>
<div class="figure" id="pone-0053083-t004"><div class="img"><a name="pone-0053083-t004" title="Click for larger image " href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0053083.t004&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t004"><img src="fetchObject_006.action" alt="thumbnail" class="thumbnail"><div class="expand"></div></a></div><div class="figure-inline-download">

            Download:
            <ul><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t004/powerpoint">

                    PPT
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t004/powerpoint">

                  PowerPoint slide
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t004/largerimage">

                    PNG
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t004/largerimage">

                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t004.PNG_L">102KB</span>)
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t004/originalimage">

                    TIFF
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t004/originalimage">

                  original image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t004.TIF">424KB</span>)
                </a></li></ul></div><p><strong>Table 4.  <span>Validation of NGS for variant/mutation detection on known CMD-positives samples.</span></strong></p><span>doi:10.1371/journal.pone.0053083.t004</span></div><div class="figure" id="pone-0053083-t005"><div class="img"><a name="pone-0053083-t005" title="Click for larger image " href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0053083.t005&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t005"><img src="fetchObject_007.action" alt="thumbnail" class="thumbnail"><div class="expand"></div></a></div><div class="figure-inline-download">

            Download:
            <ul><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t005/powerpoint">

                    PPT
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t005/powerpoint">

                  PowerPoint slide
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t005/largerimage">

                    PNG
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t005/largerimage">

                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t005.PNG_L">46KB</span>)
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t005/originalimage">

                    TIFF
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t005/originalimage">

                  original image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t005.TIF">193KB</span>)
                </a></li></ul></div><p><strong>Table 5.  <span>Sequence variants identified by NGS sequencing in control samples.</span></strong></p><span>doi:10.1371/journal.pone.0053083.t005</span></div><div class="figure" id="pone-0053083-t006"><div class="img"><a name="pone-0053083-t006" title="Click for larger image " href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0053083.t006&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t006"><img src="fetchObject_008.action" alt="thumbnail" class="thumbnail"><div class="expand"></div></a></div><div class="figure-inline-download">

            Download:
            <ul><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t006/powerpoint">

                    PPT
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t006/powerpoint">

                  PowerPoint slide
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t006/largerimage">

                    PNG
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t006/largerimage">

                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t006.PNG_L">69KB</span>)
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t006/originalimage">

                    TIFF
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t006/originalimage">

                  original image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t006.TIF">387KB</span>)
                </a></li></ul></div><p><strong>Table 6.  <span>Sequence variants identified by RainDance enrichment and NGS.</span></strong></p><span>doi:10.1371/journal.pone.0053083.t006</span></div>

<h4>Versatility of variant detection</h4>
<a id="article1.body1.sec2.sec3.p1" name="article1.body1.sec2.sec3.p1"></a><p>The ability of NGS to efficiently detect all kinds of mutations, including point mutations and small insertions/deletions, was interrogated using previously Sanger-confirmed variants or mutations in the variant-positive samples (<a href="#pone-0053083-t004">Table 4</a>). These variant-positive samples represented all the different types of variants NGS is expected to detect. These included silent, c.1770G&gt;C (p.T590) (CMD-8); missense, c.2084C&gt;T (p.D695V) (CMD-9); small deletion, c.6617delT (CMD-11); and small duplication, IVS24-3dupC (CMD-10) variants in genes <em>COL6A2</em>, <em>LAMA2</em>, <em>COL6A3</em>, and <em>COL6A2</em>, respectively (<a href="#pone-0053083-t004">Table 4</a>). Some of these insertion and deletion variants, as detected by NGS and confirmed by conventional Sanger sequencing, are represented in <a href="#pone-0053083-g001">Figure 1</a>. Additionally, potential causative mutations and variants of different types were identified in blinded samples and were concordant with previous Sanger sequencing results obtained in Dr. Carsten Bonnemann's laboratory (<a href="#pone-0053083-t007">Table 7</a>). All confirmed variants had a coverage of at least 7× and mutant allele percentage of 17–71% for heterozygous and 78–100% for homozygous variants (<a href="#pone-0053083-t004">Tables 4</a> and <a href="#pone-0053083-t007">7</a>).</p>
<div class="figure" id="pone-0053083-g001"><div class="img"><a name="pone-0053083-g001" title="Click for larger image " href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0053083.g001&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.g001"><img src="fetchObject_009.action" alt="thumbnail" class="thumbnail"><div class="expand"></div></a></div><div class="figure-inline-download">

            Download:
            <ul><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g001/powerpoint">

                    PPT
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g001/powerpoint">

                  PowerPoint slide
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g001/largerimage">

                    PNG
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g001/largerimage">

                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0053083.g001.PNG_L">3.03MB</span>)
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g001/originalimage">

                    TIFF
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g001/originalimage">

                  original image
                  (<span id="info:doi/10.1371/journal.pone.0053083.g001.TIF">3.83MB</span>)
                </a></li></ul></div><p><strong>Figure 1.  <span>Variant detection by NGS (top) and Sanger sequencing confirmation (bottom).</span></strong></p><a id="article1.body1.sec2.sec3.fig1.caption1.p1" name="article1.body1.sec2.sec3.fig1.caption1.p1"></a><p>(<em>A</em>) Example of an insertion mutation: IVS15-3-2insC in <em>COL6A2</em> identified byNGS (to the left) and confirmed by Sanger sequencing (to the right). (<em>B</em>) Example of a 30-bp deletion: IVS16-8del30 in <em>COL6A3</em> as represented by NGS analysis (top panel) and Sanger sequencing (bottom panel). The exact site or representation of the mutation is indicated by the red arrowheads.</p>
<span>doi:10.1371/journal.pone.0053083.g001</span></div><div class="figure" id="pone-0053083-t007"><div class="img"><a name="pone-0053083-t007" title="Click for larger image " href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0053083.t007&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t007"><img src="fetchObject_010.action" alt="thumbnail" class="thumbnail"><div class="expand"></div></a></div><div class="figure-inline-download">

            Download:
            <ul><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t007/powerpoint">

                    PPT
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t007/powerpoint">

                  PowerPoint slide
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t007/largerimage">

                    PNG
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t007/largerimage">

                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t007.PNG_L">90KB</span>)
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t007/originalimage">

                    TIFF
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t007/originalimage">

                  original image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t007.TIF">418KB</span>)
                </a></li></ul></div><p><strong>Table 7.  <span>Potential causative variants and mutations identified in the blinded clinical samples.</span></strong></p><span>doi:10.1371/journal.pone.0053083.t007</span></div>

<h4>Multiple parameters as data filters for identification of causative mutations</h4>
<a id="article1.body1.sec2.sec4.p1" name="article1.body1.sec2.sec4.p1"></a><p>For clinical applications, it is crucial to reduce the number of variants that need confirmation by Sanger sequencing (lower false-positive rate) to maintain an acceptable cost-benefit ratio. This was evaluated with the variant call data from known positive samples (CMD-8 to CMD-12) by including multiple parameters for variant filtering as illustrated in <a href="#pone-0053083-g002">Figure 2</a>. Following this, variants with coverage less than 20× were all filtered out, unless they were listed in HGMD, dbSNP, or EGL (Emory Genetics Laboratory) databases as definitive known pathogenic variants or mutations. Similarly, variants with high frequency (observed in multiple samples) were removed, unless they were found to be in HGMD, frameshift, or nonsense changes. Additionally, synonymous variants, dbSNP variants with allele frequency &gt;1%, were filtered out of the list as well. When the coverage is greater than 20×, variant calls with mutant allele percentages greater than 85% and less than 30% were retained as homozygous and heterozygous calls, respectively, and others were filtered out. Simultaneous consideration of the multiple parameters described above, for data filtering, significantly reduced the number of variant calls requiring Sanger confirmation (<a href="#pone-0053083-t005">Table 5</a>). Using these parameters, all potential disease-causing mutations previously identified by Carsten Bonnemann's laboratory, but initially blinded to EGL staff, were detected in the 20 blinded clinical samples (<a href="#pone-0053083-t007">Table 7</a>).</p>
<div class="figure" id="pone-0053083-g002"><div class="img"><a name="pone-0053083-g002" title="Click for larger image " href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0053083.g002&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.g002"><img src="fetchObject.action" alt="thumbnail" class="thumbnail"><div class="expand"></div></a></div><div class="figure-inline-download">

            Download:
            <ul><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g002/powerpoint">

                    PPT
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g002/powerpoint">

                  PowerPoint slide
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g002/largerimage">

                    PNG
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g002/largerimage">

                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0053083.g002.PNG_L">227KB</span>)
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g002/originalimage">

                    TIFF
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g002/originalimage">

                  original image
                  (<span id="info:doi/10.1371/journal.pone.0053083.g002.TIF">268KB</span>)
                </a></li></ul></div><p><strong>Figure 2.  <span>Schematic diagram of the analysis workflow.</span></strong></p><a id="article1.body1.sec2.sec4.fig1.caption1.p1" name="article1.body1.sec2.sec4.fig1.caption1.p1"></a><p>The flowchart demonstrates the criteria used to select variants that were Sanger sequencing confirmed. In essence, selected variants that had &gt;20× coverage, a low allele frequency, and nonsynonymous changes were Sanger confirmed if they were listed on HGMD, frameshift, or nonsense changes. In addition, interesting variants with a coverage &lt;20× were also confirmed (<a href="#pone-0053083-t007">Table 7</a>).</p>
<span>doi:10.1371/journal.pone.0053083.g002</span></div>

<h4>NGS panel approach has higher clinical yield compared to a sequential Sanger sequencing approach</h4>
<a id="article1.body1.sec2.sec5.p1" name="article1.body1.sec2.sec5.p1"></a><p>Since the NGS CMD panel was made available at EGL, a number of tests have been ordered and yielded a higher call percentage than single-gene tests (<a href="#pone-0053083-t008">Table 8</a>). Call percentage refers to the identification and report of pathogenic mutations in the respective gene. For example, <em>LAMA2</em> and <em>FKTN</em> yielded the highest percentages for the single-gene test, 64% and 35%, respectively. In contrast, panels 1 and 2, containing <em>LAMA2</em> and <em>FKTN</em>, had call percentages of 54% and 94%. Generally, the total percentage call for the gene-by-gene approach was 17% compared to the higher total call percentage of 41% obtained by the 4 NGS panels. The low call percentage or mutation detection rate for individual gene tests is due mostly to incorrect presumption of genetic etiology and suspicion of the incorrect gene.</p>
<div class="figure" id="pone-0053083-t008"><div class="img"><a name="pone-0053083-t008" title="Click for larger image " href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0053083.t008&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.t008"><img src="fetchObject_001.action" alt="thumbnail" class="thumbnail"><div class="expand"></div></a></div><div class="figure-inline-download">

            Download:
            <ul><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t008/powerpoint">

                    PPT
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t008/powerpoint">

                  PowerPoint slide
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t008/largerimage">

                    PNG
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t008/largerimage">

                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t008.PNG_L">84KB</span>)
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t008/originalimage">

                    TIFF
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.t008/originalimage">

                  original image
                  (<span id="info:doi/10.1371/journal.pone.0053083.t008.TIF">413KB</span>)
                </a></li></ul></div><p><strong>Table 8.  <span>Comparison of clinical yields of single-gene versus NGS panel tests.</span></strong></p><span>doi:10.1371/journal.pone.0053083.t008</span></div>
</div>

<div id="section3" class="section"><a id="s3" name="s3" toc="s3" title="Discussion"></a><h3>Discussion</h3><a id="article1.body1.sec3.p1" name="article1.body1.sec3.p1"></a><p>Recent studies have highlighted the potential of NGS in mutation detection <a href="#pone.0053083-Hoischen1">[2]</a>, <a href="#pone.0053083-tenBosch1">[24]</a>–<a href="#pone.0053083-Volpi1">[27]</a>; however, it remains imperative to query and validate its performance efficiency prior to implementation in a clinical testing laboratory. To this end, we investigated the potential of NGS technology and its efficiency as a clinical diagnostic tool by implementing a high-throughput gene panel for CMD. Target coverage in a panel approach varies based on the target enrichment method employed prior to NGS. The relatively high sensitivity, specificity, and accuracy of target enrichment for NGS offered by the highly multiplexed PCR-based technology (RainDance) over other hybridization technologies has been previously demonstrated for CMD <a href="#pone.0053083-Valencia1">[23]</a>.</p>
<a id="article1.body1.sec3.p2" name="article1.body1.sec3.p2"></a><p>The evaluation of NGS performance parameters is critical to offering such diagnostic strategies in a clinical laboratory. Our study design using negative (wild-type, CMD-13) and positive controls (variant-positive samples CMD-8 to CMD-12) demonstrated the efficiency of the NGS technology in detecting all potential mutation types (<a href="#pone-0053083-t004">Table 4</a>). The study also established the limitations of the technology. For the entire CMD panel, there were at least 49 exons across 9 genes that had significantly low coverage (&lt;20×) and required Sanger sequencing (<a href="#pone-0053083-t003">Table 3</a>). Among these 49 exons, were <em>POMT1</em> exons (19 exons) that failed amplification and hence had no coverage at all, most likely due to unusual GC content and sequence complexity. Amplification of exon 1 of most genes was problematic for similar reasons. <a href="#pone.0053083-Hu1">[1]</a>, <a href="#pone.0053083-Gnirke1">[18]</a>, <a href="#pone.0053083-Porreca1">[28]</a>. Empirically, the average failure rate for target enrichment is 15–20%. Though not comparable to the low failure rates of Sanger sequencing (3%: 2 out of 63 amplicons), the NGS failure rate is compensated for by other advantages of the technology <a href="#pone.0053083-Goossens1">[29]</a>. The flexibility of both batch processing and single sample processing offered by RainDance minimizes reagent wastage and maintains rapid turnaround times, even when processing is less frequently ordered for samples of rare disorders in diagnostic laboratories. One additional advantage of RainDance for target enrichment is the ability to differentiate genes from pseudogenes by the use of gene-specific primers for PCR amplification unlike other hybridization based technologies. Comparable low coverage (&lt;10× coverage in 20% of exons) has also been observed in non-CMD genes as reported by a recent study of ataxia gene targets involving array-based enrichment and NGS sequencing <a href="#pone.0053083-Hoischen1">[2]</a>. Sanger sequencing may remain the strategy of choice for confirmation of low-coverage variants.</p>
<a id="article1.body1.sec3.p3" name="article1.body1.sec3.p3"></a><p>Following this successful validation, a clinical CMD NGS panel was launched at the Emory Genetics Laboratory (EGL) and has been used successfully by clinicians in CMD cases presenting with overlapping phenotypes, inconclusive biochemical studies, non-diagnostic brain or muscle MRIs. This expedited approach to molecular diagnosis avoids the diagnostic odyssey and cost associated with a serial gene testing approach. For example, on average the number of exons comprising a CMD gene is 25 (ranging from 1–65), costing around $2500 per gene for molecular analysis, clinical interpretation, and report issuance. Alternatively, the NGS-based sequencing panel offered for just $5000 includes comprehensive analysis of the current12 disease-associated genes. In addition, as more and more disease-causing genes are identified, they can be added to the panel without a significant increase in the overall cost, which is very unlikely to be the case for a gene-be-gene approach. In this study, the CMD panel approach convincingly showed better mutation detection or diagnostic yield compared to a single-gene analysis (<a href="#pone-0053083-t008">Table 8</a>). The efficiency and better yield of the panel approach is better illustrated by the analysis of the 20 blinded samples included in our study (<a href="#pone-0053083-t007">Table 7</a>). Several samples, which underwent a series of single-gene tests, and others which remained CMDs of unknown molecular etiology due to inconclusive biochemical or immunologic assays, all received a definitive diagnosis through this NGS approach. Others, in which no mutations could be detected, could either be negative for all known genes or have larger deletions or duplications requiring array CGH analysis.</p>
<a id="article1.body1.sec3.p4" name="article1.body1.sec3.p4"></a><p>Although whole-exome sequencing is making its way into clinical genetics, the high false-positive rate, long turnaround times requiring 3 to 6 months for analysis and interpretation alone, ethical challenges involving secondary findings, and the high test price when data and interpretation are included, may present prohibitive barriers to commercial application <a href="#pone.0053083-Ng2">[12]</a>, <a href="#pone.0053083-Choi1">[30]</a>–<a href="#pone.0053083-Rios1">[32]</a>. Exome sequencing currently covers about 92% of the exome. From the experience of the whole exome analysis pipeline at EGL, approximately 10–20% of the 92% had low or zero coverage. The failed exons vary between individual runs and may involve genes of interest strongly associated with the patient phenotype. Given the large number of failed exons, they cannot practically be followed up and confirmed by Sanger sequencing, making the test necessarily incomplete. In contrast, targeted panel sequencing gives the laboratory the ability to complete the test by tracking all NGS failed exons and confirming by Sanger sequencing. These current limitations further highlight the significance of panel approaches over current gene-by-gene or potential whole exome-based tests. In agreement with the newly released CAP and Gargis et al. guidelines for clinical NGS validation and implementation, our study demonstrates that an NGS panel approach can be successfully adopted in clinical laboratories to improve disease diagnosis <a href="#pone.0053083-Gargis1">[33]</a>.</p>
<a id="article1.body1.sec3.p5" name="article1.body1.sec3.p5"></a><p>In conclusion, clinical CMD NGS panels offer cost-effective and more rapid turn-around molecular diagnostic testing than the conventional sequential Sanger sequencing of associated genes. A detailed step-wise approach to recommending NGS panel tests and diagnosing the disease subtype is illustrated as a flowchart (<a href="#pone-0053083-g003">Figure 3</a>). We anticipate that following this diagnostic algorithm will ensure a more efficient and rapid diagnosis for patients with CMD who currently lack molecular characterization.</p>
<div class="figure" id="pone-0053083-g003"><div class="img"><a name="pone-0053083-g003" title="Click for larger image " href="http://www.plosone.org/article/fetchObject.action?uri=info:doi/10.1371/journal.pone.0053083.g003&amp;representation=PNG_M" data-doi="info:doi/10.1371/journal.pone.0053083" data-uri="info:doi/10.1371/journal.pone.0053083.g003"><img src="fetchObject_002.action" alt="thumbnail" class="thumbnail"><div class="expand"></div></a></div><div class="figure-inline-download">

            Download:
            <ul><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g003/powerpoint">

                    PPT
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g003/powerpoint">

                  PowerPoint slide
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g003/largerimage">

                    PNG
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g003/largerimage">

                  larger image
                  (<span id="info:doi/10.1371/journal.pone.0053083.g003.PNG_L">309KB</span>)
                </a></li><li><div class="icon"><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g003/originalimage">

                    TIFF
                  </a></div><a href="http://www.plosone.org/article/info:doi/10.1371/journal.pone.0053083.g003/originalimage">

                  original image
                  (<span id="info:doi/10.1371/journal.pone.0053083.g003.TIF">424KB</span>)
                </a></li></ul></div><p><strong>Figure 3.  <span>Congenital muscular dystrophy next-generation sequencing algorithm.</span></strong></p><a id="article1.body1.sec3.fig1.caption1.p1" name="article1.body1.sec3.fig1.caption1.p1"></a><p>The diagram demonstrates the diagnostic work-up of congenital muscular dystrophies using RainDance enrichment in combination with next-generation sequencing.</p>
<span>doi:10.1371/journal.pone.0053083.g003</span></div></div>

<div id="section4" class="section"><a id="s4" name="s4" toc="s4" title="Materials and Methods"></a><h3>Materials and Methods</h3>
<h4>Patients/Samples</h4>
<a id="article1.body1.sec4.sec1.p1" name="article1.body1.sec4.sec1.p1"></a><p>To investigate whether an NGS panel can detect different types of mutations, 5 variant-positive samples (CMD-8, CMD-9, CMD-10, CMD-11, and CMD-12), each with a different type of mutation (missense, nonsense, insertion, deletion), were included in this study. These 5 variant-positive samples have been previously characterized for pathogenic variants/mutations through the currently available Sanger sequencing panels: CMD comprehensive panel in CMD-9 (all 12 CMD genes, <em>COL6A1</em>, <em>COL6A2</em>, <em>COL6A3</em>, <em>FKRP</em>, <em>FKTN</em>, <em>ITGA7</em>, <em>LAMA2</em>, <em>LARGE</em>, <em>POMGNT1</em>, <em>POMT1</em>, <em>POMT2</em>, and <em>SEPN1</em>); Bethlem myopathy/Ullrich CMD sequencing panel in CMD-8 and CMD-10 (<em>COL6A1</em>, <em>COL6A2</em>, and <em>COL6A3</em>); merosin-deficient CMD Type 1A (MDC1A) panel in CMD-11 (sequencing of <em>LAMA2</em>); and muscle-eye-brain disease (MEB) panel in CMD-12 (sequencing of <em>POMGNT1</em>). Additionally, a variant-negative wild-type control sample (CMD-13) expected to lack any pathogenic variants/mutations was also included.</p>
<a id="article1.body1.sec4.sec1.p2" name="article1.body1.sec4.sec1.p2"></a><p>To assess the clinical utility of the NGS panel, 20 samples from patients with clinically suspected CMD (diagnosed by Dr. Carsten Bonnemann, National Institutes of Health, Bethesda, MD, formerly at Children's Hospital of Philadelphia, PA) were analyzed. To this end, Emory Genetics Laboratory staff were blinded to the identity, clinical phenotype and prior genetic testing results of these 20 samples. Written informed consent was obtained from all patients and approved by the Children's Hospital of Philadelphia IRB. The clinical features of each patient are summarized in <a href="#pone-0053083-t001">Table 1</a>. All experiments were conducted at the Emory Genetics Laboratory (EGL), a CAP-accredited clinical laboratory.</p>


<h4>Primer Library Design</h4>
<a id="article1.body1.sec4.sec2.p1" name="article1.body1.sec4.sec2.p1"></a><p>A primer library for target amplification of all exons of the known CMD genes was designed using the manufacturer's design parameters (RainDance Technologies) and the Primer3 algorithm (<a href="http://frodo.wi.mit.edu/primer3/">http://frodo.wi.mit.edu/primer3/</a>). All SNPs from dbSNP build 130 were filtered from the primer selection region. The in-house primer design pipeline performed an exhaustive primer design and selection across the 65-kb targeted region. The final library consisted of primer pairs for successful amplification of 383 amplicons with Tm ranging from 57 to 59°C and primer length ranging from 16 to 21 bp.</p>


<h4>Target Enrichment by Droplet-Based Multiplex PCR (RainDance)</h4>
<a id="article1.body1.sec4.sec3.p1" name="article1.body1.sec4.sec3.p1"></a><p>Intact genomic DNA samples were sheared to yield 3–4 kb long fragments with the Covaris S2 instrument following the manufacturer's instructions. For target sequence amplification, input DNA template mixture was prepared by mixing 1.5 µg of the above sheared DNA fragments, 4.7 µl of High-Fidelity Buffer (Invitrogen), 1.26 µl of MgSO4 (Invitrogen), 1.6 µl of 10 mM dNTP (Invitrogen), 3.6 µl of 4 M Betaine (Sigma), 3.6 µl of RDT Droplet Stabilizer (RainDance Technologies), 1.8 µl of dimethyl sulfoxide (Sigma), 0.7 µl 5 units/µl of Platinum High-Fidelity Taq (Invitrogen), and nuclease-free water to bring to a final reaction volume of 25 µl, per the RainDance protocol. These samples were then subjected to emulsification using the RDT1000 instrument (RainDance Technologies) to generate individual PCR droplets. Droplets for each sample were automatically dispensed as an emulsion into separate PCR tubes and transferred to a standard thermal cycler for PCR amplification. Samples were cycled in an Applied Biosystems GeneAmp 9700 thermocycler as follows: initial denaturation at 94°C for 2 min, 55 cycles at 94°C for 15 s, 54°C for 15 s, 68°C for 30 s, final extension at 68°C for 10 min, and a 4°C hold.</p>
<a id="article1.body1.sec4.sec3.p2" name="article1.body1.sec4.sec3.p2"></a><p>After PCR amplification, an equal volume of RDT 1000 Droplet Destabilizer (RainDance Technologies) was added to each emulsion PCR tube, vortexed for 15 s, and centrifuged at 13,000 g for 5 min. The oil from below the aqueous phase was carefully removed from the sample. The remaining sample was then purified using a MinElute column (Qiagen) following the manufacturer's recommended protocol.</p>


<h4>NGS Sequencing by SOLiD3</h4>
<a id="article1.body1.sec4.sec4.p1" name="article1.body1.sec4.sec4.p1"></a><p>The ends of the amplicons were blunt end-repaired by adding the reagents to the purified DNA (diluted to 68 µl): 10 µl 10× Blunting Buffer (Epicentre), 10 µl 2.5 mM dNTP Mix (Invitrogen), 10 µl 10 mM ATP, 2 µl End-it enzyme mix (Epicentre), and sterile water to a total reaction volume of 100 µl. The reaction was incubated at room temperature for 30 min, and the DNA was immediately purified using Ampure XP beads (Agencourt). The amplicons were then concatenated using the NEB Quick Ligation kit according to the manufacturer's protocol. DNA was purified using Ampure XP beads and eluted in 105 µl of low TE. An Agilent 7500 Bioanalyzer chip was run to confirm the concatenation of PCR products. The sample was then fragmented and processed as described in the manufacturer's standard SOLiD3 workflow (Applied Biosystems).</p>


<h4>Validation of Variants and Mutations by Sanger Sequencing</h4>
<a id="article1.body1.sec4.sec5.p1" name="article1.body1.sec4.sec5.p1"></a><p>Exon-specific primers were designed to amplify individual exons along with 20–50 bp of flanking intronic regions on either side for <em>LAMA2</em>, <em>COL6A1</em>, <em>COL6A2</em>, <em>COL6A3</em>, <em>FKTN</em>, <em>POMGnT1</em>, <em>POMT1</em>, <em>POMT2</em>, <em>FKRP</em>, <em>LARGE</em>, <em>ITGA7</em>, and <em>SEPN1</em>. Samples were prepared by fluorescence sequencing on the ABI 3730XL DNA analyzer with BigDye Terminator chemistry and the BigDye XTerminator purification kit (Applied Biosystems). Individual sequences were aligned against reference sequences (downloaded from NCBI) using Mutation Surveyor v3.30 software (SoftGenetics) and analyzed for variations or mutations.</p>

</div>





<div><a id="ack" name="ack" toc="ack" title="Acknowledgments"></a><h3>Acknowledgments</h3>
<a id="article1.back1.ack1.p1" name="article1.back1.ack1.p1"></a><p>We thank the patients who participated and made this study possible.</p>
</div><div class="contributions"><a id="authcontrib" name="authcontrib" toc="authcontrib" title="Author Contributions"></a><h3>Author Contributions</h3><p>Conceived and designed the experiments: CAV DR SB MH. Performed the experiments: CAV DR AA SB MRL LMK. Analyzed the data: CAV AA DR SB MRL LMK MH. Contributed reagents/materials/analysis tools: CAV AR CB SS MH. Wrote the paper: CAV AA MH.</p></div><div><a id="references" name="references" toc="references" title="References"></a><h3>References</h3><ol class="references"><li><span class="label">1.
              </span><a name="pone.0053083-Hu1" id="pone.0053083-Hu1"></a>Hu H, Wrogemann K, Kalscheuer V, Tzschach A, Richard H, et al. (2009) Mutation screening in 86 known X-linked mental retardation genes by droplet-based multiplex PCR and massive parallel sequencing. Hugo J 3: 41–49  doi:<a href="http://dx.doi.org/10.1007/s11568-010-9137-y">10.1007/s11568-010-9137-y</a>. <ul class="find-nolinks"></ul></li><li><span class="label">2.
              </span><a name="pone.0053083-Hoischen1" id="pone.0053083-Hoischen1"></a>Hoischen A, Gilissen C, Arts P, Wieskamp N, van der Vliet W, et al. (2010) Massively parallel sequencing of ataxia genes after array-based enrichment. Hum Mutat 31: 494–499  doi:<a href="http://dx.doi.org/10.1002/humu.21221">10.1002/humu.21221</a>. <ul class="find-nolinks"></ul></li><li><span class="label">3.
              </span><a name="pone.0053083-Muntoni1" id="pone.0053083-Muntoni1"></a>Muntoni F, Voit T (2004) The congenital muscular dystrophies in 2004: a century of exciting progress. Neuromuscul Disord 14: 635–649  doi:<a href="http://dx.doi.org/10.1016/j.nmd.2004.06.009">10.1016/j.nmd.2004.06.009</a>. <ul class="find-nolinks"></ul></li><li><span class="label">4.
              </span><a name="pone.0053083-Reed1" id="pone.0053083-Reed1"></a>Reed UC (2009) Congenital muscular dystrophy. Part I: a review of phenotypical and diagnostic aspects. Arq Neuropsiquiatr 67: 144–168.
        doi:
        <a href="http://dx.doi.org/10.1590/S0004-282X2009000100038">10.1590/S0004-282X2009000100038</a>.
       <ul class="find" data-citedarticleid="6592891" data-doi="10.1590/S0004-282X2009000100038"><li><a href="http://dx.doi.org/10.1590/S0004-282X2009000100038" onclick="window.open(this.href, 'ambraFindArticle', ''); return false;" title="Go to article in CrossRef">

                            CrossRef
                          </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reed%5Bauthor%5D+AND+Congenital+muscular+dystrophy.+Part+I%3A+a+review+of+phenotypical+and+diagnostic+aspects" onclick="window.open(this.href, 'ambraFindArticle', ''); return false;" title="Go to article in PubMed">

                            PubMed/NCBI
                          </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AReed+%22Congenital+muscular+dystrophy.+Part+I%3A+a+review+of+phenotypical+and+diagnostic+aspects%22" onclick="window.open(this.href, 'ambraFindArticle', ''); return false;" title="Go to article in Google Scholar">

                            Google Scholar
                          </a></li></ul></li><li><span class="label">5.
              </span><a name="pone.0053083-Schessl1" id="pone.0053083-Schessl1"></a>Schessl J, Zou Y, Bönnemann CG (2006) Congenital muscular dystrophies and the extracellular matrix. Semin Pediatr Neurol 13: 80–89  doi:<a href="http://dx.doi.org/10.1016/j.spen.2006.06.003">10.1016/j.spen.2006.06.003</a>. <ul class="find-nolinks"></ul></li><li><span class="label">6.
              </span><a name="pone.0053083-Clement1" id="pone.0053083-Clement1"></a>Clement EM, Feng L, Mein R, Sewry CA, Robb SA, et al. (2012) Relative frequency of congenital muscular dystrophy subtypes: analysis of the UK diagnostic service 2001–2008. Neuromuscul Disord 22: 522–527  doi:<a href="http://dx.doi.org/10.1016/j.nmd.2012.01.010">10.1016/j.nmd.2012.01.010</a>. <ul class="find-nolinks"></ul></li><li><span class="label">7.
              </span><a name="pone.0053083-Lisi1" id="pone.0053083-Lisi1"></a>Lisi MT, Cohn RD (2007) Congenital muscular dystrophies: new aspects of an expanding group of disorders. Biochim Biophys Acta 1772: 159–172  doi:<a href="http://dx.doi.org/10.1016/j.bbadis.2006.09.006">10.1016/j.bbadis.2006.09.006</a>. <ul class="find-nolinks"></ul></li><li><span class="label">8.
              </span><a name="pone.0053083-Reed2" id="pone.0053083-Reed2"></a>Reed UC (2009) Congenital muscular dystrophy. Part II: a review of pathogenesis and therapeutic perspectives. Arq Neuropsiquiatr 67: 343–362.
        doi:
        <a href="http://dx.doi.org/10.1590/S0004-282X2009000200035">10.1590/S0004-282X2009000200035</a>.
       <ul class="find" data-citedarticleid="6592899" data-doi="10.1590/S0004-282X2009000200035"><li><a href="http://dx.doi.org/10.1590/S0004-282X2009000200035" onclick="window.open(this.href, 'ambraFindArticle', ''); return false;" title="Go to article in CrossRef">

                            CrossRef
                          </a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Search&amp;doptcmdl=Citation&amp;defaultField=Title+Word&amp;term=Reed%5Bauthor%5D+AND+Congenital+muscular+dystrophy.+Part+II%3A+a+review+of+pathogenesis+and+therapeutic+perspectives" onclick="window.open(this.href, 'ambraFindArticle', ''); return false;" title="Go to article in PubMed">

                            PubMed/NCBI
                          </a></li><li><a href="http://scholar.google.com/scholar?hl=en&amp;safe=off&amp;q=author%3AReed+%22Congenital+muscular+dystrophy.+Part+II%3A+a+review+of+pathogenesis+and+therapeutic+perspectives%22" onclick="window.open(this.href, 'ambraFindArticle', ''); return false;" title="Go to article in Google Scholar">

                            Google Scholar
                          </a></li></ul></li><li><span class="label">9.
              </span><a name="pone.0053083-Mendell1" id="pone.0053083-Mendell1"></a>Mendell JR, Boué DR, Martin PT (2006) The congenital muscular dystrophies: recent advances and molecular insights. Pediatr Dev Pathol 9: 427–443  doi:<a href="http://dx.doi.org/10.2350/06-07-0127.1">10.2350/06-07-0127.1</a>. <ul class="find-nolinks"></ul></li><li><span class="label">10.
              </span><a name="pone.0053083-Gilissen1" id="pone.0053083-Gilissen1"></a>Gilissen C, Arts HH, Hoischen A, Spruijt L, Mans DA, et al. (2010) Exome sequencing identifies WDR35 variants involved in Sensenbrenner syndrome. Am J Hum Genet 87: 418–423  doi:<a href="http://dx.doi.org/10.1016/j.ajhg.2010.08.004">10.1016/j.ajhg.2010.08.004</a>. <ul class="find-nolinks"></ul></li><li><span class="label">11.
              </span><a name="pone.0053083-Ng1" id="pone.0053083-Ng1"></a>Ng SB, Bigham AW, Buckingham KJ, Hannibal MC, McMillin MJ, et al. (2010) Exome sequencing identifies MLL2 mutations as a cause of Kabuki syndrome. Nat Genet 42: 790–793  doi:<a href="http://dx.doi.org/10.1038/ng.646">10.1038/ng.646</a>. <ul class="find-nolinks"></ul></li><li><span class="label">12.
              </span><a name="pone.0053083-Ng2" id="pone.0053083-Ng2"></a>Ng SB, Buckingham KJ, Lee C, Bigham AW, Tabor HK, et al. (2010) Exome sequencing identifies the cause of a mendelian disorder. Nat Genet 42: 30–35  doi:<a href="http://dx.doi.org/10.1038/ng.499">10.1038/ng.499</a>. <ul class="find-nolinks"></ul></li><li><span class="label">13.
              </span><a name="pone.0053083-Albert1" id="pone.0053083-Albert1"></a>Albert TJ, Molla MN, Muzny DM, Nazareth L, Wheeler D, et al. (2007) Direct selection of human genomic loci by microarray hybridization. Nat Methods 4: 903–905  doi:<a href="http://dx.doi.org/10.1038/nmeth1111">10.1038/nmeth1111</a>. <ul class="find-nolinks"></ul></li><li><span class="label">14.
              </span><a name="pone.0053083-Hodges1" id="pone.0053083-Hodges1"></a>Hodges E, Rooks M, Xuan Z, Bhattacharjee A, Benjamin Gordon D, et al. (2009) Hybrid selection of discrete genomic intervals on custom-designed microarrays for massively parallel sequencing. Nat Protoc 4: 960–974  doi:<a href="http://dx.doi.org/10.1038/nprot.2009.68">10.1038/nprot.2009.68</a>. <ul class="find-nolinks"></ul></li><li><span class="label">15.
              </span><a name="pone.0053083-Hodges2" id="pone.0053083-Hodges2"></a>Hodges E, Xuan Z, Balija V, Kramer M, Molla MN, et al. (2007) Genome-wide in situ exon capture for selective resequencing. Nat Genet 39: 1522–1527  doi:<a href="http://dx.doi.org/10.1038/ng.2007.42">10.1038/ng.2007.42</a>. <ul class="find-nolinks"></ul></li><li><span class="label">16.
              </span><a name="pone.0053083-Okou1" id="pone.0053083-Okou1"></a>Okou DT, Steinberg KM, Middle C, Cutler DJ, Albert TJ, et al. (2007) Microarray-based genomic selection for high-throughput resequencing. Nat Methods 4: 907–909  doi:<a href="http://dx.doi.org/10.1038/nmeth1109">10.1038/nmeth1109</a>. <ul class="find-nolinks"></ul></li><li><span class="label">17.
              </span><a name="pone.0053083-Nikolaev1" id="pone.0053083-Nikolaev1"></a>Nikolaev SI, Iseli C, Sharp AJ, Robyr D, Rougemont J, et al. (2009) Detection of genomic variation by selection of a 9 mb DNA region and high throughput sequencing. PLoS ONE 4: e6659  doi:<a href="http://dx.doi.org/10.1371/journal.pone.0006659">10.1371/journal.pone.0006659</a>. <ul class="find-nolinks"></ul></li><li><span class="label">18.
              </span><a name="pone.0053083-Gnirke1" id="pone.0053083-Gnirke1"></a>Gnirke A, Melnikov A, Maguire J, Rogov P, LeProust EM, et al. (2009) Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing. Nat Biotechnol 27: 182–189  doi:<a href="http://dx.doi.org/10.1038/nbt.1523">10.1038/nbt.1523</a>. <ul class="find-nolinks"></ul></li><li><span class="label">19.
              </span><a name="pone.0053083-Tewhey1" id="pone.0053083-Tewhey1"></a>Tewhey R, Warner JB, Nakano M, Libby B, Medkova M, et al. (2009) Microdroplet-based PCR enrichment for large-scale targeted sequencing. Nat Biotechnol 27: 1025–1031  doi:<a href="http://dx.doi.org/10.1038/nbt.1583">10.1038/nbt.1583</a>. <ul class="find-nolinks"></ul></li><li><span class="label">20.
              </span><a name="pone.0053083-Kirkness1" id="pone.0053083-Kirkness1"></a>Kirkness EF (2009) Targeted sequencing with microfluidics. Nat Biotechnol 27: 998–999  doi:<a href="http://dx.doi.org/10.1038/nbt1109-998">10.1038/nbt1109-998</a>. <ul class="find-nolinks"></ul></li><li><span class="label">21.
              </span><a name="pone.0053083-Metzker1" id="pone.0053083-Metzker1"></a>Metzker ML (2010) Sequencing technologies - the next generation. Nat Rev Genet 11: 31–46  doi:<a href="http://dx.doi.org/10.1038/nrg2626">10.1038/nrg2626</a>. <ul class="find-nolinks"></ul></li><li><span class="label">22.
              </span><a name="pone.0053083-Voelkerding1" id="pone.0053083-Voelkerding1"></a>Voelkerding KV, Dames SA, Durtschi JD (2009) Next-generation sequencing: from basic research to diagnostics. Clin Chem 55: 641–658  doi:<a href="http://dx.doi.org/10.1373/clinchem.2008.112789">10.1373/clinchem.2008.112789</a>. <ul class="find-nolinks"></ul></li><li><span class="label">23.
              </span><a name="pone.0053083-Valencia1" id="pone.0053083-Valencia1"></a>Valencia CA, Rhodenizer D, Bhide S, Chin E, Littlejohn MR, et al. (2012) Assessment of target enrichment platforms using massively parallel sequencing for the mutation detection for congenital muscular dystrophy. J Mol Diagn 14: 233–246  doi:<a href="http://dx.doi.org/10.1016/j.jmoldx.2012.01.009">10.1016/j.jmoldx.2012.01.009</a>. <ul class="find-nolinks"></ul></li><li><span class="label">24.
              </span><a name="pone.0053083-tenBosch1" id="pone.0053083-tenBosch1"></a>ten Bosch JR, Grody WW (2008) Keeping up with the next generation: massively parallel sequencing in clinical diagnostics. J Mol Diagn 10: 484–492  doi:<a href="http://dx.doi.org/10.2353/jmoldx.2008.080027">10.2353/jmoldx.2008.080027</a>. <ul class="find-nolinks"></ul></li><li><span class="label">25.
              </span><a name="pone.0053083-Daiger1" id="pone.0053083-Daiger1"></a>Daiger SP, Sullivan LS, Bowne SJ, Birch DG, Heckenlively JR, et al. (2010) Targeted high-throughput DNA sequencing for gene discovery in retinitis pigmentosa. Adv Exp Med Biol 664: 325–331  doi:<a href="http://dx.doi.org/10.1007/978-1-4419-1399-9_37">10.1007/978-1-4419-1399-9_37</a>. <ul class="find-nolinks"></ul></li><li><span class="label">26.
              </span><a name="pone.0053083-Rehman1" id="pone.0053083-Rehman1"></a>Rehman AU, Morell RJ, Belyantseva IA, Khan SY, Boger ET, et al. (2010) Targeted capture and next-generation sequencing identifies C9orf75, encoding taperin, as the mutated gene in nonsyndromic deafness DFNB79. Am J Hum Genet 86: 378–388  doi:<a href="http://dx.doi.org/10.1016/j.ajhg.2010.01.030">10.1016/j.ajhg.2010.01.030</a>. <ul class="find-nolinks"></ul></li><li><span class="label">27.
              </span><a name="pone.0053083-Volpi1" id="pone.0053083-Volpi1"></a>Volpi L, Roversi G, Colombo EA, Leijsten N, Concolino D, et al. (2010) Targeted next-generation sequencing appoints c16orf57 as clericuzio-type poikiloderma with neutropenia gene. Am J Hum Genet 86: 72–76  doi:<a href="http://dx.doi.org/10.1016/j.ajhg.2009.11.014">10.1016/j.ajhg.2009.11.014</a>. <ul class="find-nolinks"></ul></li><li><span class="label">28.
              </span><a name="pone.0053083-Porreca1" id="pone.0053083-Porreca1"></a>Porreca GJ, Zhang K, Li JB, Xie B, Austin D, et al. (2007) Multiplex amplification of large sets of human exons. Nat Methods 4: 931–936  doi:<a href="http://dx.doi.org/10.1038/nmeth1110">10.1038/nmeth1110</a>. <ul class="find-nolinks"></ul></li><li><span class="label">29.
              </span><a name="pone.0053083-Goossens1" id="pone.0053083-Goossens1"></a>Goossens D, Moens LN, Nelis E, Lenaerts A-S, Glassee W, et al. (2009) Simultaneous mutation and copy number variation (CNV) detection by multiplex PCR-based GS-FLX sequencing. Hum Mutat 30: 472–476  doi:<a href="http://dx.doi.org/10.1002/humu.20873">10.1002/humu.20873</a>. <ul class="find-nolinks"></ul></li><li><span class="label">30.
              </span><a name="pone.0053083-Choi1" id="pone.0053083-Choi1"></a>Choi M, Scholl UI, Ji W, Liu T, Tikhonova IR, et al. (2009) Genetic diagnosis by whole exome capture and massively parallel DNA sequencing. Proc Natl Acad Sci USA 106: 19096–19101  doi:<a href="http://dx.doi.org/10.1073/pnas.0910672106">10.1073/pnas.0910672106</a>. <ul class="find-nolinks"></ul></li><li><span class="label">31.
              </span><a name="pone.0053083-Ng3" id="pone.0053083-Ng3"></a>Ng SB, Turner EH, Robertson PD, Flygare SD, Bigham AW, et al. (2009) Targeted capture and massively parallel sequencing of 12 human exomes. Nature 461: 272–276  doi:<a href="http://dx.doi.org/10.1038/nature08250">10.1038/nature08250</a>. <ul class="find-nolinks"></ul></li><li><span class="label">32.
              </span><a name="pone.0053083-Rios1" id="pone.0053083-Rios1"></a>Rios J, Stein E, Shendure J, Hobbs HH, Cohen JC (2010) Identification by whole-genome resequencing of gene defect responsible for severe hypercholesterolemia. Hum Mol Genet 19: 4313–4318  doi:<a href="http://dx.doi.org/10.1093/hmg/ddq352">10.1093/hmg/ddq352</a>. <ul class="find-nolinks"></ul></li><li><span class="label">33.
              </span><a name="pone.0053083-Gargis1" id="pone.0053083-Gargis1"></a>Gargis AS, Kalman L, Berry MW, Bick DP, Dimmock DP, et al. (2012) Assuring the quality of next-generation sequencing in clinical laboratory practice. Nat Biotechnol 30: 1033–1036  doi:<a href="http://dx.doi.org/10.1038/nbt.2403">10.1038/nbt.2403</a>. <ul class="find-nolinks"></ul></li></ol></div>

  </div>

<!--end corrections-->

      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.XML" value="149126" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.PDF" value="1877849" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t001.TIF" value="475104" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t001.PNG_I" value="15617" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t001.PNG_S" value="2970" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t001.PNG_M" value="36695" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t001.PNG_L" value="83279" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t002.TIF" value="382124" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t002.PNG_I" value="16212" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t002.PNG_S" value="2963" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t002.PNG_M" value="38773" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t002.PNG_L" value="91778" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t003.TIF" value="75704" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t003.PNG_I" value="11743" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t003.PNG_S" value="2606" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t003.PNG_M" value="25301" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t003.PNG_L" value="22695" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t004.TIF" value="423532" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t004.PNG_I" value="18445" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t004.PNG_S" value="3080" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t004.PNG_M" value="43244" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t004.PNG_L" value="102299" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t005.TIF" value="192918" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t005.PNG_I" value="9083" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t005.PNG_S" value="2139" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t005.PNG_M" value="20959" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t005.PNG_L" value="46282" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t006.TIF" value="387336" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t006.PNG_I" value="13053" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t006.PNG_S" value="2825" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t006.PNG_M" value="28890" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t006.PNG_L" value="69162" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g001.TIF" value="3827902" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g001.PNG_I" value="137687" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g001.PNG_S" value="22679" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g001.PNG_M" value="260559" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g001.PNG_L" value="3032281" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t007.TIF" value="418310" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t007.PNG_I" value="16024" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t007.PNG_S" value="2996" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t007.PNG_M" value="37892" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t007.PNG_L" value="90065" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g002.TIF" value="268332" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g002.PNG_I" value="50070" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g002.PNG_S" value="14392" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g002.PNG_M" value="106131" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g002.PNG_L" value="226605" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t008.TIF" value="413428" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t008.PNG_I" value="15123" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t008.PNG_S" value="2903" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t008.PNG_M" value="35873" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.t008.PNG_L" value="84044" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g003.TIF" value="424028" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g003.PNG_I" value="39427" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g003.PNG_S" value="16840" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g003.PNG_M" value="66271" type="hidden">
      <input class="assetSize" name="info:doi/10.1371/journal.pone.0053083.g003.PNG_L" value="308510" type="hidden">

</div>
<div class="sidebar">

  <div class="article-actions cf">
    <div class="download">
      <span class="btn"><a href="http://www.plosone.org/article/fetchObject.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0053083&amp;representation=PDF" title="Download article PDF" target="_top">Download PDF</a></span>
    </div>

    <div class="btn-reveal dropdown">
      <div class="dropdown-icon">
        <span class="btn">&nbsp;</span>
      </div>
      <div class="content">
        <ul class="bullet">
          <li><a href="http://www.plosone.org/article/citationList.action?articleURI=info%3Adoi%2F10.1371%2Fjournal.pone.0053083" title="Download citations">Citation</a></li>
          <li><a href="http://www.plosone.org/article/fetchObjectAttachment.action?uri=info%3Adoi%2F10.1371%2Fjournal.pone.0053083&amp;representation=XML" title="Download article XML">XML</a></li>
        </ul>
      </div>
    </div>

    <div class="btn-reveal flt-l">
      <span class="btn">Print</span>
      <div class="content">
        <ul class="bullet">
          <li style="display: list-item;" id="print-article"><a href="#" onclick="if(typeof(_gaq) != 'undefined'){ _gaq.push(['_trackEvent','Article', 'Print', 'Click']); } window.print(); return false;" title="Print Article">Print article</a></li>
          <li>
            <a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=7&amp;doi=10.1371/journal.pone.0053083&amp;volume=&amp;issue=&amp;title=Comprehensive%20Mutation%20Analysis%20for%20Congenital%20Muscular%20Dystrophy:%20A%20Clinical%20PCR-Based%20Enrichment%20and%20Next-Generation%20Sequencing%20Panel&amp;author_name=C.%20Alexander%20Valencia%2C%20Arunkanth%20Ankala%2C%20Devin%20Rhodenizer%2C%20Shruti%20Bhide%2C%20Martin%20Robert%20Littlejohn%2C%20Lisa%20Mari%20Keong%2C%20Anne%20Rutkowski%2C%20Susan%20Sparks%2C%20Carsten%20Bonnemann%2C%20Madhuri%20Hegde&amp;start_page=1&amp;end_page=11" title="Odyssey Press">EzReprint</a>
          </li>
        </ul>
      </div>
    </div>

    <div class="btn-reveal flt-r">
      <span class="btn">Share</span>
      <div class="content">
        <ul class="social">
          <li><a href="http://www.reddit.com/submit?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0053083" target="_top" title="Submit to Reddit"><img src="icon.reddit.16.png" alt="Reddit" height="16" width="16">Reddit</a></li>

          <li><a href="https://plus.google.com/share?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0053083" target="_top" title="Share on Google+"><img src="icon.gplus.16.png" alt="Google+" height="16" width="16">Google+</a></li>

          <li><a href="http://www.stumbleupon.com/submit?url=http%3A%2F%2Fwww.plosone.org%2Farticle%2Finfo%253Adoi%252F10.1371%252Fjournal.pone.0053083" target="_top" title="Add to StumbleUpon"><img src="icon.stumble.16.png" alt="StumbleUpon" height="16" width="16">StumbleUpon</a></li>

          <li><a href="http://www.facebook.com/share.php?u=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0053083&amp;t=Comprehensive%20Mutation%20Analysis%20for%20Congenital%20Muscular%20Dystrophy%3A%20A%20Clinical%20PCR-Based%20Enrichment%20and%20Next-Generation%20Sequencing%20Panel" target="_top" title="Share on Facebook"><img src="icon.fb.16.png" alt="Facebook" height="16" width="16">Facebook</a></li>

          <li><a href="http://www.linkedin.com/shareArticle?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0053083&amp;title=Comprehensive%20Mutation%20Analysis%20for%20Congenital%20Muscular%20Dystrophy%3A%20A%20Clinical%20PCR-Based%20Enrichment%20and%20Next-Generation%20Sequencing%20Panel&amp;summary=Checkout%20this%20article%20I%20found%20at%20PLOS" target="_top" title="Add to LinkedIn"><img src="icon.linkedin.16.png" alt="Mendeley" height="16" width="16">LinkedIn</a></li>

          <li><a href="http://www.citeulike.org/posturl?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0053083&amp;title=Comprehensive%20Mutation%20Analysis%20for%20Congenital%20Muscular%20Dystrophy%3A%20A%20Clinical%20PCR-Based%20Enrichment%20and%20Next-Generation%20Sequencing%20Panel" target="_top" title="Add to CiteULike"><img src="icon.cul.16.png" alt="CiteULike" height="16" width="16">CiteULike</a></li>

          <li><a href="http://www.mendeley.com/import/?url=http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0053083" target="_top" title="Add to Mendeley"><img src="icon.mendeley.16.png" alt="Mendeley" height="16" width="16">Mendeley</a></li>

          <script type="text/javascript">

            // replace tweet with one that's pre-shortened to 140 chars
            function truncateTweetText() {
              var twtTitle = 'Comprehensive Mutation Analysis for Congenital Muscular Dystrophy: A Clinical PCR-Based Enrichment and Next-Generation Sequencing Panel';
              var twtUrl = 'http://dx.plos.org/10.1371/journal.pone.0053083';
              // all URLs posted to twitter get auto-shortened to 20 chars.
              var maxLength = 140 - (20 + 1);
              // truncate the title to include space for twtTag and ellipsis (here, 10 = tag length + space + ellipsis)
              if (twtTitle.length > maxLength) { twtTitle = twtTitle.substr(0, (maxLength - 10)) + '...'; }
              // set the href to use the shortened tweet
              $('#twitter-share-link').prop('href', 'http://twitter.com/intent/tweet?text=' + encodeURIComponent('#PLOSONE: ' + twtTitle + ' ' + twtUrl));
            }
          </script>
          <li><a href="http://twitter.com/intent/tweet?text=#PLOSONE%3A%20Comprehensive%20Mutation%20Analysis%20for%20Congenital%20Muscular%20Dystrophy%3A%20A%20Clinical%20PCR-Based%20Enrichment%20and%20Next-Generation%20Sequencing%20Panel%20http%3A%2F%2Fdx.plos.org%2F10.1371%2Fjournal.pone.0053083" onclick="truncateTweetText();" target="_top" title="Share on Twitter" id="twitter-share-link"><img src="icon.twtr.16.png" alt="Twitter" height="16" width="16">Twitter</a></li>

          <li><a href="http://www.plosone.org/article/email/info%3Adoi%2F10.1371%2Fjournal.pone.0053083" title="Email this article"><img src="icon.email.16.png" alt="Email" height="16" width="16">Email</a></li>
        </ul>
      </div>
    </div><!--end btn-reveal flt-r-->
  </div><!-- end article-actions-->



    <div class="block" id="subject-area-sidebar-block">
      <div class="header">
        <h3>Subject Areas</h3><div title="More information" id="subject-area-sidebar-block-help-icon"><img alt="info" src="button_info.png" align="right"><div id="subject-area-sidebar-block-help"><img src="button_info.png" align="right"><p>
            <b>We want your feedback.</b> Do these subject areas make sense for this article? If not, click the flag
        next to the incorrect subject area and we will review it. Thanks for your help!
        </p></div></div>
      </div>


      <ul id="subject-area-sidebar-list">








        <li>
            <a href="http://www.plosone.org/search/advanced?unformattedQuery=subject%3A%22Diagnostic+medicine%22" title="Search for articles in the subject area:'Diagnostic medicine'"><div class="flagText">Diagnostic medicine</div></a>
          <div data-categoryid="3267" data-articleid="139359" data-categoryname="Diagnostic medicine" class="flagImage" title="Flag 'Diagnostic medicine' as inappropriate"></div>
        </li>
        <li>
            <a href="http://www.plosone.org/search/advanced?unformattedQuery=subject%3A%22Gene+targeting%22" title="Search for articles in the subject area:'Gene targeting'"><div class="flagText">Gene targeting</div></a>
          <div data-categoryid="17701" data-articleid="139359" data-categoryname="Gene targeting" class="flagImage" title="Flag 'Gene targeting' as inappropriate"></div>
        </li>
        <li>
            <a href="http://www.plosone.org/search/advanced?unformattedQuery=subject%3A%22Muscular+dystrophies%22" title="Search for articles in the subject area:'Muscular dystrophies'"><div class="flagText">Muscular dystrophies</div></a>
          <div data-categoryid="20551" data-articleid="139359" data-categoryname="Muscular dystrophies" class="flagImage" title="Flag 'Muscular dystrophies' as inappropriate"></div>
        </li>
        <li>
            <a href="http://www.plosone.org/search/advanced?unformattedQuery=subject%3A%22Mutation%22" title="Search for articles in the subject area:'Mutation'"><div class="flagText">Mutation</div></a>
          <div data-categoryid="31809" data-articleid="139359" data-categoryname="Mutation" class="flagImage" title="Flag 'Mutation' as inappropriate"></div>
        </li>
        <li>
            <a href="http://www.plosone.org/search/advanced?unformattedQuery=subject%3A%22Insertion+mutation%22" title="Search for articles in the subject area:'Insertion mutation'"><div class="flagText">Insertion mutation</div></a>
          <div data-categoryid="35247" data-articleid="139359" data-categoryname="Insertion mutation" class="flagImage" title="Flag 'Insertion mutation' as inappropriate"></div>
        </li>
        <li>
            <a href="http://www.plosone.org/search/advanced?unformattedQuery=subject%3A%22Gene+deletion%22" title="Search for articles in the subject area:'Gene deletion'"><div class="flagText">Gene deletion</div></a>
          <div data-categoryid="33017" data-articleid="139359" data-categoryname="Gene deletion" class="flagImage" title="Flag 'Gene deletion' as inappropriate"></div>
        </li>
        <li>
            <a href="http://www.plosone.org/search/advanced?unformattedQuery=subject%3A%22Dideoxy+DNA+sequencing%22" title="Search for articles in the subject area:'Dideoxy DNA sequencing'"><div class="flagText">Dideoxy DNA sequencing</div></a>
          <div data-categoryid="25399" data-articleid="139359" data-categoryname="Dideoxy DNA sequencing" class="flagImage" title="Flag 'Dideoxy DNA sequencing' as inappropriate"></div>
        </li>
        <li>
            <a href="http://www.plosone.org/search/advanced?unformattedQuery=subject%3A%22Congenital+disorders%22" title="Search for articles in the subject area:'Congenital disorders'"><div class="flagText">Congenital disorders</div></a>
          <div data-categoryid="17739" data-articleid="139359" data-categoryname="Congenital disorders" class="flagImage" title="Flag 'Congenital disorders' as inappropriate"></div>
        </li>
      </ul>
    </div>

  <div class="ad">
    <div class="title">Advertisement</div>






  <iframe id="a0852f54" name="a0852f54" src="infodoi10.1371journal.pone_2.html" frameborder="0" height="600" scrolling="no" width="160">

    <a href='http://ads.plos.org/www/delivery/ck.php?n=a0852f54&amp;cb=2198'
      target='_top'><img src='http://ads.plos.org/www/delivery/avw.php?zoneid=381&amp;cb=3904&amp;n=a0852f54'
      border='0' alt=''/>
    </a>
  </iframe>



  </div>


</div><!-- sidebar -->
    </div>
  </div>
</div>
<script src="p_widget.js" type="text/javascript"></script><script src="plos_widget.js" type="text/javascript"></script><script src="jwplayer.js" type="text/javascript"></script><div id="pageftr">
  <div class="ftr-cols cf">
    <div class="col col-1">
      <img src="logo-plos-footer.png" alt="PLOS Logo" class="logo">
      <p><a href="http://www.plosone.org/static/releaseNotes">Ambra 2.8.4</a> Managed Colocation provided <br>by <a href="http://www.isc.org/">Internet Systems Consortium</a>.</p><p>
      </p><div class="nav nav-aux">
        <a href="http://www.plosone.org/static/privacy">Privacy Policy</a> |
        <a href="http://www.plosone.org/static/terms">Terms of Use</a> |
        <a href="http://www.plos.org/advertise/">Advertise</a> |
        <a href="http://www.plos.org/about/media-inquiries/">Media Inquiries</a>
      </div>
    </div>
    <div class="col col-2">
      <p><a href="http://www.plos.org/publications/journals/">Publications</a></p>
      <div class="nav">
        <ul>
          <li><a href="http://www.plosbiology.org/">PLOS Biology</a></li>
          <li><a href="http://www.plosmedicine.org/">PLOS Medicine</a></li>
          <li><a href="http://www.ploscompbiol.org/">PLOS Computational Biology</a></li>
          <li><a href="http://currents.plos.org/">PLOS Currents</a></li>
          <li><a href="http://www.plosgenetics.org/">PLOS Genetics</a></li>
          <li><a href="http://www.plospathogens.org/">PLOS Pathogens</a></li>
          <li><a href="http://www.plosone.org/">PLOS ONE</a></li>
          <li><a href="http://www.plosntds.org/">PLOS Neglected Tropical Diseases</a></li>
        </ul>
      </div>
    </div>
    <div class="col col-3">
      <div class="nav">
        <p><a href="http://www.plos.org/">plos.org</a></p>
        <p><a href="http://blogs.plos.org/">Blogs</a></p>
        <p><a href="http://www.ploscollections.org/">Collections</a></p>
        <p><a href="http://www.plosone.org/feedback/new">Send us feedback</a></p>
      </div>
    </div>
  </div>
</div><!-- pageftr -->

</div><!-- end page-wrap, this div is in header.ftl -->
<script type="text/javascript" src="jquery-1.8.1-min.js"></script>
<script type="text/javascript" src="ga-min.js"></script>
<script type="text/javascript" src="jquery.hoverIntent-min.js"></script>
<script type="text/javascript" src="jquery.placeholder-min.js"></script>
<script type="text/javascript" src="jquery.jsonp-2.4.0-min.js"></script>
<script type="text/javascript" src="jquery-ui-1.9.0-min.js"></script>
<script type="text/javascript" src="jquery.tooltip-min.js"></script>
<script type="text/javascript" src="jquery.uniform-min.js"></script>
<script type="text/javascript" src="jquery.pjax-min.js"></script>
<script type="text/javascript" src="global-min.js"></script>
<script type="text/javascript" src="alm-min.js"></script>
<script type="text/javascript" src="taxonomy-browser-min.js"></script>
<script type="text/javascript" src="imagesloaded-min.js"></script>
<script type="text/javascript" src="jquery.filterize-min.js"></script>
<script type="text/javascript" src="plosone-min.js"></script>
<script type="text/javascript">

  var _sf_async_config={uid:16579,domain:"plosone.org"};
  (function(){
    function loadChartbeat() {
      window._sf_endpt=(new Date()).getTime();
      var e = document.createElement('script');
      e.setAttribute('language', 'javascript');
      e.setAttribute('type', 'text/javascript');
      e.setAttribute('src',
          (("https:" == document.location.protocol) ? "https://a248.e.akamai.net/chartbeat.download.akamai.com/102508/" : "http://static.chartbeat.com/") +
              "js/chartbeat.js");
      document.body.appendChild(e);
    }
    var oldonload = window.onload;
    window.onload = (typeof window.onload != 'function') ?
        loadChartbeat : function() { oldonload(); loadChartbeat(); };
  })();
</script>


<div id="title-banner"><div class="content"><div class="btn-g"><img src="logo.plos.95.png" alt="PLOS logo" class="btn-logo"><a href="#close" class="btn-close">close</a></div>

  <h1 property="dc:title" datatype="" rel="dc:type" href="http://purl.org/dc/dcmitype/Text">

    Comprehensive Mutation Analysis for Congenital Muscular Dystrophy: A Clinical PCR-Based Enrichment and Next-Generation Sequencing Panel
  </h1>

  <ul class="authors">
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            C. Alexander Valencia
              <span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: C. Alexander Valencia, Arunkanth Ankala</p>

              
              

                <p>Affiliations:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America, 
                  Division of Human Genetics, Cincinnati Children's Hospital Medical Center, Cincinnati, Ohio, United States of America, 
                  Department of Pediatrics, University of Cincinnati Medical School, Cincinnati, Ohio, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Arunkanth Ankala
              <span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">

                <p><span class="equal-contrib" title="These authors contributed equally to this work">equal contributor</span>
                Contributed equally to this work with: C. Alexander Valencia, Arunkanth Ankala</p>

              
              

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Devin Rhodenizer, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Shruti Bhide, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Martin Robert Littlejohn, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Lisa Mari Keong, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Anne Rutkowski, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Cure CMD and Kaiser SCPMG, Los Angeles, California, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Susan Sparks, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Carolinas Medical Center, Charlotte, North Carolina, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Carsten Bonnemann, 
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              

                <p>Affiliation:
                  Neuromuscular and Neurogenetic Disorders of Childhood Section, Neurogenetics Branch, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, Maryland, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
      <li>


        <span rel="dc:creator" class="author">
          <span class="person" property="foaf:name" typeof="foaf:Person">

            Madhuri Hegde
              <span class="corresponding">mail</span>
          </span>
        </span>

          <div class="author_meta">
            <div class="author_inner">


              
              <p><span class="email">* E-mail:</span> <a href="mailto:mhegde@emory.edu">mhegde@emory.edu</a></p>

                <p>Affiliation:
                  Department of Human Genetics, Emory University School of Medicine, Atlanta, Georgia, United States of America
                </p>


              <span class="close">X</span>

            </div>
          </div>
      </li>
  </ul>

</div></div></body>
</html>
